\chapter{\label{ch:5-bulk}Bulk RNA-seq analysis of PRS inhibitors}

%\minitoc

\section{Introduction}
Although MM treatment has improved significantly in the last 10-20 years, MM remains an incurable disease.
Most MM patients relapse and become resistant to drugs they have previously been treated with.
Therefore, research into novel therapeutics that can overcome multi-drug resistance and can be used to treat relapsed patients is of great importance.

There is a new exciting class of compounds for the treatment of many diseases, including numerous cancers.
This class of compounds are analogues of, and have been derived from the drug Febrifugine.
Febrifugine is the biologically active component of the herb \textit{Dichroa febrifuga}, which is considered one of the fundamental herbs in traditional Chinese medicine.
Febrifugine was shown to have strong anti-malarial effects.
One such Febrifugine derivative, Halofuginone (HF), has been shown to inhibit T Helper 17 (TH17) cell differentiation, by activating the amino acid response (AAR)\cite{sundrud2009halofuginone}.
Halofuginone inhibits the enzyme glutamyl-prolyl tRNA synthetase (EPRS).
EPRS is a bifunctional aminoacyl-tRNA synthetase (AARS) and catalyses the the aminoacylation of glutamic acid and proline tRNA species (such that it charges its cognate tRNAs with glutamic acid and proline).
Halofuginone and Febrifugine compete with proline at the prolyl-tRNA synthetase active site of EPRS, specifically targeting utilisation of proline during translation\cite{keller2012halofuginone}.
This results in an accumulation of uncharged tRNA\textsuperscript{pro}s, giving the same cellular environment as if the cell were proline deficient, triggering the AAR to respond to the apparent proline deprived state.

AARSs are essential in protein synthesis, aiding in building chains of amino acids.
Human cancer cells often have an increased rate of protein synthesis, this is especially true in multiple myeloma, creating huge amounts of non-functional paraprotein, therefore are more reliant on aaRSs.
As discussed in chapter \ref{ch:2-litreview}, HF has previously shown anti-MM activity in vitro and in-vivo.
HF induced cytotoxicity and apoptosis in numerous MM cell-lines and primary MM cells.
HF was also shown to inhibit MM growth and prolong survival in a mouse xenograft MM model.
However, the mechanism by which HF exerted its affect was not elucidated, and it is not clear if the AAR plays a role in HF's effectiveness in MM.

It has also been shown that HF's anti-MM effect can be reduced in the presence of excessive proline.
Tumours have more proline than healthy cells <REF>.
This means that HF has a very narrow therapeutic window, and exhibits many side effects.

% Ralph drug drawing
\begin{figure}[ht]
    \centering
    \includegraphics[width=0.9\textwidth]{figures/Results/Transcriptomics/ralph_figure.pdf}
    \caption[Halofuginone and NCP26 structures]{Diagrams of Halofuginone/MAZ1392 (a) and NCP26 (b) and their chemical structures.
    Halofuginone is an ATP dependent, proline and tRNA competitive ProRS inhibitor.
    NCP26 is an ATP competitive and proline uncompetitive ProRS inhibitor.
    Aminoacylation is an ATP-dependent process, requiring ATP to activate amino acids.
    Figures by Ralph Mazitschek.
    }
    \label{fig:ralph_diagrams}
\end{figure}

Recently, The Mazitschek group have synthesized numerous other compounds which target the ProRS site of EPRS.
One such example, NCP26 (figure \ref{fig:ralph_diagrams}), binds to the ATP binding site of ProRS, inhibiting utilization of ATP.
NCP26 does not compete with proline for the active site of ProRS, unlike Febrifugine and Halofuginone.
Aminoacylation is an ATP-dependent process so blocking ATP binding inhibits this process, and also leads to an accumulation of uncharged tRNA\textsuperscript{pro}s.
NCP26 will hopefully alleviate the issues associated with HF treatment.
More PRS inhibitors have been synthesized by the group, including NCP22.

% Flush figures
\clearpage

\section{Assay results}

\subsection{Halofuginone and NCP26 are cytotoxic to drug sensitive and drug resistant MM cell lines in a dose-dependent manner}
The effect of the PRS inhibitors NCP22, NCP26, MAZ1805 (Halofuginol) and MAZ1392 (Halofuginone) on cell viability was investigated using the MM cell lines AMO-1 and L363 CFZ-r.
The cell lines were treated with a range of concentrations of each compound (3.9\si{\nano\Molar}- 1\si{\micro\Molar}).
Cell viability was assessed using presto-blue assays (section \ref{subsec:method_doseresponse}), and dose response curves were generated (Figures \ref{fig:dose}A and C).
Halofuginone and NCP26 reduced cell viability of PI-sensitive AMO-1 cells and carfilzomib resistant L363 cells in a dose-dependent fashion.
For this concentration range, MAZ1805 and NCP22 seemed to have little effect on cell viability of WT or CFZ-r cells, and IC50 values were unable to be calculated.
Halofuginone was found to be more cytotoxic/potent(??) than NCP26.

% dose response figure
\begin{figure}[h]
\centering
\includegraphics[width=0.9\textwidth]{figures/Results/dose_response/MAZ_ncp_wt_cfz.pdf}
\caption[PRS inhibitor dose response curves]{Multiple myeloma (MM) cell lines treated with PRS inhibitors- MAZ1392 (Halofuginone), MAZ1805, NCP26 and NCP22.
MM cell lines were treated for 48 hours with a range of concentrations (3.91\si{\nano\Molar}-1\si{\micro\Molar}) of PRS inhibitors.
a) and c) Dose response curves.
a and b) WT AMO-1 cells.
c) and d) Carfilzomib resistant L363 cells.
b) and d) Proportion of cells viable after 48 hours of 1\si{\micro\Molar} treatment with each agent.}
\label{fig:dose}
\end{figure}
%%

For WT AMO-1 cells, Halofugione had an IC\textsubscript{50} of 141.8nM and NCP26 an IC\textsubscript{50} of 502nM. For CFZ-r cells, Halofuginone had an IC50 of 1185nM, NCP26's IC\textsubscript{50} for CFZ-r was ambiguous, a higher stock concentration would be required for calculation.
Figure \ref{fig:dose}B and D show the proportion of viable cells following 48 hours of treatment of the PRS inhibitors.
WT AMO-1 cells were found to be more sensitive to NCP26 and Halofuginone treatment than carfilzomib resistant L363 cells.
This may indicate some acquired cross-resistance built up from carfilzomib exposure.

\subsection{Carfilzomib and NCP26 have an additive or mild antagonistic effect together}\label{subsec:synergy}
Drug combinations have proved effective in MM in recent years, for example the combination of bortezomib, lenalidomide, and dexamethasone (VRd) is used extensively for newly diagnosed MM patients.
Drugs are often used in combination so that outcomes are improved (synergistic efficacy) or to reduce off-target effects and toxicity by minimizing the doses of the drugs (synergistic potency) \cite{meyer2019quantifying}.

To assess if NCP26 and carfilzomib work together synergistically, AMO-1 cells were treated with varying concentrations of NCP26 and Carfilzomib for 72 hours, then presto blue assays were performed to determine cell viability (see figure \ref{fig:synergy}).
SynergyFinder \cite{zheng2021synergyfinder} was used to calculate the compounds' synergy scores (-4.66 ZIP; -4.18 Loewe; -5.53 Bliss).
From these values it is unlikely that NCP26 and Carfilzomib work together synergistically.
NCP26 and carfilzomib seem to have an additive effect together, or slight antagonistic effect.
This reflects a previous result where HF demonstrated moderate antagonism with the bortezomib \cite{leiba2012halofuginone}.

% synergy figure
\begin{figure}[h]
\centering
\includegraphics[width=0.9\textwidth]{figures/Results/Transcriptomics/synergy.pdf}
\caption[NCP26 and carfilzomib synergism]{Investigating potential synergy between NCP26 and carfilzomib.
AMO-1 cells were treated with varying concentration combinations of NCP26 and carfilzomib for 72 hours, then cell viability was determined using presto blue assays.
\textbf{A)} Dose response curves for NCP26 with different carfilzomib concentrations.
\textbf{B)} Matrix view of NCP26 and Carfilzomib concentration responses.
}
\label{fig:synergy}\end{figure}
%%

% Flush figures
\clearpage

\section{Bulk RNA-seq}

\subsection{Experiment overview}
The transcriptome of drug-sensitive and drug-resistant MM cells treated with four ProRS inhibitors was investigated using bulk RNA-seq.
PI-sensitive WT AMO-1 cells and carfilzomib resistant L363 cells (CFZr) were used.
Cells were treated for 6 and 24 hours with a DMSO control, or 1\si{\micro\Molar} of a ProRS inhibitor (MAZ1392 (Halofuginone), NCP26, NCP22 and MAZ1805 (Halofuginol)).
WT cells were also treated with 100\si{\nano\Molar} carfilzomib.
CFZr cells were treated in the presence of 100\si{\nano\Molar} carfilzomib.
The computational workflow for bulk RNA-seq analysis is outlined in section ...

\subsection{Clustering}

% bulk clustering figure
\begin{figure}[p]
\centering
\includegraphics[width=0.9\textwidth]{figures/Results/Transcriptomics/bulk_clustering_maz_ncp_wt_cfz.pdf}
\caption[Bulk RNA-seq sample clustering]{Bulk RNA-seq sample clustering.
WT AMO-1 cells and CFZr L363 cells treated for 6 or 24 hours with a DMSO control, 100nM carfilzomib or 1uM of a PRS inhibitor (MAZ1392/Halofuginone, MAZ1805/Halofuginol, NCP26 and NCP22).
Clustering analysis of sample-sample distances (a, c and e) and principal component analysis (PCA; b, d and f).
a) and b) both cell types (WT and CFZr) displayed; c) and d) WT AMO-1 only; e) and f) CFZr samples only.
}
\label{fig:clustering_bulk}\end{figure}
%%

Figure \ref{fig:clustering_bulk} shows clustering analysis of the samples.
Figure \ref{fig:clustering_bulk} A and B show the samples distinctly separate into respective cell types, and this makes up the majority (69\%) of the variance in the dataset.
Therefore, the different cell types (WT and CFZr) were separated and analysed individually.

The less active compounds from IC\textsubscript{50} data, MAZ1805 and NCP22, cluster closely with the DMSO-treated controls.
The more active inhibitors, NCP26 and Halofuginone, cluster separately from DMSO controls and less active inhibitors, indicating they have elicited a stronger transcriptional response.
At 6 hours NCP26 and Halofuginone cluster closely together.
At 24 hours NCP26 and Halofuginone separate more.
This may suggest that they work slightly differently in the cells, or this could just be reflecting the differences in their potency.

In WT cells, carfilzomib clusters separately from DMSO and the less active inhibitors.
At 6 hours, carfilzomib is separate from the NCP26 and Halofuginone cluster, but clusters closer at 24 hours.
This could suggest an initial difference in mechanism of action and transcriptional response to the ProRS inhibitors, but culminating in a similar response as time progresses, such as cell stress and apoptotic markers.

% PCA pathway figure
\begin{figure}[htb]
\centering
\includegraphics[width=0.9\textwidth]{figures/Results/Transcriptomics/bulk_clustering_PCs_analysed.pdf}
\caption[PCA pathway enrichment]{PCA pathway enrichment.
Pathway enrichment analysis performed using REACTOME of top contributing genes from principal component analysis (PCA).
A) WT AMO-1 dataset with carfilzomib-treated samples removed.
PC1 seems to account for the separation between DMSO controls/ less active compounds and the more active ProRS inhibitors (NCP26 and Halofuginone).
Genes contributing positively towards PC1 are upregulated in NCP26/ Halofuginone compared to controls/ less active compounds.
Enriched pathways from top genes in PC1 shown beneath PCA plot.
B) CFZr L363 dataset.
PC2 seems to account for the separation between DMSO controls/ less active compounds and the more active ProRS inhibitors (NCP26 and Halofuginone).
Genes contributing negatively towards PC2 (down arrow), are upregulated in NCP26/ Halofuginone compared to controls.
Pathways enriched from top genes in PC2 shown beneath PCA plot.}
\label{fig:pca_pathway}
\end{figure}
%%

The principal components of the WT AMO-1 and CFZr datasets were examined more closely.
Carfilzomib-treated samples were removed from the WT dataset, and differential expression, variance stabilising transformation and PCA was performed again.
This was to ensure that the difference between controls and active ProRS inhibitors was captured in PC1 or PC2 following dimensionality reduction.
Pathway enrichment analysis was performed for the top genes from the principal components accounting for the difference between controls and Halofuginone/NCP26 treatment (PC1 for WT cells, PC2 for CFZr cells; Figure \ref{fig:pca_pathway}).
The pathways `ATF4 activates genes in response to ER stress', `Response of GCN2 to amino acid deficiency' and `cytosolic tRNA aminoacylation' were all enriched, suggesting that the amino acid response is activated.
Additionally, genes involved in the cell cycle and G1/S transition contributed to this treatment-related separation.

\subsection{Drug sensitive MM}

\subsubsection{Differential expression}
For WT cells at 6 hours, 2119 genes were differentially expressed (adjusted p value< 0.05 and log2FC > 0.5; DE) for NCP26 treated-samples, 3019 DE genes (DEGs) for Halofuginone, 33 DE genes for MAZ1805, 218 DE genes for NCP22, and 983 DE genes for carfilzomib-treated samples compared to DMSO controls.
At 24 hours, NCP26-treated samples had 3323 DE genes, Halofuginone 3426, MAZ1805 2, NCP22 1, and carfilzomib 2260 DE genes compared to DMSO controls.
DEGs for NCP26 and Halofuginone treatment are shown in figure xyz.

% DE volcano scatter WT %
\begin{figure}[htb]
\centering
\includegraphics[width=0.9\textwidth]{figures/Results/Transcriptomics/WT/wt_de_volcano_scatter.pdf}
\caption[Differentially expressed genes WT AMO-1 cells]{Differentially expressed genes (DEGs) for Halofuginone and NCP26 treated WT AMO-1 cells at 6 and 24 hours.
Scatter plot of genes for 6 hours treated vs 24 hours treated.
Red points indicate genes which are differentially expressed (p\textsubscript{adj} < 0.01) at both 6 and 24 hours.
Volcano plots are also shown.
Blue points indicate downregulated DEGs (p\textsubscript{adj} < 0.01 \& logFC < -1).
Red points indicate upregulated DEGs (p\textsubscript{adj} < 0.01 \& logFC > 1).
Top DEGs are annotated with HGNC symbols.
}
\label{fig:wt_de}
\end{figure}
%%

PATHWAY enrichment analysis...

% AAR heatmap
\begin{figure}[p]
\centering
\includegraphics[width=0.5\textwidth]{figures/Results/Transcriptomics/WT/wt_halo_ncp26_aar_heatmap.pdf}
\caption[Amino acid starvation response genes heatmap WT cells]{Amino acid starvation response (AAR) genes heatmap for WT cells.
Differentially expressed genes (DEGs) from WT AMO-1 cells intersected with genes involved in the AAR.
A list of known AAR genes was compiled by collating AAR genesets from the Molecular Signatures Database (MSigDb).
The colour scale shows $\log_{2}$ fold change of expression for each treated sample, compared to its DMSO time control.
Red indicates an upregulated gene and blue a downregulated gene.
}
\label{fig:wt_aar_heatmap}
\end{figure}
%%

AAR genesets `GOBP RESPONSE TO AMINO ACID STARVATION', `REACTOME RESPONSE OF EIF2AK4 GCN2 TO AMINO ACID DEFICIENCY' and `KRIGE AMINO ACID DEPRIVATION' were collated from the Molecular Signatures Database (MSigDb), making up a list of <XYZ number> unique genes.
This list of AAR genes was intersected with genes that were differentially expressed for any of the ProRS inhibitor-treated samples (i.e NCP26 vs DMSO and Halofuginone vs DMSO at 6 or 24 hours).
Figure \ref{fig:wt_aar_heatmap} shows a heatmap of DE AAR genes for WT AMO-1 cells.
The AAR transcription factors \textit{ATF3}, \textit{DDIT3} (CHOP), \textit{CEBPB} and \textit{CEBPG} are all markedly upregulated following NCP26 and Halofuginone treatment.
Genes coding for aminoacyl tRNA synthetases are also shown to be upregulated, such as \textit{WARS1}, \textit{SARS1} and \textit{CARS1}.
Amino acid transporters, such as \textit{SLC7A11}, were also upregulated following ProRS treatment.

Figure <ref> 5.8 shows a heatmap for the response genes for ATF4 (genesets `REACTOME ATF4 ACTIVATES GENES IN RESPONSE TO ENDOPLASMIC RETICULUM STRESS' and `ATF4 Q2') and DEG genes for ProRS inhibitors.

%%% FIGURE FOR ATF4 response genes

Taken together it is clear that the amino acid starvation response is activated in MM cells by NCP26 and HF treatment.

%% vs carfilzomib
\subsubsection{ProRS inhibitors vs carfilzomib}
The mechanism of action of proteome inhibitors has been extensively studied and well described.
Figure \ref{fig:clustering_bulk}d demonstrates similarities at 24 hours between the transcriptional effects of ProRS inhibitors and the proteasome inhibitor carfilzomib on AMO-1 cells.
It also demonstrates ProRS and carfilzomib-treated samples separation at 6 hours, highlighting differences in their initial mechanism of action.


\subsection{Carfilzomib resistant cells}


% DE volcano scatter WT %
\begin{figure}[htb]
\centering
\includegraphics[width=0.9\textwidth]{figures/Results/Transcriptomics/CFzr/cfz_de_volcano_scatter.pdf}
\caption[Differentially expressed genes CFZr L363 cells]{Differentially expressed genes (DEGs) for Halofuginone and NCP26 treated CFZr L363 cells at 6 and 24 hours.
Scatter plot of genes for 6 hours treated vs 24 hours treated.
Red points indicate genes which are differentially expressed (p\textsubscript{adj} < 0.01) at both 6 and 24 hours.
Volcano plots are also shown.
Blue points indicate downregulated DEGs (p\textsubscript{adj} < 0.01 \& logFC < -1).
Red points indicate upregulated DEGs (p\textsubscript{adj} < 0.01 \& logFC > 1).
Top DEGs are annotated with HGNC symbols.
}
\label{fig:cfz_de}
\end{figure}
%%

% AAR heatmap
\begin{figure}[p]
\centering
\includegraphics[width=0.5\textwidth]{figures/Results/Transcriptomics/CFzr/cfz_halo_ncp26_aar_heatmap.pdf}
\caption[Amino acid starvation response genes heatmap CFZr cells]{Amino acid starvation response (AAR) genes heatmap for CFZr cells.
Differentially expressed genes (DEGs) from CFZr AMO-1 cells intersected with genes involved in the AAR.
A list of known AAR genes was compiled by collating AAR genesets from the Molecular Signatures Database (MSigDb).
}
\label{fig:cfz_aar_heatmap}
\end{figure}
%%


%\subsection{Difference between Halofuginone and NCP26}

