\chapter{\label{ch:3-methods}Methods}

%\minitoc

\section{Cell culture}
\subsection{AMO-1 cells}
AMO-1 cells, plasma cells from a 64-year old female myeloma patient, were used as a model cell-line for multiple myeloma.
Proteasome inhibitor-sensitive AMO-1 cells are referred to as WT cells.
Bortezomib resistant (aBTZ) and carfilzomib resistant cells (aCFZ), believed to be AMO-1 cells were generated and gifted by the Driessen lab\cite{soriano2016proteasome}.
After typing these cells, they were found to be a mix of AMO-1 cells and L363 cells.
AMO-1 cells were cultivated in RPMI-1640 medium (Thermofisher, UK), supplemented with 10\% fetal bovine serum (FBS) and 2\si{\milli\Molar} L-glutamine (Invitrogen, UK).
Cells were passaged when they reached approximately 1.5-2 million cells per \ml{}.
AMO-1 cells are suspension cells and were split twice a week to approximately 0.5 million cells per \ml{}.
All media was replaced with fresh media every two to three weeks, or prior to performing experiments with the cells.

\subsection{L363 cells}
After typing the cells gifted by the Driessen lab, they were found to be a mix of AMO-1 MM cells and L363 MM cells.
In-house PI-resistant cell lines were produced by Dr James Dunford by continual and escalating drug exposure of drug-sensitive (WT) AMO-1 cells.
However after these cells were typed, they were found to be L363 cells.
This was due to the drug exposure selecting the L363 contaminate population over the AMO-1 cells, due to their natural increased resistance to PI, compared to AMO-1 cells.
Once this mistake made by our collaborators was noticed, WT L363 cells were purchased.
WT, aCFZ and aBTZ cells were cultivated in RPMI-1640 medium (Thermofisher, UK), supplemented with 10\% fetal bovine serum (FBS) and 2\si{\milli\Molar} L-glutamine (Invitrogen, UK), and kept in 100\si{\nano\Molar} of their respective proteasome inhibitor.
Cells were passaged when they reached approximately 1.5-2 million cells per \ml{}.
L363 cells are suspension cells and were split twice a week to approximately 0.5 million cells per \ml{}.
All media was replaced with fresh media every two to three weeks, or prior to performing experiments with the cells.

L363 cells resistant to NCP26 (NCP26R) were also developed by Dr James Dunford by continual and escalating drug exposure of WT L363 cells to NCP26.

\section{Compounds}

\subsection{Proteasome inhibitors}
Stock concentrations of proteasome inhibitors, bortezomib (BTZ) and carfilzomib (CFZ) were purchased from Thermofisher ??? ask jim

\subsection{ProRS inhibitors}
The synthesis of NCP22 (T-3767758) is detailed in \cite{adachi2017discovery}.
NCP26 was synthesised by collaborators as a novel inhibitor of ProRS enzymes, synthesis detailed in \cite{tyediscovery2021} (currently under review).
ProRS inhibitors: halofuginone (MAZ1392; HF) and halofuginol (MAZ1805) were purchased from..
ProRS inhibitors NCP26 and NCP22 were kindly donated by Swiss lab... NAME
ProSA
WHERE OBTAINED??

%% TRIM24 structure
%\begin{figure}[htb]
%\centering
%\includegraphics[width=0.7\textwidth]{figures/Methods/trim24_structure.png}
%\caption[TRIM24i structure]{TRIM24 inhibitor chemical structure}
%\label{fig:trim24_structure}
%\end{figure}
%%

\section{Assays}
\subsection{Cell viability assays}
10X presto blue was added in a 1:10 ratio to cells in suspension and incubated at 37\C{} for two to three hours.
Plates were read [DETAILS OF MACHINE AND PROTOCOL, e.g. wavelength]

\subsection{Dose response curves}\label{subsec:method_doseresponse}
90\ul{} of cells in fresh media were seeded into 96-well plates a day prior to treatment with compound.
A total of 20,000 cells were seeded into each well.
No cells were placed in edge wells, to avoid edge effects.
The following day, media 0\% viability controls were placed in the first and last row.
Drug concentrations were made up 1000x the desired final concentration.
Drugs were diluted once in media (usually 1 in 100), then into the final plate with seeded cells (usually in 10), depending on the experiment.
All drug concentrations/combinations were performed in triplicate.
Cells were treated with DMSO in triplicate as 100\% viability controls.

\section{Bulk RNA-seq}\label{sec:bulk_lib_prep}
\subsection{RNA extraction}\label{subsec:rna_extraction}
RNA was isolated and purified using the Direct-Zol RNA MiniPrep kit (Zymo, USA), following the manufacturer's protocol.
In brief, for each sample, approximately 100,000 cells were lysed in 300\ul{} of TRIzol and the lysate was transferred to a microcentrifuge tube.
300\ul{} of ethanol was added to the lysed samples and vortexed.
The mixture was transferred to miniPrep columns and centrifuged at 13,000g for 30 seconds.
The column was washed twice with 400\ul{} of Direct-Zol pre-wash and once with 700\ul{} of RNA wash buffer
The column was transferred to an RNase-free tube and eluted with 50\ul{} of nuclease-free water and centrifuged.

The RNA concentration was quantified using a NanoDrop ND-1000 Spectrophotometer (Thermo Fisher Scientific, USA), and samples were stored at -80\C{}.

\subsection{RNA library preparation}
Samples were normalised to 100\si{\ng} and made up to 50\ul{} with nuclease-free water.
NEBNext\textsuperscript{\textregistered} Ultra II directional RNA library prep kit for Illumina\textsuperscript{\textregistered} with TruSeq indexes was used to prepare RNA libraries, following the manufacturer's protocol.
Samples were plated onto eppendorf DNA lobind 96 well plates.
A VIAFLO robot was used to reduce time spent on washes and precipitations.
Volumes were adjusted for elution steps to account for the dead volume of the robot.
Prior to any steps involving the VIAFLO robot, thermocycler, or incubations on the magnetic rack, plates were sealed tightly with foil lids, briefly vortexed and then centrifuged for < 10 seconds at 100g.
Poly-adenylated RNA was enriched using the NEBNext Poly(A) mRNA magnetic isolation module (NEB, USA), following the manufacturer's instructions with several modifications.
Oligo dT Beads were washed appropriately, then 10\ul{} of oligo dT Beads (manufacturer's protocol says 20\ul{}) was added to each RNA sample, and incubated for 10 minutes at room temperature.
Following polyA selection and first and second strand synthesis, cDNA was purified using 1.8X solid-phase reversible immobilisation (SPRI)
beads.
Purified cDNA was eluted in 55\ul{} 0.1X TE Buffer.
The cDNA was then end-prepped and adaptor ligated.
The cDNA was diluted 50-fold in the adaptor dilution buffer (as 100ng starting RNA was used).
PCR amplification was carried out using 11 PCR cycles (the manufacturer's protocol recommends 12-13 for 100ng starting RNA), in order to minimise PCR duplicates.

\subsection{Pre-sequencing preparation}\label{subsec:preseq}
The molarities of the libraries were determined by electrophoresis on an Agilent 22000 TapeStation (Agilent, USA) and High Sensitivity D1000 ScreenTape (Agilent, USA).
The samples were then pooled according to their peak molarity.
The pooled library was quantified using an Agilent 22000 TapeStation (Agilent, USA) and High Sensitivity D1000 ScreenTape (Agilent, USA).
If significant adaptor-dimers were present, then an extra clean-up using 1.2X SPRI beads was performed.
The pooled library was then denatured and diluted ready for sequencing.
In short, 10\ul{} of the approximately 2\si{\nano\Molar} library was denatured with 0.2N 10\ul{} NaOH.
The mixture was vortexed briefly, centrifuged at 300g for 1 minute, and then incubated at room temperature for 5 minutes.
10\ul{} 200\si{\milli\Molar} Tris-HCl (pH 7.0) was then added, vortexed and spun as above.
The denatured library was then diluted to 20\si{\pico\Molar}.
970\ul{} chilled HT1 buffer was added, vortexed and spun.
117\ul{} of the 20\si{\pico\Molar} library was then mixed with 1183\ul{} chilled HT1 buffer to give a 1.8\si{\pico\Molar} library ready for sequencing on the Illumina NextSeq platform.

Sequencing of the resultant libraries was performed on the NextSeq 500 (Illumina, USA) platform using a paired-end run, according to the manufacturer's instructions.

\section{Single-cell RNA-seq}
\subsection{Drop-Seq}

\subsubsection{Cell encapsulation}
The Drop-Seq protocol\cite{macosko2015highly} was followed for single-cell RNA-seq sample preparation.
Cells were loaded into a microfluidics cartridge.
Nadia, an automated microfluidics device (Dolomite Bio, UK), performed cell capture, cell lysis and reverse transcription.
Reverse transcription reactions were performed using ChemGene beads.

\subsubsection{Library preparation}
Beads were collected from the device and cDNA amplification was performed.
The beads were treated with Exo-I prior to PCR.
The amplified, purified cDNA then underwent tagmentation reactions.
A TapeStation (Agilent, USA) was used to assess library quality.
The samples were pooled together and split across multiple sequencing runs.

\subsection{10X Chromium V3}\label{subsec:10x_method}
Bone marrow samples were collected from two newly diagnosed multiple myeloma patients and two relapsed multiple myeloma patients;
anonymised human tissue samples used in this project were obtained with informed consent by the HaemBio Tissue Bank (REC reference: 17/SC/0572).
After Ficoll gradient separation, mononuclear bone marrow cells were diluted to 500,000 cells/\ml{} in RPMI media supplemented with 2\si{\milli\Molar}
L-glutamine and 10\% FBS and 1\ml{} was added to 15\ml{} polypropylene tubes.
Compounds were dissolved in DMSO, and 1\ul{} of compound solution was added to achieve a final concentration of 1\si{\micro\Molar} and incubated for 24 hours.
Cells were counted and single-cell RNA-seq library preparation was performed using the Chromium Next GEM Single Cell 3' GEM, Library and Gel Bead Kit v3.1 according to the manufacturer's instructions.
Indexed libraries were quantitated by TapeStation, pooled and sequenced on an Illumina NovaSeq 6000 (Novogene, UK).

\section{QuantM tRNA-seq}\label{sec:quantm}
Approximately 1 million cells were collected per sample and seeded overnight in six-well plates.
Cells were treated with 700\si{\nano\Molar} NCP26, 300\si{\nano\Molar} Halofuginone or 2\si{\micro\Molar} ProSA.
Controls were treated with equal volumes of DMSO.
Cells were collected at time = 0, time = 3 hours or time = 6 hours.
Samples were centrifuged at 300g for 5 minutes, the supernatant was discarded and the pellets were resuspended in 300\ul{} of Trizol.
RNA was extracted as above (section \ref{subsec:rna_extraction}) and quantified using a nanodrop.
Concentrations ranged from  189.7\si{\ng}/\ul{} to 398.85\si{\ng}/\ul{}.

QuantM tRNA-seq as outlined in \cite{pinkard2020quantitative} was used for library preparation with significant adaptations made to the protocol.
Each sample was normalised to 500\si{\ng} total RNA in 2.65\ul{} of nuclease-free water.
Samples were deacylated with deacylation buffer (Tris-HCl pH9.0; final concentration 20mM) and incubated at 37\C{} for 45 minutes.
Samples were not demethylated.

\subsection{Annealing and ligating adapters}
The samples were transferred to LoBind PCR plates, and 10\si{\pico\Molar} of 3` adaptor and 2.5\si{\pico\Molar} of each 5' adaptor (A, U, C and G) were added (1\ul{} of mix that is 10\si{\micro\Molar} for 3` and 2.5\si{\micro\Molar} for 5`).
The plate was incubated at 95\C{} for 2 minutes in a thermocycler.
1\ul{} of 5x annealing buffer (table \ref{tab:5x_annealing_buffer}) was added to each sample and the plate was incubated at 37\C{} for 15 minutes.
% Annealing buffer
\begin{table}[ht]
\centering
\begin{tabular}{|l|l|}
\hline
\textbf{Annealing buffer} & \textbf{Volume (\ul{})} \\ \hline
\rowcolor[HTML]{EFEFEF}
1\si{\Molar}  Tris-HCl (pH 8.0) & 250 \\ \hline
0.5\si{\Molar}  EDTA (pH 8.0) & 50 \\ \hline
\rowcolor[HTML]{EFEFEF}
1\si{\Molar}  MgCl\textsubscript{2} & 400 \\ \hline
Nuclease-free water & 9200 \\ \hline
\rowcolor[HTML]{EFEFEF}
\textbf{Total} & \textbf{10,000 (10\ml{})} \\ \hline
\end{tabular}
\caption{Annealing buffer recipe}
\label{tab:5x_annealing_buffer}
\end{table}
%%
%
Ligation of the adapters to the tRNA was performed.
0.5\ul{} T4 RNA ligase 2 (NEB), 1\ul{} 10x reaction buffer and 3.2\ul{} nuclease-free water was added to each 5.3\ul{} of sample, to total a final volume of 10\ul{} (0.5U/\ul{}).
The plate was placed in a thermocycler and incubated at 37\C{} for an hour, and then 4\C{} for an hour.
The ligated samples were then transferred to 1.5\ml{} eppendorfs.

\subsection{RNA precipitation}\label{subsec:rna_precip}
1.5\ul{} GlycoBlue was added to each tube.
Each sample was made up to 100\ul{} with nuclease-free water.
10\ul{} of 3\si{\Molar} sodium acetate (pH 5.2) and 250\ul{} 100\% ethanol was added to each tube and vortexed.
Samples were precipitated overnight at -80\C{}.
The following morning, tubes were centrifuged at >12,000g at 4\C{} for 30 minutes to form a pellet.
2 washes were performed with ice-cold, freshly prepared 75\% ethanol, spinning for 10 minutes at 12,000g.
All ethanol was removed with an extra 10 second top speed spin, and 10 minutes of air drying with the tube cap off.

\subsection{Hybridization of RT primer}
Samples were resuspended in 10\ul{} nuclease-free water and transferred to a PCR plate.
1\ul{} 10\si{\micro\Molar} RT primer <PRIMER TABLE of sequences ref AT BOTTOM>, 1\ul{} 10\si{\micro\Molar} dNTP mix, and 1\ul{} nuclease free water was added to each sample.
The PCR plate was placed in a thermocyler and incubated at 70\C{} for 2 minutes.

\subsection{cDNA synthesis}
cDNA was synthesised using SuperScript IV Reverse Transcriptase (Invitrogen), following the manufacturer's instructions.
4\ul{} 5x SuperScript IV buffer, 1\ul{} 100mM DTT, 1\ul{} SuperScript IV Reverse Transcriptase and 0.25\ul{} RNase Nxgen inhibitor (Lucigen) was added to each sample (totalling 19.25\ul{}).
The plate was heated in a thermocycler at 55\C{} for an hour.
19.25\ul{} 0.2N NaOH (final concentration 0.1N) was added to each sample, and heated in a thermocycler at 98\C{} for 20 minutes to hydrolyze RNA.
The samples were then transferred to 1.5\ml{} eppendorfs.
Ethanol precipitation was performed overnight (as in section \ref{subsec:rna_precip}), and nucleic acids were resuspended in 12\ul{}.

\subsection{Separating cDNA libraries}
Two 18-well 10\% Criterion TBE-Urea Polyacrylamide Gels (Bio-Rad) were used to separate cDNA libraries, following the manufacturer's instructions.
1X TBE (89\si{\milli\Molar} Tris, 89\si{\milli\Molar} boric acid, 2\si{\milli\Molar} EDTA) was used as running buffer.
5x sample buffer (89\si{\milli\Molar} Tris, 89\si{\milli\Molar} boric acid, 2\si{\milli\Molar}, 12\% Ficoll 400, 0.01\% bromophenol blue, 0.02\% xylene cyanole FF, 7\si{\Molar} urea) was made up, and 3\ul{} was added to each sample.
20\ul{} 1xTBE, 2\ul{} PCR marker (N3234; NEB) and 5.5\ul{} 5x sample buffer was mixed and used as a ladder for each gel.
cDNA libraries and ladders were pipetted into their corresponding well and the gels were ran for approximately an hour at 90V.
Gels were removed and placed back into their plastic tray for staining.
Gels were covered in excess 1xTBE and 3\ul{} SYBR gold (Invitrogen) was added to each tray and stained for 15 minutes on a mixing tray.
Gels were excised on 300 (????)\si{\nano\Molar} UV light.
A clean scalpel was used to cut out gel bands between 75nt and 300nt (the region representing tRNAs).
Gel fragments were then placed in labelled eppendorfs, trying to limit contamination.
The eppendorfs were placed in the fridge overnight.

A 25-gauge needle was used to pierce holes at the bottom of 500\ul{} eppendorfs.
The pierced 500\ul{} eppendorfs were nested inside 1.5\ml{} eppendorfs, and gel pieces were transferred into their corresponding, labelled nested tube.
The nested eppendorfs were centrifuged at 18,500g for 5 minutes, until the gel was completely sheared into the bottom tube.
The 500\ul{} eppendorfs were discarded.
400\ul{} of DNA extraction buffer (table \ref{tab:extraction_buffer}) was added to each 1.5\ml{} eppendorf and the tubes were frozen on dry ice for 30 minutes.
The tubes were placed on a rotator overnight at room temperature.
% extraction buffer
\begin{table}[ht]
\centering
\begin{tabular}{|l|l|}
\hline
\textbf{DNA extraction buffer} & \textbf{Volume (\ul{})} \\ \hline
\rowcolor[HTML]{EFEFEF}
4\si{\Molar}  NaCl & 1500 \\ \hline
1\si{\Molar}  Tris-HCl (pH 8.0) & 200 \\ \hline
\rowcolor[HTML]{EFEFEF}
0.5\si{\Molar}  EDTA  & 40 \\ \hline
Deionised water & 18260 \\ \hline
\rowcolor[HTML]{EFEFEF}
\textbf{Total} & \textbf{20000 (20\ml{})} \\ \hline
\end{tabular}
\caption{Extraction buffer recipe}
\label{tab:extraction_buffer}
\end{table}
%

UltraFree MC-VV centrifugal filters with a 0.1\si{\micro\Molar} pore were used to remove small gel pieces.
The filters were pre-wet with 7\ul{} DNA extraction buffer, and the gel/ extraction buffer slurry was transferred to them.
Tubes were spun at 20,000g for 3 minutes.
The filter columns were discarded.
1.5\ul{} GlycoBlue and 500\ul{} propan-2-ol was added to each tube and precipitated on dry ice for 30 minutes.
The samples were centrifuged at 18,500g at 4\C{} for 30 minutes, and washed once with 70\% ethanol for 5 minutes.
Ethanol was removed completely and air dried for 10 minutes.
Samples were resuspended in 12\ul{} nuclease-free water and transferred to a PCR plate.

\subsection{Circularization}
The cDNA libraries were circularized using CircLigase II (Lucigen).
1.5\ul{} 10X reaction buffer, 0.75\ul{} 50\si{\milli\Molar} MnCl\textsubscript{2} and 0.75\ul{} CircLigase II ssDNA Ligase was added to each sample.
The plate was placed in a thermocycler and incubated at 60\C{} for an hour, followed by 80\C{} for 20 minutes.
The libraries were transferred to eppendorfs and ethanol precipitation was performed (section \ref{subsec:rna_precip}).
Samples were resuspended in 15\ul{} nuclease-free water and transferred to a LoBind PCR plate.

\subsection{Amplification}
The circularized cDNA libraries were amplified using NEBnext Ultra II Q5 master mix and custom i5 and i7 primers.
5\ul{} 10\si{\micro\Molar} PCR primer, 5\ul{} 10\si{\micro\Molar} specific itRNA primer, and 25\ul{} Q5 master mix was added to each sample (a master mix was made up of PCR primer and Q5 master mix).
The PCR plate was placed in a thermocycler and amplified following the thermocycling conditions in table \ref{tab:my-pcr_tRNA}.

% PCR amplification
\begin{table}[]
\centering
\begin{tabular}{|l|l|l|l|}
\hline
\rowcolor[HTML]{9B9B9B}
\textbf{Stage}       & \textbf{Temperature} & \textbf{Time} & \textbf{Cycles}      \\ \hline
Initial Denaturation & 98\C{}                  & 30 seconds    & 1                    \\ \hline
Denaturation         & 98\C{}                    & 10 seconds    &                      \\ \cline{1-3}
Annealing/extension  & 65\C{}                    & 75 seconds    & \multirow{-2}{*}{12} \\ \hline
Final extension      & 65\C{}                    & 5 minutes     & 1                    \\ \hline
Hold                 & 4\C{}                     & Inf           & -                    \\ \hline
\end{tabular}
\caption[tRNA libraries PCR amplification thermocycling conditions]{tRNA libraries PCR amplification thermocycling conditions}
\label{tab:my-pcr_tRNA}
\end{table}

\subsection{Library purification}
The amplified libraries were purified using 2\% agarose gels stained with SYBR gold.
20\ul{} 10,000x SYBR gold was added to 200\ml{} 2\% agarose gel.
5\ul{} 10x bluejuice gel loading buffer (Invitrogen) was added to each sample.
4\ul{} PCR marker, 36\ul{} nuclease-free water and 4\ul{} loading buffer were combined to form the ladders of the gel.
The gel was ran at 120V for an hour.
Bands corresponding to cDNA libraries (100bp-250bp) were excised on a UV light box with a clean scalpel and transferred to 2\ml{} eppendorfs.
Gel extraction was performed using the GeneJET gel extraction kit, following the manufacturer's protocol.
In short, approximately 700\ul{} binding buffer was added to each tube and the gel mixtures were incubated at 55\C{} until dissolved.
The solubilized gel solutions were added to purification columns and centrifuged at 12,000g for 1 min and the through-flow discarded.
100\ul{} of binding buffer was added to each column and centrifuged again.
Two washes with 700\ul{} of wash buffer were performed, followed by an additional spin and air dry to remove any residual ethanol from the columns.
The collection tubes were discarded and the columns were nested in 1.5\ml{} eppendorfs.
50\ul{} of elution buffer was added to each column and centrifuged for 2 minutes.
The columns were discarded and the eppendorfs stored at -20\C{}.

The libraries were quantified on a tape station (Agilent) and pooled according to their peak molarity.
The pooled library was then denatured and diluted ready for sequencing (as above in Section \ref{subsec:preseq}).


\subsubsection{Sequencing}
Sequencing of the resultant libraries was performed on the NextSeq 500 (Illumina, USA) platform using a single-end run, according to the manufacturer's instructions.
% Base numbers etc. read 1 or read 2 length

%
%%%%%%%%%%%%%%%%%%%%%%%%%
% Computational methods
%%%%%%%%%%%%%%%%%%%%%%%%%

\section{Data Processing}\label{sec:data_processing}
\subsection{Bulk RNA-seq}\label{subsec:bulk_data_pro}
Fasta files were processed using a CGAT-flow\cite{sims2014cgat} pipeline, the workflow can be found at: \url{https://github.com/cgat-developers/cgat-flow/blob/master/cgatpipelines/tools/pipeline_rnaseqdiffexpression.py}.
The pseudo-alignment tool, Kallisto\cite{bray2016near}, was implemented to pseudo-align reads to the reference human genome sequence (GRCH38 (hg38) assembly) and to construct a counts matrix of samples against transcripts.
DESeq2\cite{love2014moderated} was used for differential expression analysis of  counts  matrices  (using  negative  binomial  generalized  linear  models) within the R statistical framework (v3.5.1).
XGR\cite{fang2016xgr}, Reactome\cite{fabregat2017reactome} and KEGG\cite{kanehisa2017kegg} were used to perform pathway analysis, within R\@.
Org.Hs.eg.db\cite{carlson2019org}, AnnotationDbi\cite{pages2020annotationdbi} and biomaRt\cite{durinck2009mapping} were used for converting between Ensembl IDs, HGNC symbols and ENTREZ IDs.

\subsection{Single-cell RNA-seq}
The computational pipeline outlined in Section \ref{sec:scRNA_pipeline} was used to process scRNA-seq data.
Downstream analysis was performed in Jupyter lab notebooks\cite{Kluyver2016jupyter} using R kernels.





