\chapter{\label{ch:3-methods}Methods}

%\minitoc

\section{Cell culture}
\subsection{AMO-1 cells}
AMO-1 cells, plasma cells derived from a 64-year old female myeloma patient, were used as a model cell-line for multiple myeloma. %\cite{cellosaurus2022}
Proteasome inhibitor-sensitive AMO-1 cells are referred to as WT cells.
Bortezomib resistant (aBTZ) and carfilzomib resistant cells (aCFZ), believed to be AMO-1 cells were generated and gifted by the Driessen lab\cite{soriano2016proteasome}.
After typing these cells, they were found to be a mix of AMO-1 cells and L363 cells.
AMO-1 cells were cultivated in RPMI-1640 medium (Thermofisher, UK), supplemented with 10\% fetal bovine serum (FBS) and 2\si{\milli\Molar} L-glutamine (Invitrogen, UK).
Cells were passaged when they reached approximately 1.5-2 million cells per \ml{}.
AMO-1 cells are suspension cells and were split twice a week to approximately 0.5 million cells per \ml{}.
All media was replaced with fresh media every two to three weeks, or prior to performing experiments with the cells.

\subsection{L363 cells}
After typing the cells gifted by the Driessen lab, they were found to be a mix of AMO-1 MM cells and L363 MM cells.
In-house PI-resistant cell lines were produced by Dr James Dunford by continued and escalating drug exposure of drug-sensitive (WT) AMO-1 cells.
However after these cells were typed, they were found to be L363 cells.
This was due to the drug exposure selecting the L363 contaminate population over the AMO-1 cells, due to their natural increased resistance to PI, compared to AMO-1 cells.
Once this mistake made by our collaborators was noticed, WT L363 cells were purchased.
WT, aCFZ and aBTZ cells were cultivated in RPMI-1640 medium (Thermofisher, UK), supplemented with 10\% fetal bovine serum (FBS) and 2\si{\milli\Molar} L-glutamine (Invitrogen, UK), and kept in 100\si{\nano\Molar} of their respective proteasome inhibitor.
Cells were passaged when they reached approximately 1.5-2 million cells per \ml{}.
L363 cells are suspension cells and were split twice a week to approximately 0.5 million cells per \ml{}.
All media was replaced with fresh media every two to three weeks, or prior to performing experiments with the cells.

L363 cells resistant to NCP26 (NCP26R) were also developed by Dr James Dunford by continued and escalating drug exposure of WT L363 cells to NCP26.

\section{Compounds}

\subsection{Proteasome inhibitors}
Stock concentrations of proteasome inhibitors, bortezomib (BTZ) and carfilzomib (CFZ) were purchased from Thermofisher, UK.
BTZ and CFZ were made up to desired concentrations with DMSO\@.

\subsection{ProRS inhibitors}
The synthesis of NCP22 (T-3767758) is detailed in \cite{adachi2017discovery}.
NCP26 was synthesised by collaborator,  Dr Ralph Mazitschek, co-discoverer of HFG mechanism, as a novel inhibitor of ProRS enzymes, synthesis detailed in \cite{tyediscovery2021} (currently under review).
ProRS inhibitors: halofuginone (MAZ1392; HF) and halofuginol (MAZ1805), NCP26 and NCP22 were provided by Dr Mazitschek.
The chemical structures of these compounds are shown in Figure \ref{fig:chem_structures}.

% Chemical structures
\begin{figure}[ht]
%1
\centering
\begin{subfigure}{0.49\textwidth}
    \includegraphics[width=\textwidth]{figures/Methods/HF_structure.png}
    \caption{Halofuginone}
\end{subfigure}
%2
\begin{subfigure}{0.49\textwidth}
    \includegraphics[width=\textwidth]{figures/Methods/halofuginol_structure.png}
    \caption{Halofuginol}
\end{subfigure}
%
\medskip
%3
\begin{subfigure}{0.45\textwidth}
    \includegraphics[width=0.8\textwidth]{figures/Methods/NCP26_structure.png}
    \caption{NCP26}
\end{subfigure}
%4
\begin{subfigure}{0.45\textwidth}
    \includegraphics[width=0.8\textwidth]{figures/Methods/NCP22_structure.png}
    \caption{NCP22}
\end{subfigure}
\caption[ProRS inhibitor chemical structures]{ProRS inhibitor chemical structures.
}
\label{fig:chem_structures}
\end{figure}

%% TRIM24 structure
%\begin{figure}[htb]
%\centering
%\includegraphics[width=0.7\textwidth]{figures/Methods/trim24_structure.png}
%\caption[TRIM24i structure]{TRIM24 inhibitor chemical structure}
%\label{fig:trim24_structure}
%\end{figure}
%%

\section{Assays}
\subsection{Cell viability assays}
10X presto blue was added directly to the cell media in a 1:10 ratio and incubated at 37\C{} for two to three hours.
The fluorescence was measured at 560 nm excitation and 590 nm emission on FLUOstar OMEGA (BMG Labtech, Offenburg, Germany).

\subsection{Dose-response curves}\label{subsec:method_doseresponse}
90\ul{} of cells in fresh media were seeded in single wells into 96-well plates a day prior to treatment with compound.
A total of 20,000 cells were seeded into each well.
No cells were placed in edge wells, to avoid edge effects.
The following day, media 0\% viability controls were placed in the first and last row.
Drug concentrations were made up 1000x the desired final concentration.
Drugs were diluted once in media (usually 1 in 100), then into the final plate with seeded cells (usually in 10), depending on the experiment.
All drug concentrations/combinations were performed in triplicate.
Cells were treated with 0.1\% DMSO final concentration in triplicate as 100\% viability controls.
Throughout experiments, DMSO concentration was kept the same for every sample.

\section{Bulk RNA-seq}\label{sec:bulk_lib_prep}
\subsection{RNA extraction}\label{subsec:rna_extraction}
RNA was isolated and purified using the Direct-Zol RNA MiniPrep kit (Zymo, USA), following the manufacturer's protocol.
In brief, for each sample, approximately 100,000 cells were lysed in 300\ul{} of TRIzol and the lysate was transferred to a microcentrifuge tube.
300\ul{} of ethanol was added to the lysed samples and vortexed.
The mixture was transferred to miniPrep columns and centrifuged at 13,000g for 30 seconds.
The column was washed twice with 400\ul{} of Direct-Zol pre-wash and once with 700\ul{} of RNA wash buffer
The column was transferred to an RNase-free tube and eluted with 50\ul{} of nuclease-free water and centrifuged.

The RNA concentration was quantified using a NanoDrop ND-1000 Spectrophotometer (Thermo Fisher Scientific, USA), and samples were stored at -80\C{}.

\subsection{RNA library preparation}
Samples were normalised to 100\si{\ng} and made up to 50\ul{} with nuclease-free water.
NEBNext\textsuperscript{\textregistered} Ultra II directional RNA library prep kit for Illumina\textsuperscript{\textregistered} with TruSeq indexes was used to prepare RNA libraries, following the manufacturer's protocol.
Samples were plated onto Eppendorf DNA lobind 96 well plates.
A VIAFLO robot was used to reduce time spent on washes and precipitations.
Volumes were adjusted for elution steps to account for the dead volume of the robot.
Prior to any steps involving the VIAFLO robot, thermocycler, or incubations on the magnetic rack, plates were sealed tightly with foil lids, briefly vortexed and then centrifuged for < 10 seconds at 100g.
Poly-adenylated RNA was enriched using the NEBNext Poly(A) mRNA magnetic isolation module (NEB, USA), following the manufacturer's instructions with several modifications.
Oligo dT Beads were washed appropriately, then 10\ul{} of oligo dT Beads (manufacturer's protocol says 20\ul{}) was added to each RNA sample, and incubated for 10 minutes at room temperature.
Following polyA selection and first and second strand synthesis, cDNA was purified using 1.8X solid-phase reversible immobilisation (SPRI)
beads.
Purified cDNA was eluted in 55\ul{} 0.1X TE Buffer.
The cDNA was then end-prepared and adaptor ligated.
The cDNA was diluted 50-fold in the adaptor dilution buffer (as 100ng starting RNA was used).
PCR amplification was carried out using 11 PCR cycles (the manufacturer's protocol recommends 12-13 for 100ng starting RNA), in order to minimise PCR duplicates.

\subsection{Pre-sequencing preparation}\label{subsec:preseq}
The molarities of the libraries were determined by electrophoresis on an Agilent 22000 TapeStation (Agilent, USA) and High Sensitivity D1000 ScreenTape (Agilent, USA).
The samples were then pooled according to their peak molarity.
The pooled library was again quantified using an Agilent 22000 TapeStation (Agilent, USA) and High Sensitivity D1000 ScreenTape (Agilent, USA).
If significant adaptor-dimers were present, then an extra clean-up using 1.2X SPRI beads was performed.
The pooled library was then denatured and diluted ready for sequencing.
In short, 10\ul{} of the approximately 2\si{\nano\Molar} library was denatured with 0.2N 10\ul{} NaOH.
The mixture was vortexed briefly, centrifuged at 300g for 1 minute, and then incubated at room temperature for 5 minutes.
10\ul{} 200\si{\milli\Molar} Tris-HCl (pH 7.0) was then added, vortexed and spun as above.
The denatured library was then diluted to 20\si{\pico\Molar}.
970\ul{} chilled HT1 buffer was added, vortexed and spun.
117\ul{} of the 20\si{\pico\Molar} library was then mixed with 1183\ul{} chilled HT1 buffer to give a 1.8\si{\pico\Molar} library ready for sequencing on the Illumina NextSeq platform.

Sequencing of the resultant libraries was performed on the NextSeq 500 (Illumina, USA) platform using a paired-end run, according to the manufacturer's instructions.

\section{Single-cell RNA-seq}
\subsection{Drop-Seq}

\subsubsection{Cell encapsulation}
The Drop-Seq protocol\cite{macosko2015highly} was followed for single-cell RNA-seq sample preparation.
Cells were loaded into a microfluidics cartridge.
Cell capture, cell lysis and reverse transcription was performed on a Nadia instrument, an automated microfluidics device (Dolomite Bio, UK)
Reverse transcription reactions were performed using ChemGene beads.

\subsubsection{Library preparation}
Beads were collected from the device and cDNA amplification was performed.
The beads were treated with Exo-I nuclease prior to PCR.
The amplified, purified cDNA then underwent tagmentation reactions.
A TapeStation (Agilent, USA) was used to assess library quality.
The samples were pooled together and split across multiple sequencing runs.
Sequencing of the resultant libraries was performed on the NextSeq 500 (Illumina, USA) platform using a paired-end run, according to the manufacturer's instructions.

\subsection{10X Chromium V3}\label{subsec:10x_method}
Bone marrow samples were collected from two newly diagnosed multiple myeloma patients and two relapsed multiple myeloma patients;
anonymised human tissue samples used in this project were obtained with informed consent by the HaemBio Tissue Bank (REC reference: 17/SC/0572).
After Ficoll gradient separation, mononuclear bone marrow cells were diluted to 500,000 cells/\ml{} in RPMI media supplemented with 2\si{\milli\Molar}
L-glutamine and 10\% FBS and 1\ml{} was added to 15\ml{} polypropylene tubes.
Compounds were dissolved in DMSO, and 1\ul{} of compound solution was added to achieve a final concentration of 1\si{\micro\Molar} and incubated for 24 hours.
Cells were counted and single-cell RNA-seq library preparation was performed using the Chromium Next GEM Single Cell 3' GEM, Library and Gel Bead Kit v3.1 according to the manufacturer's instructions.
Indexed libraries were quantitated by TapeStation, pooled and sequenced on an Illumina NovaSeq 6000 (Novogene, UK) platform using a paired-end run.

%
%%%%%%%%%%%%%%%%%%%%%%%%%
% Computational methods
%%%%%%%%%%%%%%%%%%%%%%%%%

\section{Data Processing}\label{sec:data_processing}
\subsection{Bulk RNA-seq}\label{subsec:bulk_data_pro}
Fasta files were processed using a CGAT-flow\cite{sims2014cgat} pipeline, the workflow can be found at: \url{https://github.com/cgat-developers/cgat-flow/blob/master/cgatpipelines/tools/pipeline_rnaseqdiffexpression.py}.
The pseudo-alignment tool, Kallisto\cite{bray2016near}, was implemented to pseudo-align reads to the reference human genome sequence (GRCH38 (hg38) assembly) and to construct a counts matrix of samples against transcripts.
DESeq2\cite{love2014moderated} was used for differential expression analysis of  counts  matrices  (using  negative  binomial  generalized  linear  models) within the R statistical framework (v3.5.1).
XGR\cite{fang2016xgr}, ClusterProfiler\cite{yu2012clusterprofiler,wu2021clusterprofiler}, Reactome\cite{fabregat2017reactome}, GSEA\cite{shi2007gene}, Gene Ontology (GO)\cite{gene2019gene} and KEGG\cite{kanehisa2017kegg} were used to perform pathway analysis, within R\@.
Org.Hs.eg.db\cite{carlson2019org}, AnnotationDbi\cite{pages2020annotationdbi} and biomaRt\cite{durinck2009mapping} were used for converting between Ensembl IDs, HGNC symbols and ENTREZ IDs.

\subsection{Single-cell RNA-seq}
The computational pipeline outlined in Section \ref{sec:scRNA_pipeline} was used to process scRNA-seq data.
Downstream analysis was performed in Jupyter lab notebooks\cite{Kluyver2016jupyter} using R kernels.
inferCNV\cite{patel2014single, infercnv2014} was used to detect large-scale chromosomal copy number variations (CNVs) in scRNA-seq data and help inform cell type annotation of the clusters.