\section{Myeloma bone marrow classifier}\label{sec:MM_classifier}
\subsection{Introduction}

Many single-cell MM studies separate MM cells from immune cells prior to sequencing, using biological markers.
For example Zavidiji et al. (2020) exclusively studied the immune microenvironment of MM by isolating CD138\textsuperscript{-} and CD45\textsuperscript{+} BM cell fractions, to exclude MM cells\cite{zavidij2020single}.
Many other studies sequence only MM cells, for example Cohen et al. (2021) who sorted cells prior to sequencing, isolating MM cells by magnetic CD138\textsuperscript{+} bead enrichment and staining CD138\textsuperscript{+}/CD38\textsuperscript{+} cells with antibodies\cite{cohen2021identification}.
Although CD138 expression is considered a hallmark of MM cells, markedly decreased CD138 expression has been observed in relapsed MM, and this has also been correlated to worse overall survival outcomes\cite{kawano2012multiple}.
Therefore, by separating cells based on the expression of CD138 and other markers, clonal populations of MM may be excluded (also perhaps the MM cells contributing the most to disease progression.)
Additionally, as discussed in Chapter \ref{ch:6-sc}, MM cells interact with their immune microenvironment and are supported as they grow by immune cells.
Therefore, by sorting MM patient cells prior to sequencing, substantial information may be lost.

In Chapter \ref{ch:6-sc}, two scRNA-seq experiments were performed.
For these experiments, no cell sorting took place and the whole BM niche was sequenced.
Therefore, cell types had to be determined computationally.
For annotation of these datasets, references based on healthy tissue were used to inform R annotation packages: clustifyr, scClassify and singleR.
As the references originated from healthy tissue, they were unable to label the pathological myeloma cells, and MM cells had to be identified manually using expression of known biological markers.
This took considerable time and required significant biological knowledge.
An MM classifier or model that could automate cell-type annotation of MM patient BM scRNA-seq samples would save time for researchers and could encourage studies where the whole BM niche is sequenced, ensuring clonal MM populations lacking CD138 expression are not missed by traditional cell sorting techniques.
This could also help remove some of the ambiguity of defining cell clusters.

\subsection{Classifier building}\label{subsec:MM_classifier_model_building}
Using the two scRNA-seq datasets from Chapter \ref{ch:6-sc}, cell classifiers were constructed for MM patient BM samples.
scClassify's function \texttt{train\char`_scClassify} was used to train reference models for scClassify annotation.
Log-normalised expression data and a vector corresponding to each cell's manually defined cell-type annotation was used as input for model training.
Three models were generated.
The first model was trained using the newly-diagnosed MM patient dataset, the second using the relapsed MM dataset, and the third was trained using  both datasets.
These models will be referred to as `naive', `relapsed' and `joint'.
The manual annotations used to train the models were much broader than the detailed annotations given in Chapter \ref{ch:6-sc}.
This is to minimise the number of cells being labelled as `intermediate' where significant overlap exists between cell subtypes
%
% UMAP- build classifiers.
\begin{figure}[htb]
\centering
\includegraphics[width=0.7\textwidth]{figures/workflow_generation/classifier_building.pdf}
\caption[Classifier annotation building]{Cell type annotation for classifier building.
Cell type classifications were made broader than the detailed annotation in given in Chapter \ref{ch:6-sc}.
a) Newly-diagnosed (naive) MM dataset.
b) Relapsed MM dataset.}
\label{fig:classifier_building}
\end{figure}
%
The broad training annotations can be seen in Figure \ref{fig:classifier_building}.

\subsection{Classifier testing}
% GSM5687372 GSE188632
In order to to test the performance of the MM classifiers, publically available MM scRNA-seq data was downloaded from GEO\@.
The test data comprises one PBMC sample from a relapsed and refractory MM (RRMM) patient, which contains both MM and immune cells (GEO accession number \textit{GSE188632}).
The deposited counts matrix was processed using the clustering and annotation modules of the scRNA-seq analysis workflow outlined in Section \ref{subsec:updated_scrna}.

% UMAP clutsifyr and scClassify HCA PBMC
\begin{figure}[htb]
\centering
\includegraphics[width=\textwidth]{figures/workflow_generation/MM_classifier_umap_HCA_joint.pdf}
\caption[Public scRNA-seq data clustering and annotation]{Public scRNA-seq data (GEO accession number \textit{GSE188632}) clustering and annotation using references: from healthy tissue.
a) UMAP clustering.
b) Clustifyr annotation performed with the HumanCellAtlas (HCA) reference.
c) scClassify annotation performed with the joint PBMC model as training data.
}
\label{fig:mm_class_umap_annotate}
\end{figure}
%
16 distinct cell clusters were identified using Seurat (Figure \ref{fig:mm_class_umap_annotate}a).
Firstly, the data was annotated using scClassify and clustifyr with healthy tissue references.
A PBMC model reference was used with scClassify and a HumanCellAtlas (HCA) reference with clustifyr (Figures \ref{fig:mm_class_umap_annotate}b and \ref{fig:mm_class_umap_annotate}c).
Both packages labelled the majority of cell clusters as monocytes or other myeloid cell types.
Clustifyr-HCA assigned clusters 4, 5, 11 and 13 as NK cells, whilst scClassify-PBMC assigned clusters 5 and 13 as T cells and cluster 11 as NK cells.
Clusters 7 and 10 were left unassigned using clustifyr-HCA annotation, whilst clusters 4, 7, 9 and 10 were labelled as B cells by scClassify-PBMC.
From the combination of the two annotations it seems like clusters 5 and 13 are T cells, cluster 11 is NK cells, and clusters 0, 1, 2, 3, 6, 8, 12 and 14 are myeloid cells.
However, from these annotations alone, it is unclear which clusters are the MM cell population.

Next, the scRNA-seq data was annotated using the MM dataset-trained model classifiers generated above (Figure \ref{fig:mm_classifier_scclassify_annotate}).
%
% scclassify annotation with MM classifier
\begin{figure}[htb]
\centering
\includegraphics[width=\textwidth]{figures/workflow_generation/MM_classifier_annotation_scclassify.pdf}
\caption[Public scRNA-seq data MM classifier annotation]{scClassify annotation performed on public scRNA-seq data (GEO accession number \textit{GSE188632}) using MM classifier models generated above.
a) Newly-diagnosed (naive) MM patient dataset.
b) Relapsed MM patient dataset.
c) Joint newly-diagnosed/relapsed MM patient dataset.
}
\label{fig:mm_classifier_scclassify_annotate}
\end{figure}
%
%
Using each model, scClassify identified a number of cells as MM cells (coloured pink in Figure \ref{fig:mm_classifier_scclassify_annotate}).
The naive model annotation identified the majority of cells in clusters 7, 9 and 10 as MM cells.
The relapsed model annotation identified cells in clusters 4, 7, 9 and 10 as MM cells.
The joint model annotation mainly identified clusters 7, 9, and 10 as MM cells, and some cells in cluster 4 as MM cells.
Additionally, all three models seem to agree with each other and scClassify-PBMC's assignment of non-MM cells, with clusters 5 and 13 being assigned as T cells, cluster 11 as NK cells and the rest of the non-MM cells clusters as myeloid cells.

To assess if the models' MM cell classifications were correct, MM marker expression was examined (Figure \ref{fig:mm_class_ftp_cd138_cd45_bcma}).
%\textit{CD19} and \textit{CD20} were not expressed in the dataset.
% cd138, cd45 and bcma
\begin{figure}[htb]
\centering
\includegraphics[width=0.7\textwidth]{figures/workflow_generation/MM_classifier_feature_cd138_bcma_cd45.pdf}
\caption[Public scRNA-seq data bioligical MM markers featureplots]{Public scRNA-seq data (GEO accession number \textit{GSE188632}) biological MM markers featureplots.
a) \textit{CD138} expression- cluster 4, 7, 9 and 10 mainly expressing \textit{CD138}.
b) \textit{CD45} expression- clusters 7, 9 and 10 not expressing \textit{CD45}.
c) \textit{BCMA} expression- clusters 4, 7, and 9 mainly expressing \textit{BCMA}.
d) \textit{IGKC} expression- clusters 4, 7, 9 and 10 have the highest expression of \textit{IGKC}.
Expression of \textit{CD19} and \textit{CD20} was not detected in this dataset.
There was minimal expression of \textit{SLAMF7} and \textit{IGLC2}.
}
\label{fig:mm_class_ftp_cd138_cd45_bcma}
\end{figure}
%
%%%% <PAST OR PRESENT TENSE>
As discussed in Section \ref{subsec:sc_annotate}, MM cells tend to express or over-express \textit{CD138}, \textit{BCMA}, \textit{IGKC}/\textit{IGLC2}, \textit{SLAMF7}, and not express or under-express \textit{CD20}, \textit{CD19} and \textit{CD45}.
\textit{CD20} and \textit{CD19} were not expressed in this dataset, and \textit{SLAMF7} and \textit{IGLC2} were not expressed very highly in this dataset.
\textit{CD138} and \textit{BCMA} were shown to be expressed by clusters 4, 7, 9 and 10.
\textit{IGKC} was highly expressed by clusters 4, 7, 9 and 10.
Clusters 7, 9 and 10 did not express \textit{CD45}, whilst cluster 4 expressed \textit{CD45}, but to a lesser degree than other clusters.
Therefore, it can be concluded that clusters 4, 7, 9 and 10 are likely to be the MM cell population.
Clusters 7, 9 and 10 are mainly \textit{CD45}\textsuperscript{-} MM cells, and cluster 4 is mainly comprised of \textit{CD45}\textsuperscript{+} MM cells.
Therefore, all three MM model annotations were correct in assigning clusters 7, 9 and 10 as MM cells.
Therefore there is high confidence, based on the MM gene marker expression knowledge and literature, that the automated annotation using the three MM models correctly assigned clusters 7, 9 and 10 as MM cells.
Based on gene expression knowledge, it also seems highly likely that cluster 4 is also comprised of MM cells.
However the naive and joint models were unable to label many of the cells in this cluster as MM cells, instead labelling them as intermediate cell-types or as myeloid cells.

Next, performance was evaluated quantitatively using ground-truth data.
Knowledge on MM gene expression was used to inform the ground truth data, where cells in clusters 4, 7, 9 and 10 were labelled as MM cells, and other clusters as non-MM cells.
To quantify each model's accuracy of classifying MM cells, precision, recall and F1 scores were calculated for each of the three models' assignment of cells (as MM vs non-MM) compared to the ground truth results (Figure \ref{fig:mm_class_accuracy_bar}).
%
\begin{figure}[htb]
\centering
\includegraphics[width=0.5\textwidth]{figures/workflow_generation/precision_recall_f1_barchart.png}
\caption[MM classifier accuracy]{MM cell classifier performance.
Precision, recall and F1 scores for the three models ability to accurately label MM cells; where true positive indicates a cell from the MM clusters (4, 7, 9 and 10) being labelled as `MM\char`_cell' by the model, false positive- a cell from any non-MM cluster being assigned the `MM\char`_cell' label, false negative- a cell from one of the MM clusters being assigned any label other than `MM\char`_cell'.
}
\label{fig:mm_class_accuracy_bar}
\end{figure}
%
All three models had very high precision values: 0.998, 0.965, 0.987, for the naive, relapsed and joint models, respectively.
This reflects a very small false positive rate- the models did not label many non-MM cells as MM cells.
The recall score of the models was not as high, indicating there were more false negatives- the models assigned numerous cells in the MM clusters (4, 7, 9 and 10) as non-MM cells types.
The model with the highest F1 score (0.778) was the model generated using the relapsed MM dataset only.
This is a good score, indicating that the relapsed model was fairly accurate at classifying MM cells in the test dataset, and demonstrates the effectiveness of using the MM dataset trained models to annotate further MM patient BM scRNA-seq samples.

The GEO dataset originated from a RRMM patient, therefore it is logical that the relapsed model was the most accurate at labelling MM cells.
MM patients' transcriptomes change considerably throughout the course of disease progression and treatment cycles, hence why the naive dataset (although being extremely cell rich- with over 60,000 cells) had the lowest recall score and was not able to classify MM cells as well as the other models.
This should be considered when using the MM classifiers in future, applying the correct classifier to a dataset, such that the naive model is applied to newly-diagnosed MM patient datasets and the relapsed model to RRMM patient datasets.
In addition, as shown in the joint model, scClassify models can be trained using multiple patients or scRNA-seq experiments.
As more MM scRNA-seq experiments (including the full BM niche) are performed and the data made publically available, more comprehensive models can be generated, with further MM scRNA-seq annotations added to the training set.
More patients samples would make the classifiers more accurately represent the heterogeneity seen in MM, and therefore more accurately annotate future MM datasets.


% significant T cell and B cell depletion in RRMM.