\chapter{\label{ch:3-methods}Methods}

%\minitoc

\section{Cell culture}
\subsection{AMO-1 cells}
AMO-1 cells, plasma cells from a 64-year old female myeloma patient, were used as a model cell-line for multiple myeloma. Bortezomib and carfilzomib resistant AMO-1 cells were generated and kindly gifted by the Driessen lab (1). Bortezomib, carfilzomib, pomolidimide and bortezomib plus pomolidimide resistant AMO-1 cells were also generated by Dr James Dunford by continual and escalating drug exposure. (WHERE did we get AMO-1 WT cells from, kindly gifted by MARIA???)
AMO-1 cells were cultivated in RPMI-1640 medium (WHERE we get it from), supplemented with 10\% fetal bovine serum (FBS), 100$\mu$g/ml streptomycin and 100 U/ml penicillin (P/S) and 2mM L-glutamine (Invitrogen, UK). Cells were passaged at approximately 1.5-2 million cells per ml (IS THIS THE RIGHT MEASURE??). Media was replaced twice a week.


\section{Bulk RNA-seq}
\subsection{RNA extraction}
RNA was extracted from AMO-1 cells using the Direct-Zol RNA MiniPrep kit (Zymo), following the manufacturer's protocol.  The RNA was quantified using a NanoDrop ND-1000 Spectrophotometer, and normalised to 100ng of RNA per sample.

% Give a thorough explanantion too

\subsection{RNA library preparation}
NEBNext\textsuperscript{\textregistered} UltraTM RNA library prep kit for Illumina\textsuperscript{\textregistered} with TruSeq indexes was used to prepare RNA libraries, following the manufacturer's protocol. In short...
The molarities of the libraries were determined by electrophoresis on a TapeStation (Agilent).

\section{Single-cell RNA-seq}
\subsection{Cell encapsulation}
Nadia instrument etc...

\subsection{Library preparation}
kit ...

\section{ATAC maybe}
\subsection{ATAC stuff}

\section{ChIP maybe}
\subsection{ChIP stuff}

\section{Sequencing}
Sequencing of the resultant libraries was performed on the NextSeq 500 (Illumina, USA) platform using a paired-end run, according to the manufacturer's instructions.

%
\section{Phosphoproteomics}
%
\subsection{Cell lysis}
Approximately 20 million cells for each condition in triplicate were pelleted and stored in 500$\mu$l of PBS at -80$^{\circ}$C. 300$\mu$l of fresh lysis buffer (10ml RIPA buffer, 3$\mu$l benzonase, 1 tablet phos stop) was added to each pellet, vortexed and left for 10 minutes on ice and then sonicated. The supernatant was transferred to a fresh (WHAT TYPE) tube.

\subsection{Protein quantification}
Protein concentrations were determined by BCA protein assay (Thermofisher, UK). 400$\mu$g of protein was taken from each sample. Samples were made up to a volume of 200$\mu$l with MilliQ-H\textsubscript{2}O.

\subsection{Protein Digestion}
Kessler lab protocols were followed (\url{https://www.tdi.ox.ac.uk/research/research/tdi-mass-spectrometry-laboratory/mass-spectrometry/protocols-and-tools}).
The lysed samples were reduced with 5\%$\mu$l of 200 mM DTT in 0.1 M Tris buffer and incubated for 40 minutes at room temperature. The reduced samples were alkylated with 20$\mu$l of 200mM iodoacetamide in 0.1M Tris buffer, vortexed and then incubated for 45 minutes in the dark at room temperature.
The protein was precipitated using methanol/chloroform extraction. The alkylated samples were transferred to 2ml eppendorfs. 600$\mu$l of methanol was added to each sample, followed by 150$\mu$l of chloroform and then vortexed gently. 450$\mu$l of MilliQ-H\textsubscript{2}O was then added and vortexed gently. The samples were centrifuged at maximum speed on a table top centrifuge for one minute. The upper aqueous phase was removed, without disturbing the precipitate at the interface. 450$\mu$l of methanol was added to each sample, without disturbing the disc and centrifuged for two minutes. Protein pellets were resuspended, one sample at a time: the supernatant was removed and 100$\mu$l of 6M urea in 0.1M Tris buffer was added. The samples were vortexed and then sonicated (???). Samples were diluted with 500$\mu$l MilliQ-H\textsubscript{2}O, to ensure the final urea concentration was below 1M. Porcine trypsin (Sequencing Grade Modified Trypsin; Promega) was added in a 1:50 ratio of enzyme:total protein content of sample, such that 40$\mu$l of trypsin solution containing 8$\mu$g trypsin in 0.1M Tris buffer was added to each sample. Samples were left to digest overnight at 37$^{\circ}$C in an incubator shaker.

\subsection{Peptide purification}
The following day, the reaction was stopped, acidifying samples to 1\% Trifluoroacetic acid (TFA). Samples were desalted and concentrated using 1ml C-18 Sep-Pak (Waters) cartridges. Two reagents were used: solution A (98\% MilliQ-H\textsubscript{2}O, 2\% Acetonitrile (CH\textsubscript{3}CN) and 0.1\% TFA) for washing and solution B (65\% Acetonitrile, 35\% MilliQ-H\textsubscript{2}O and 0.1\% TFA) for activation and elution. The columns were flushed with 1ml of solution B and then washed with 1ml of solution A. The digested samples were added to the columns and vacuumed through slowly. Two 1ml washes with solution A were performed. Fresh, labelled eppendorfs were placed beneath the columns and peptides were eluted with 500$\mu$l of solution B.
For phosphopeptide-enrichment, 90\% of the peptides were removed for Immobilized Metal Affinity Chromatography (IMAC) on a Bravo Automated Liquid Handling Platform (Agilent). 10\% of the peptides were used for total proteome analysis. Eluted peptides were dried using a vacuum concentrator (Speedvac, Eppendorf) and stored at -20$^{\circ}$C until analysis by mass spectrometry (MS). Prior to MS analysis, dried peptides were resuspended in solution A.


\section{Ubiquitinomics}
\subsection{Cell lysis}
Approximately 100 million cells for each condition in triplicate were pelleted and stored in 500$\mu$l of PBS at -80$^{\circ}$C. PMTScan Ubiquitin Remnant Motif Kit (K-$\varepsilon$-GG; Cell signalling) was used, following the manufacturer's protocol (REF).
Pellets were solubilized and denatured in 4ml urea lysis buffer (20mM HEPES, pH 8.0, 9M urea, 1mM sodium orthovanadate, 2.5mM sodium pyrophosphate, 1mM $\beta$-glycerophosphate). The lysates were sonicated on ice, with two bursts of 15 seconds with a one minute break in-between.

\subsection{Protein quantification}
Protein concentrations were determined by BCA protein assay (Thermofisher, UK). All samples were found to contain between 10mg and 20mg of protein, so all of the available protein was used, with no normalisation.

\subsection{Protein Digestion}
Lysates were reduced using dithiothreitol (DTT) at a final concentration of 4.5 mM for 30 minutes at room temperature. The reduced samples were alkylated using iodoacetamide (100mM final) for 15 minutes in the dark at room temperature. The alkylated samples were diluted four-fold with 20mM HEPES (pH 8.0) and digested with 400$\mu$l trypsin solution, containing 1mg/ml trypsin-TPCK (Worthington, LS003744) in 1mM HCl. Samples were left to digest overnight at room temperature on a rotator.

\subsection{Peptide purification}
The following day, the reaction was stopped, acidifying samples to 1\% Trifluoroacetic acid (TFA). Samples were desalted and concentrated using 10ml C-18 Sep-Pak (Waters) cartridges. The columns were activated using 5ml of solution B, washed with 10ml of solution A. The samples were added to the columns and ran through slowly. The peptides were washed with 10ml of solution A. The cartridges were then removed from the vacuum and the peptides were eluted into fresh falcon tubes with 6ml of solution B, using the plunger of the syringes. 20$\mu$g of digested protein was removed from each sample for matching total proteome analysis. The eluate was kept at -80$^{\circ}$C overnight. The frozen peptide solutions were lyophilized for two days and then stored at -80$^{\circ}$C.
%

\subsection{Immunoaffinity purification}
10X immunoaffinity purification (IAP) buffer provided with PTMScan Kit was diluted to 1X concentration with MilliQ-H\textsubscript{2}O. Resuspended purified peptides pellets in 1.4ml of IAP buffer by pipetting up and down and transferred to 1.7ml eppendorfs. The samples were centrifuged at 4$^{\circ}$C for 5 minutes at 10000xg and kept on ice whilst preparing antibody beads.
The anti-body bead slurry was centrifuged (30 seconds at 2000xg) and 1ml of PBS was added and then centrifuged. The supernatant was removed and the antibody beads were washed a further four times with PBS and resuspended in 40$\mu$l of PBS. The peptide solution was transferred to the antibody vial and the solution was incubated on a rotator for two hours at 4$^{\circ}$C. The samples were centrifuged, put on ice and the supernatant was removed. The beads were washed twice with 1ml IAP, followed by three washes with 1ml chilled HLPC water. Immunoprecipitated material was eluted at room temperature in 55$\mu$l and 50$\mu$l 0.15\% TFA in water, letting the sample stand for 10 minutes after each elution, with gentle mixing every two-three minutes. The eluates were centrifuged and the supernatant was transferred to new tubes. Peptide material was desalted and concentrated using 1ml C-18 Sep-Pak cartridges as above. Prior to mass spectrometry analysis, purified GlyGly-modified peptide eluates and matching proteome material were dried by vacuum centrifugation, and re-suspended in solution A.
%

\section{Liquid-chromatography-tandem mass spectrometry}
Liquid-chromatography-tandem mass spectrometry (LC-MS/MS) analysis was performed using a Dionex Ultimate 3000 nano-ultra high pressure reverse-phase chromatography coupled on-line to an Orbitrap Fusion Lumos mass spectrometer (Thermo Scientific) (REF: adan's 3-5 dropbox). In brief, samples were separated on an EASY-Spray PepMap RSLC C18 column (500 mm × 75$\mu$m, 2$\mu$m particle size; Thermo Scientific) over a 60 min (120 min in the case of the matching proteome) gradient of 2–35\% acetonitrile in 5\% dimethyl sulfoxide (DMSO), 0.1\% formic acid at 250 nl/min. MS1 scans were acquired at a resolution of 60000 at \textit{m/z} 200 and the top 12 most abundant precursor ions were selected for high collision dissociation (HCD) fragmentation.

%%%%%%
\section{Data Processing}
\subsection{Bulk RNA-Seq}
Fasta files were processed using a CGAT-flow (REF) pipeline, workflow can be found at: \url{https://github.com/cgat-developers/cgat-flow/blob/master/cgatpipelines/tools/pipeline_rnaseqdiffexpression.py}. Pseudo-alignment tool, Kallisto (REF), was implemented to pseudo-align reads to the reference human genome sequence (GRCH38 (hg38) assembly) and to construct a counts matrix of samples against transcripts (/GENES??). DESeq2 (REFERENCE) was used for differential expression analysis of the generated  counts  matrix  (using  negative  binomial  generalized  linear  models) within the R statistical framework (v3.5.1).
%% XGR and GSEA ????

\subsection{Single-cell RNA-Seq}
Pipeline etc...

\subsection{LC-MS/MS}
Mass-spectrometry raw data were searched against the UniProtKB human sequence data base and label-free quantitation (LFQ) was performed using MaxQuant Software (v1.5.5.1). Digestion was set to trypsin/P. Search parameters were set to include carbamidomethyl (C) as a fixed modification, oxidation (M), deamidation (NQ), and phosphorylation (STY) as variable modifications. A maximum of 2 missed cleavages were allowed for phosphoproteome analysis and 3 for the GlyGly peptidome analysis, with matching between runs. LFQ quantitation was performed using unique peptides only. Label-free interaction data analysis was performed using Perseus (v1.6.0.2). Results were exported to Microsoft Office Excel Sheets and imported into the R statistical framework (v3.5.1) for further analysis.


\section{CyTOF}
Get data off ADAM
\subsection{CyTOF stuff}
