\chapter{\label{ch:3-methods}Methods}

%\minitoc

\section{Cell culture}
\subsection{AMO-1 cells}
AMO-1 cells, plasma cells from a 64-year old female myeloma patient, were used as a model cell-line for multiple myeloma.
Proteasome inhibitor-sensitive AMO-1 cells (WT), bortezomib resistant AMO-1 cells (aBTZ) and carfilzomib resistant AMO-1 cells (aCFZ) were kindly gifted by the Driessen lab\cite{soriano2016proteasome}.
Bortezomib resistant, carfilzomib resistant, pomalidomide resistant (aPom) and dual bortezomib-pomalidomide resistant (aBTZPom) AMO-1 cells were also generated by Dr James Dunford by continual and escalating drug exposure of drug-sensitive (WT) cells.
The Driessen lab AMO-1 resistant cells are commonly referred to as `batch 1' in this work, and the in-house resistant AMO-1 cells as `batch 2'.
aCFZ and aBTZ cells were kept in 100\si{\nano\Molar} of their respective proteasome inhibitor and aPom cells were kept in 6\si{\micro\Molar} pomalidomide.
AMO-1 cells were cultivated in RPMI-1640 medium (Thermofisher, UK), supplemented with 10\% fetal bovine serum (FBS), 100\si{\ug\per\ml} streptomycin and 100 U/ml penicillin (P/S) and 2\si{\milli\Molar} L-glutamine (Invitrogen, UK).
Cells were passaged when they reached approximately 1.5-2 million cells per \ml{}.
AMO-1 cells are suspension cells and were split twice a week to approximately 0.5 million cells per \ml{}.

\section{Compounds}

\subsection{Proteasome inhibitors}
<WHERE were they obtained> etc etc.

\subsection{Epigenetic inhibitors}
The Oppermann group has an epigenetic compound screening library, consisting of 144 compounds.
The compounds were obtained XYZ <where did Jim get compounds> SGC????
A dual TRIM24/BRPF inhibitor was identified as a possible candidate to reverse drug-resistance in AMO-1 cells.
The structure of the inhibitor is shown below in figure \ref{fig:trim24_structure}.

%% TRIM24 structure
\begin{figure}[htb]
\centering
\includegraphics[width=0.7\textwidth]{figures/Methods/trim24_structure.png}
\caption[TRIM24i structure]{TRIM24 inhibitor chemical structure}
\label{fig:trim24_structure}
\end{figure}
%%

\section{Assays}
\subsection{Cell viability assays}
10X presto blue (alamar??) was added in a 1:10 ratio to cells in suspension and incubated at 37\C for two to three hours.
Plates were read [DETAILS OF MACHINE AND PROTOCOL, e.g. wavelength]

\subsection{Dose response curves}
90\ul{} of cells in fresh media were seeded into 96-well plates a day prior to treatment with compound.
A total of 10,000 cells were seeded into each well.
No cells were placed in edge wells, to avoid edge effects.
The following day, media 0\% viability controls were placed in the first and last row.
Drug concentrations were made up 1000x the desired final concentration in eppendorfs.
Drugs were diluted 1 in 100 in 96 well round bottom plates with media and then 10\ul was added to the 90\ul of seeded cells in triplicate.
Cells were treated with DMSO in triplicate as 100\% viability controls.

\section{Bulk RNA-seq}
\subsection{RNA extraction}
RNA was extracted and purified using the Direct-Zol RNA MiniPrep kit (Zymo, USA), following the manufacturer's protocol.
In brief, for each sample, approximately 100,000 cells were lysed in 300\ul{} of TRIzol and the lysate was transferred to a microcentrifuge tube.
300\ul{} of ethanol was added to the lysed samples and vortexed.
The mixture was transferred to miniPrep columns and centrifuged at 10,000-16,000g for 30 seconds.
The column was washed twice with 400\ul{} of Direct-Zol pre-wash and once with 700\ul{} of RNA wash buffer
The column was transferred to an RNase-free tube and eluted with 50\ul{} of nuclease-free water and centrifuged.

The RNA concentration was quantified using a NanoDrop ND-1000 Spectrophotometer (Thermo Fisher Scientific, USA), and samples were stored at -80\C{}.
Samples were normalised to 100\si{\ng} with nuclease-free water.


\subsection{RNA library preparation}
NEBNext\textsuperscript{\textregistered} Ultra II directional RNA library prep kit for Illumina\textsuperscript{\textregistered} with TruSeq indexes was used to prepare RNA libraries, following the manufacturer's protocol.
RNA concentration was normalised to 100\si{\ng} with nuclease-free water, made up to 50\ul{}.
The NEBNext Poly(A) mRNA Magnetic Isolation Module (NEB, USA) was used to enrich poly-adenylated RNA. READ booklet in lab


The molarities of the libraries were determined by electrophoresis on a TapeStation (Agilent, USA).

\section{Single-cell RNA-seq}
\subsection{Cell encapsulation}
The Drop-Seq protocol\cite{macosko2015highly} was followed for single-cell RNA-Seq sample preparation.
Cells were loaded into a microfluidics cartridge.
Nadia, an automated microfluidics device (Dolomite Bio, UK), performed cell capture, cell lysis and reverse transcription.
Reverse transcription reactions were performed using ChemGene beads or (ATDBio beads 2020 onwards!!!!! might need to change if reperform).

\subsection{Library preparation}
Beads were collected from the device and cDNA amplification was performed.
The beads were treated with Exo-I prior to PCR.
The amplified, purified cDNA then underwent tagmentation reactions.
A TapeStation (Agilent, USA) was used to assess library quality.
The samples were pooled together and split across multiple sequencing runs.


\section{ATAC-seq}\label{sec:methods_atac}

\subsection{Cell lysis}
Approximately 2 million cells were collected in 15\ml{} falcon tubes for each condition.
The cells were centrifuged at 300g for 5 minutes at 4\C{} and the supernatant was discarded.
The cell pellets were resuspended in 1\ml{} of cold PBS and centrifuged at 300g for 5 minutes at 4{\C}, the supernatant was then discarded.
Fresh lysis buffer was prepared (see tables \ref{tab:resuspension_buffer,tab:lysis_buffer}) with occasional gentle flicking.
The falcons were then centrifuged at 500g for 10 minutes at 4\C{}.
The supernatant (cytoplasm) was discarded, leaving the nuclei pellet.

% Buffer tables

% \usepackage[table,xcdraw]{xcolor}
% If you use beamer only pass "xcolor=table" option, i.e. \documentclass[xcolor=table]{beamer}
\begin{table}[h]
\centering
\begin{tabular}{|l|l|}
\hline
\textbf{Resuspension buffer} & \textbf{Volume (\ul{})} \\ \hline
\rowcolor[HTML]{EFEFEF}
1\si{\Molar}  Tris-HCl (pH 7.5) & 500 \\ \hline
5\si{\Molar}  NaCl & 100 \\ \hline
\rowcolor[HTML]{EFEFEF}
1\si{\Molar}  MgCl\textsubscript{2} & 150 \\ \hline
Nuclease-free water & 49,250 \\ \hline
\rowcolor[HTML]{EFEFEF}
\textbf{Total} & \textbf{50,000 (50\ml{})} \\ \hline
\end{tabular}
\caption{Resuspension buffer recipe}
\label{tab:resuspension_buffer}
\end{table}

% Lysis
\begin{table}[h]
\centering
\begin{tabular}{|l|l|}
\hline
\textbf{Lysis buffer} & \textbf{Volume (\ul{})} \\ \hline
\rowcolor[HTML]{EFEFEF}
Resuspension buffer & 940 \\ \hline
10\% non-iodet P40 & 50 \\ \hline
\rowcolor[HTML]{EFEFEF}
10\% tween 20 & 10 \\ \hline
\textbf{Total} & \textbf{1000 (1\ml{})} \\ \hline
\end{tabular}
\caption{Lysis buffer recipe}
\label{tab:lysis_buffer}
\end{table}

\subsection{Transposition}
Pellets were resuspended in 890\ul{} transposition mix (500\ul{} 2X TD buffer, 330\ul\ 1X PBS, 10\ul{} 10\% Tween-20, 10\ul{} 5\% Digitonin, 40\ul{} nuclease-free water).
For each condition, 176\ul{} was taken in triplicate and transferred to LoBind 1.5\ml{} eppendorfs (Eppendorf, UK).
4\ul{} Tn5 enzyme was added to each eppendorf.
The samples were then incubated at 37\C{} for an hour at 500rpm.

\subsection{DNA purification}
Magic bead clean-ups were performed to purify the DNA.
220\ul{} of magic beads was added to each tube (~1.2X), vortexed, centrifuged for 1-2 seconds and incubated at room temperature for 5 minutes.
Tubes were placed on a magnetic rack for 2 minutes, until the solution was clear.
The liquid from the tubes was aspirated away, leaving about 10\ul\ of liquid remaining.
200\ul{} of 80\% ethanol was dispensed over the beads, the tubes were vortexed, spun and placed back on the magnetic rack until the solution was clear and then the ethanol was aspirated away.
This wash was repeated for a total of two ethanol washes.
Following aspiration on the 2\nd wash, an additional spin was performed and the tubes were placed back on the magnetic rack and any remaining liquid was aspirated away, to ensure all ethanol was removed.
The beads were left to air dry for 3-5 minutes on the magnetic rack with the lids of the tubes open.
The tubes were removed from the magnetic rack and eluted with 26\ul{} 0.1X TE buffer (Zymo Research, UK).
The tubes were vortexed, spun and left to incubate for 5 minutes at room temperature, before being placed back on the magnetic rack.
The eluant was transferred to fresh LoBind tubes.
The purified DNA was then stored at -20\C{} until PCR amplification was ready to be performed.

\subsection{PCR amplification}
20\ul\ of purified DNA from each sample was mixed with 20\ul{} nuclease-free water, 5\ul{} ATAC-seq universal primer, 50\ul{} Nebnext high fidelity 2X master mix and 5\ul{} unique ATAC-seq index primer, and split across two PCR tubes.
The PCR tubes were put in a thermocycler with a lid temperature of 103.5\C{}, they were heated to 72\C\ for 5 minutes, 98\C\ for 30 seconds, and then cycled at 98\C{} for 10 seconds, 63\C{} for 30 seconds and 72\C{} for 1 minute, 13 times.
Samples were then held at 4\C{}.
The paired PCR tubes for each sample were then combined into single 1.5\ml{} LoBind eppendorfs.
Magic bead clean-up (as above) was performed, with 110\ul{} magic beads (1.1X).
The purified amplified DNA was eluted in 20\ul{} 0.1X TE buffer and transferred to new LoBind tubes.
D1000 high sensitivity screen tapes and 2200 TapeStation (Agilent, USA) were used to quantify libraries.


%\section{ChIP maybe}
%\subsection{ChIP stuff}

\section{Pooling, denaturing and diluting libraries}
Libraries were then denatured and diluted, folowing the NextSeq denature and dilute libraries guide, ready for sequencing.


\section{Sequencing}
Sequencing of the resultant libraries was performed on the NextSeq 500 (Illumina, USA) platform using a paired-end run, according to the manufacturer's instructions.

%
\section{Phosphoproteomics}\label{sec:methods-phospho}
%
\subsection{Collecting cell pellets}
Greater than 20 million cells for each condition (in triplicate) was taken.
The cell suspension was centrifuged at 1500g for five minutes.
The supernatant was removed, the pellet was re-suspended in 500\ul{} of ice-cold PBS, transferred to a 1.5\ml{} eppendorf and centrifuged for a further five minutes.
The supernatant was removed using a pipette and the pellet was stored at -80\C{}.

\subsection{Cell lysis}
300\ul\ of fresh lysis buffer (10\ml{} RIPA buffer, 3\ul{} benzonase, 1 tablet phos stop) was added to each pellet, vortexed and left for 10 minutes on ice and then sonicated.
The supernatant was transferred to a fresh tube.

\subsection{Protein quantification}
Protein concentrations were determined by BCA protein assay (Thermofisher, UK). 400\si{\ug} of protein was taken from each sample. Samples were made up to a volume of 200\ul{} with MilliQ-H\textsubscript{2}O.

\subsection{Protein Digestion}
Kessler lab protocols were followed (\url{https://www.tdi.ox.ac.uk/research/research/tdi-mass-spectrometry-laboratory/mass-spectrometry/protocols-and-tools}).
The lysed samples were reduced with 5\ul{} of 200\si{\milli\Molar} DTT in 0.1 M Tris buffer and incubated for 40 minutes at room temperature.
The reduced samples were alkylated with 20\ul{} of 200\si{\milli\Molar} iodoacetamide in 0.1\si{\Molar} Tris buffer, vortexed and then incubated for 45 minutes in the dark at room temperature.
The protein was precipitated using methanol/chloroform extraction.
The alkylated samples were transferred to 2ml eppendorfs.
600\ul{} of methanol was added to each sample, followed by 150\ul{} of chloroform and then vortexed gently.
450\ul{} of MilliQ-H\textsubscript{2}O was then added and vortexed gently.
The samples were centrifuged at maximum speed on a table top centrifuge for one minute.
The upper aqueous phase was removed, without disturbing the precipitate at the interface.
450\ul{} of methanol was added to each sample, without disturbing the disc and centrifuged for two minutes.
Protein pellets were resuspended, one sample at a time: the supernatant was removed and 100\ul{} of 6M urea in 0.1M Tris buffer was added.
The samples were vortexed and then sonicated (???).
Samples were diluted with 500\ul{} MilliQ-H\textsubscript{2}O, to ensure the final urea concentration was below 1\si{\Molar}.
Porcine trypsin (Sequencing Grade Modified Trypsin; Promega, USA) was added in a 1:50 ratio of enzyme:total protein content of sample, such that 40\ul{} of trypsin solution containing 8\si{\ug} trypsin in 0.1\si{\Molar} Tris buffer was added to each sample.
Samples were left to digest overnight at 37\C{} in an incubator shaker.

\subsection{Peptide purification}
The following day, the reaction was stopped, acidifying samples to 1\% Trifluoroacetic acid (TFA).
Samples were desalted and concentrated using 1ml C-18 Sep-Pak (Waters) cartridges.
Two reagents were used: solution A (98\% MilliQ-H\textsubscript{2}O, 2\% Acetonitrile (CH\textsubscript{3}CN) and 0.1\% TFA) for washing and solution B (65\% Acetonitrile, 35\% MilliQ-H\textsubscript{2}O and 0.1\% TFA) for activation and elution.
The columns were flushed with 1ml of solution B and then washed with 1\ml{} of solution A.
The digested samples were added to the columns and vacuumed through slowly.
Two 1\ml{} washes with solution A were performed.
Fresh, labelled eppendorfs were placed beneath the columns and peptides were eluted with 500\ul{} of solution B.
For phosphopeptide-enrichment, 90\% of the peptides were removed for Immobilized Metal Affinity Chromatography (IMAC) on a Bravo Automated Liquid Handling Platform (Agilent).
10\% of the peptides were used for total proteome analysis.
Eluted peptides were dried using a vacuum concentrator (Speedvac, Eppendorf) and stored at -20\C{} until analysis by mass spectrometry (MS).
Prior to MS analysis, dried peptides were resuspended in solution A.


\section{Ubiquitinomics}
%
\subsection{Collecting cell pellets}
100 million cells were taken for each condition in triplicate.
The cell suspension was centrifuged at 1500g for five minutes.
The supernatant was removed, the pellet was re-suspended in 500\ul{} of ice-cold PBS and centrifuged for a further five minutes.
The supernatant was removed and the pellet was stored at -80\C{}.

\subsection{Cell lysis}
PMTScan Ubiquitin Remnant Motif Kit (K-$\varepsilon$-GG; Cell signalling, USA) was used, following the manufacturer's protocol (REF).
Pellets were solubilized and denatured in 4\ml{} urea lysis buffer (20\si{\milli\Molar} HEPES, pH 8.0, 9\si{\Molar} urea, 1\si{\milli\Molar} sodium orthovanadate, 2.5\si{\milli\Molar} sodium pyrophosphate, 1\si{\milli\Molar} $\beta$-glycerophosphate).
The lysates were sonicated on ice, with two bursts of 15 seconds with a one minute break in-between.

\subsection{Protein quantification}
Protein concentrations were determined by BCA protein assay (Thermofisher, UK).
All samples were found to contain between 10\si{\mg} and 20\si{\mg} of protein, so all of the available protein was used, with no normalisation.

\subsection{Protein digestion}
Lysates were reduced using dithiothreitol (DTT) at a final concentration of 4.5 mM for 30 minutes at room temperature.
The reduced samples were alkylated using iodoacetamide (100\si{\milli\Molar} final) for 15 minutes in the dark at room temperature.
The alkylated samples were diluted four-fold with 20\si{\milli\Molar} HEPES (pH 8.0) and digested with 400\ul{} trypsin solution, containing 1\si{\mg\per\ml} trypsin-TPCK (Worthington, LS003744) in 1\si{\milli\Molar} HCl.
Samples were left to digest overnight at room temperature on a rotator.

\subsection{Peptide purification}
The following day, the reaction was stopped, acidifying samples to 1\% Trifluoroacetic acid (TFA)\@.
Samples were desalted and concentrated using 10ml C-18 Sep-Pak (Waters) cartridges.
The columns were activated using 5\ml{} of solution B, washed with 10\ml{} of solution A\@.
The samples were added to the columns and ran through slowly.
The peptides were washed with 10\ml{} of solution A\@.
The cartridges were then removed from the vacuum and the peptides were eluted into fresh falcon tubes with 6\ml{} of solution B, using the plunger of the syringes.
20\si{\ug} of digested protein was removed from each sample for matching total proteome analysis.
The eluate was kept at -80\C{} overnight.
The frozen peptide solutions were lyophilized for two days and then stored at -80\C{}.
%

\subsection{Immunoaffinity purification}
10x immunoaffinity purification (IAP) buffer provided with PTMScan Kit was diluted to 1x concentration with MilliQ-H\textsubscript{2}O.
Purified peptides pellets were resuspended in 1.4\ml{} of IAP buffer by pipetting up and down and transferred to 1.7\ml{} eppendorfs.
The samples were centrifuged at 4\C{} for 5 minutes at 10000xg and kept on ice whilst preparing antibody beads.
The anti-body bead slurry was centrifuged (30 seconds at 2000 g) and 1\ml{} of PBS was added and then centrifuged.
The supernatant was removed and the antibody beads were washed a further four times with PBS and resuspended in 40\ul{} of PBS.
The peptide solution was transferred to the antibody vial and the solution was incubated on a rotator for two hours at 4\C{}.
The samples were centrifuged, put on ice and the supernatant was removed.
The beads were washed twice with 1\ml{} IAP, followed by three washes with 1\ml{} chilled HLPC water.
Immunoprecipitated material was eluted at room temperature in 55\ul{} and 50\ul{} 0.15\% TFA in water, letting the sample stand for 10 minutes after each elution, with gentle mixing every two-three minutes.
The eluates were centrifuged and the supernatant was transferred to new tubes.
Peptide material was desalted and concentrated using 1\ml{} C-18 Sep-Pak cartridges as above.
Prior to mass spectrometry analysis, purified GlyGly-modified peptide eluates and matching proteome material were dried by vacuum centrifugation, and re-suspended in solution A.
%

\section{Liquid-chromatography-tandem mass spectrometry}
Liquid-chromatography-tandem mass spectrometry (LC-MS/MS) analysis was performed using a Dionex Ultimate 3000 nano-ultra high pressure reverse-phase chromatography coupled on-line to an Orbitrap Fusion Lumos mass spectrometer (Thermo Scientific) (REF: adan's 3-5 dropbox).
In brief, samples were separated on an EASY-Spray PepMap RSLC C18 column (500\si{\mm} × 75\si{\um}, 2\si{\um} particle size; Thermo Scientific) over a 60 min (120 min in the case of the matching proteome) gradient of 2–35\% acetonitrile in 5\% dimethyl sulfoxide (DMSO), 0.1\% formic acid at 250\si{\nano\litre\per\minute}.
MS1 scans were acquired at a resolution of 60000 at \textit{m/z} 200 and the top 12 most abundant precursor ions were selected for high collision dissociation (HCD) fragmentation.

\section{CyTOF}
Get data off ADAM
\subsection{CyTOF stuff}

%%%%%%%%%%%%%%%%%%%%%%%%%
% Computational methods
%%%%%%%%%%%%%%%%%%%%%%%%%

\section{Data Processing}\label{sec:data_processing}
\subsection{Bulk RNA-Seq}
Fasta files were processed using a CGAT-flow\cite{sims2014cgat} pipeline, the workflow can be found at: \url{https://github.com/cgat-developers/cgat-flow/blob/master/cgatpipelines/tools/pipeline_rnaseqdiffexpression.py}.
The pseudo-alignment tool, Kallisto\cite{bray2016near}, was implemented to pseudo-align reads to the reference human genome sequence (GRCH38 (hg38) assembly) and to construct a counts matrix of samples against transcripts.
DESeq2\cite{love2014moderated} was used for differential expression analysis of  counts  matrices  (using  negative  binomial  generalized  linear  models) within the R statistical framework (v3.5.1).
XGR\cite{fang2016xgr}, Reactome\cite{fabregat2017reactome} and KEGG\cite{kanehisa2017kegg} were used to perform pathway analysis, within R\@.
Org.Hs.eg.db\cite{carlson2019org}, AnnotationDbi\cite{pages2020annotationdbi} and biomaRt\cite{durinck2009mapping} were used for converting between Ensembl IDs, HGNC symbols and ENTREZ IDs.

\subsection{ATAC-Seq}
Raw ATAC reads (in fasta file format) were mapped to the GRCh38 reference genome using the CGAT-flow mapping pipeline (\url{https://github.com/cgat-developers/cgat-flow/blob/master/cgatpipelines/tools/pipeline_mapping.py}), using the mapper Bowtie.
The mapped bam files were then used as input for the CGAT-flow peak calling pipeline ((\url{https://github.com/cgat-developers/cgat-flow/blob/master/cgatpipelines/tools/pipeline_peakcalling.py})).
Filtering was performed to filter out [what is filterd out!!] and peak calling was implemented using macs2 (v2.2.7)\cite{zhang2008model}.


\subsection{Single-cell RNA-Seq}
The computational pipeline outlined in section \ref{sec:scRNA_pipeline} was used to process scRNA data.

\subsection{LC-MS/MS}
Mass-spectrometry raw data were searched against the UniProtKB human sequence data base and label-free quantitation (LFQ) was performed using MaxQuant Software (v1.5.5.1).
Digestion was set to trypsin/P.
Search parameters were set to include carbamidomethyl (C) as a fixed modification, oxidation (M), deamidation (NQ), and phosphorylation (STY) as variable modifications.
A maximum of 2 missed cleavages were allowed for phosphoproteome analysis and 3 for the GlyGly peptidome analysis, with matching between runs.
LFQ quantitation was performed using unique peptides only.
Label-free interaction data analysis was performed using Perseus (v1.6.0.2).
Results were exported to Microsoft Office Excel and imported into the R statistical framework (v3.5.1) for further analysis.


