\chapter{\label{ch:6-results_omics}Multi-omics characterisation of PI resistance in AMO-1 cells}

%\minitoc

\section{Introduction}\label{sec:omics-intro}
Numerous transcriptional changes have been implicated in the development of acquired drug-resistance in MM.
It is also pertinent to ascertain if the changes seen in the transcriptome are reflected in protein expression.
Therefore both transcriptional changes and proteomic changes must be studied.

The changes between PI-sensitive (WT) AMO-1 cells and PI-resistant (aCFZ/aBTZ) AMO-1 cells will be looked at.
In addition, the BRPF1B/TRIM24 dual inhibitor (TRIM24i) was shown in the previous chapter to effectively kill carfilzomib resistant AMO-1 cells in the presence of carfilzomib (but not without carfilzomib), whilst not affecting WT AMO-1 cells.
Thus suggesting TRIM24i has a mechanism of action that is capable of reversing aCFZ cells' resistance, and re-sensitising them to carfilzomib again.
aCFZ cells treated with TRIM24i will be compared to control aCFZ cells, to better understand the mechanism of action of TRIM24i.
Furthermore, these changes must be compared to the differences seen between WT and aCFZ cells.
Considering TRIM24i seems to make aCFZ cells sensitive to carfilzomib again, it may be the case that it is reversing some of the changes seen between WT and aCFZ cells.

\section{Results}\label{sec:omics-results}

\subsection{Transcriptional changes}\label{sec:omics-transcriptomics}
To investigate the transcriptional changes that underlie the development of acquired-PI-resistance in multiple myeloma a number of RNA-seq experiments to compare WT, carfilzomib resistant (aCFZ) and bortezomib (aBTZ) AMO-1 cells were performed, both at the bulk and single-cell level.

Principle component analysis (PCA) showed differential clustering of the different cell lines...
[PCA of different cell lines, WT, aCFZ, aBTZ]


xyz differentially expressed genes (DEGs) were identified between WT and aCFZ cells.

\subsubsection{TRIM24i reversal}
To investigate how TRIM24i reverses the resistance phenotype in aCFZ cells, bulk RNA-seq was employed.
aCFZ cells (100\si{\nano\Molar} carfilzomib) were treated with DMSO or 10\si{\mu\Molar} TRIM24i (in triplicate).
The results from this experiment were compared against the results from the previous bulk-RNA experiment of WT controls vs aCFZ cells (100\si{\nano\Molar} carfilzomib).

%% PCA, sample-sample clustering, volcano, venn diagram
%% Transcriptomics figure 1
\begin{figure}[htb]
\centering
\includegraphics[width=\textwidth]{figures/Results/Transcriptomics/bulk_rna_figure1.pdf}
\caption[WT, aCFZ and TRIM24i reversal, transcriptomics differential expression and clustering]{\textbf{A} shows a PCA plot for WT AMO-1 cells, aCFZ cells (100\si{\nano\Molar} carfilzomib) and aCFZ (100\si{\nano\Molar} carfilzomib) cells treated with 10\si{\mu\Molar} TRIM24i.
\textbf{B} shows a heatmap of sample-sample distances using hierarchical clustering of the samples.
\textbf{C} shows volcano plots ($\log_{2}$ fold change vs $-\log_{10}$ Student's T-test $p$-value) for WT vs aCFZ AMO-1 cells (100\si{\nano\Molar} carfilzomib) and for aCFZ cells (100\si{\nano\Molar} carfilzomib) vs aCFZ cells (100\si{\nano\Molar} carfilzomib) treated with 10\si{\mu\Molar} TRIM24i.
\textbf{D} shows a four-way venn diagram for differentially expressed genes ($|\log_{2}FC| \geq0.5$) , either up-regulated or down-regulated, between WT and aCFZ cells (CFZ) or aCFZ cells and aCFZ cells treated with TRIM24i (TRIM24).}
\label{fig:bulk_fig1_de_clustering}
\end{figure}
%%

As seen in figure \ref{fig:bulk_fig1_de_clustering}, PCA and sample-sample clustering shows TRIM24i treated aCFZ cells clustering away from aCFZ controls.
The sample-sample clustering shows TRIM24i treated aCFZ cells clustering closer to WT cells than the aCFZ controls are, suggesting the TRIM24i treatment has made them more similar to WT cells, agreeing with our hypothesis following the dose-response curve results.

%%%%%%%%%%%%%%%%%%%%%%%%
% Transcription factors
%%%%%%%%%%%%%%%%%%%%%%%%

%% TF heatmap
%% Transcriptomics figure 2
\begin{figure}[p]
\centering
\includegraphics[width=0.8\textwidth]{figures/Results/Heatmaps/WT_CFZ_TRIM24_combined_TFs_heatmap_cropped.png}
\caption[Transcription factor heatmap]{Transcription factor heatmap.
Variance stabilised expression of transcription factors differentially expressed ($p \leq 0.01$) between aCFZ controls and TRIM24i treated aCFZ cells are shown for WT cells, aCFZ controls and TRIM24i treated aCFZ cells.
Samples from experiment 2 have an asterisk (*) next to them, samples from experiment 1 have no asterisk.
Batch effects have been removed between experiments}
\label{fig:bulk_tf_heatmap}
\end{figure}
%%

%% TF venn diagrams
%% Transcriptomics figure 3
\begin{figure}[htb]
\centering
\includegraphics[width=0.9\textwidth]{figures/Results/Transcriptomics/bulk_rna_venn_TFs.pdf}
\caption[Transcription factor venn diagrams: WT vs aCFZ and aCFZ vs aCFZ TRIM24i]{Venn diagrams are shown for up-regulated and down-regulated (DE, $p\leq0.01$) transcription factors (TFs) in WT vs aCFZ cells and aCFZ cells vs TRIM24i treated aCFZ cells.
The intersection of the venn diagrams are of interest as they represent `reversed' TFs by TRIM24i treatment which make aCFZ resistant cells more like WT cells again.
The HGNC symbols of the TF genes in each intersection are listed below the venn diagrams.}
\label{fig:bulk_tf_venn}
\end{figure}
%%

Next, transcription factors (TFs) were explored.
Figure \ref{fig:bulk_tf_heatmap} shows a heatmap for DE ($p\leq0.01$) transcription factors between aCFZ controls and TRIM24i treated aCFZ cells.
Figure \ref{fig:bulk_tf_venn} shows a venn diagram for differentially expressed ($p\leq0.01$) TFs.
276 TFs were DE between WT and aCFZ cells, 101 of which were up-regulated in aCFZ cells.
133 TFs were DE between DMSO treated aCFZ cells and TRIM24i treated aCFZ cells, 82 of which were up-regulated in TRIM24i treated aCFZ cells.
The intersection of the venn diagrams represent TFs which may be considered `reversed' by TRIM24i treatment (i.e. TFs that are up-regulated in WT vs aCFZ, but down-regulated in aCFZ vs TRIM24i treated aCFZ cells, and vice versa).
This list of TFs is of interest, as some of them are likely to be the master regulators by which TRIM24i exerts its PI resistance reversal effects.

%%%%%%%%%%%%%%%%%%%%
% Pathway analysis
%%%%%%%%%%%%%%%%%%%%

Following, transcription factor analysis, pathway analysis was performed (method outlined in section \ref{sec:data_processing}).
The top 500 DE genes between aCFZ controls and TRIM24i treated aCFZ cells (ordered by fold-change magnitude) were compared to the background 11529 genes.
Using Reactome, eight distinct pathways were identified as significantly enriched (False Discovery Rate (FDR; adj-p) $\leq0.05$).
Using KEGG, two distinct pathways were identified as significantly enriched.
The enriched pathways are shown in figure \ref{fig:pathway_cfz_trim_barplots}.

% Pathway analysis, reactome KEGG acfz vs acfz trim24
\begin{figure}[htb]
%1
\begin{subfigure}[t]{0.5\textwidth}
    \includegraphics[width=\textwidth]{figures/Results/Transcriptomics/pathway_cfz_trim_reactome.png}
    \caption{Reactome}
    \label{fig:pathway_barplot_cfz_trim_reactome}
\end{subfigure}
%
%\medskip
\begin{subfigure}[t]{0.5\textwidth}
    \includegraphics[width=\textwidth]{figures/Results/Transcriptomics/pathway_cfz_trim_kegg.png}
    \caption{KEGG}
    \label{fig:pathway_barplot_cfz_trim_kegg}
\end{subfigure}
    \caption[Enriched pathways- aCFZ TRIM24i vs WT]{Pathway analysis of the top 500 differentially expressed genes between aCFZ cells (in 100\si{\nano\Molar} carfilzomib) treated with DMSO and aCFZ cells (in 100\si{\nano\Molar} carfilzomib) treated with 10\si{\micro\Molar} TRIM24i. \ref{fig:pathway_barplot_cfz_trim_reactome} used the Reactome pathway database, it shows all eight significantly enriched pathways (FDR$\leq0.05$). \ref{fig:pathway_barplot_cfz_trim_kegg} used the pathway database, KEGG, only two pathways were significantly enriched, the top 10 are displayed.}
\label{fig:pathway_cfz_trim_barplots}
\end{figure}

Pathway analysis was also performed for DE genes that were `reversed' following TRIM24i treatment, to be more like WT again.
1206 DE `reversed' genes ($p-adj\leq0.01$) (up-regulated/down-regulated in TRIM24i treated aCFZ cells vs aCFZ controls but down-regulated/up-regulated in aCFZ cells vs WT cells) were compared to a background of 13715 genes.
Using Reactome, 12 distinct pathways were identified as significantly enriched (figure \ref{fig:trim24_reversed_pahtway_barplot_reactome}).



%% Reversed pathway analysis reactome
\begin{figure}[hbt]
\centering
\includegraphics[width=0.6\textwidth]{figures/Results/Transcriptomics/pathway_reactome_trim24_reversed.png}
\caption[Enriched pathways- TRIM24i reversed genes]{Barplot of significantly enriched pathways and their FDR.
Pathway analysis of 1206 genes differentially expressed ($p-adj\leq0.01$) and their expression reversed between WT vs aCFZ and aCFZ vs TRIM24i-treated aCFZ, against a background of 13715 genes, using the Reactome pathway database.)}
\label{fig:trim24_reversed_pahtway_barplot_reactome}
\end{figure}
%%

%% TALK ABOUT UPR and UBC, ATF4 and PERK, ER STRESS-- discussion section ??
As shown in figures \ref{fig:trim24_reversed_pahtway_barplot_reactome} and \ref{fig:pathway_cfz_trim_barplots}, the unfolded protein response (UPR) and cell death/apoptosis were enriched.
DE genes involved in the UPR up-regulated by TRIM24i include \textit{ATF3}, \textit{ATF4}, \textit{DDIT3} (CHOP), \textit{ASNS} and \textit{EIF2AK3} (PERK).
This suggests that TRIM24i promotes cell death by triggering the unfolded protein response and causing ER stress.
etc etc etc... 

\subsection{Total proteome}\label{subsec:omics-proteome}
To investigate proteomic changes in drug resistance and the affect of TRIM24i treatment, label-free proteomic analysis by liquid chromatography coupled with tandem mass spectrometry (LC-MS/MS) was employed.
Phosphoproteomics and ubiquitinomics were performed, both alongside total proteome analysis.



\subsection{Phosphoproteomics}\label{subsec:omics-phospho}
Phosphoproteomics was performed (method outlined in section \ref{sec:methods-phospho}).
Five conditions were investigated: WT, aCFZ (+100\si{\nano\Molar} carfilzomib), aCFZ treated with TRIM24 inhibitor (+100\si{\nano\Molar} carfilzomib), aCFZ treated with TRIM24 inhibitor (no carfilzomib), and WT treated with TRIM24 inhibitor.

%% Phospho PCA
\begin{figure}[htb]
\centering
\includegraphics[width=0.7\textwidth]{figures/Results/PCA/phospho_PCA_WT_CFZ_TRIM24_conditions_cb.png}
\caption[Phosphoproteomics PCA]{PCA plot for phosphoproteomic data.
WT, WT treatd with TRIM24i, aCFZ + 100\si{\nano\Molar} carfilzomib, aCFZ treated with 10\si{\mu\Molar} TRIM24i and 100\si{\nano\Molar} carfilzomib, and aCFZ treated with 10\si{\micro\Molar} TRIM24i and no carfilzomib.}
\label{fig:phospho_PCA}
\end{figure}
%%

\subsection{Ubiquitinomics}\label{subsec:omics-glygly}

%% Phospho PCA
\begin{figure}[htb]
\centering
\includegraphics[width=0.7\textwidth]{figures/Results/PCA/PCA_ubiquitinomics_WT_CFZ_TRIM24_cb.png}
\caption[Ubiquitinomics PCA]{PCA plot for Ubiquitinomics data.
WT, WT treatd with TRIM24i, aCFZ + 100\si{\nano\Molar} carfilzomib, aCFZ treated with 10\si{\mu\Molar} TRIM24i and 100\si{\nano\Molar} carfilzomib, and aCFZ treated with 10\si{\micro\Molar} TRIM24i and no carfilzomib.}
\label{fig:glygly_PCA}
\end{figure}
%%


