\chapter{\label{ch:6-sc}Single-cell RNA-seq analysis of PRS inhibitors}

%\minitoc

\section{Introduction}
MM cells grow within the bone marrow and are supported as they grow by their microenvironment.
The MM microenvironment comprises a cellular compartment (composed of immune cells, endothelial cells, osteoblasts, osteoclasts and stromal cells) and a non-cellular compartment (composed of the extracellular matrix (ECM), cytokines, chemokines and growth factors)\cite{manier2012bone, kawano2015targeting}.
There are interactions between malignant plasma cells and the surrounding microenvironment.
The bone marrow microenvironment has been indicated to play a supportive role in migration, proliferation, differentiation and drug resistance of malignant plasma cells.
There is evidence linking the tumour microenvironment to progression of MGUS to active MM, for example significant matrix remodelling has been seen between the bone marrow of healthy individuals, MGUS and MM patients\cite{kawano2015targeting}.
Therefore, to get an accurate picture of MM, information must be acquired about the surrounding niche.

Historically, the tumour environment has been investigated following the isolation of populations of cells sorted from the tumour and then sequenced using traditional microarray or bulk RNA-seq techniques.
Bulk techniques measure the average expression across a sample, which is the sum of cell type specific expression weighted by cell type proportions.
Single-cell techniques measure expression for each individual cell and therefore provide information on clonal diversity that may be lost when pooling cells into bulk samples.
Furthermore, multiple myeloma is an extremely heterogeneous disease, this is seen both between patients and within an individual's own tissue.
Applying single-cell techniques to capture the inter- and intra-individual heterogeneity is fundamental to identifying molecular and cellular signatures that define MM\@.

The advent of single-cell technologies has led to a better understanding of the complexity and diversity of the tumour microenvironment.
Seminal papers from Melnekoff et al. (2017)\cite{melnekoff2017single} and Ledergor et al. (2018)\cite{ledergor2018single} use scRNA-seq to reveal clonal transcriptomic heterogeneity in MM samples.
Melnekoff et al. (2017) demonstrated the clonal heterogeneity within MM using samples that were collected from eight relapsed MM patients.
The group performed t-SNE clustering analysis and the samples separated into eight transcriptionally distinct clones, each corresponding to a different patient.
This highlights the inter-patient differences of MM\@.
Ledergor et al. (2018) performed a similar study to evaluate clonal heterogeneity within MM but also had a set of controls with which to compare the MM group.
They found that MM patients have greater inter-individual transcriptional variation, where each MM patient possessed a unique and individual plasma cell transcriptional program.
They also showed substantial intra-tumour heterogeneity (subclonal structures) of plasma cells in a third of their MM patient cohort.
These papers established the importance of using single-cell techniques to study MM, as to not miss the underlying clonal heterogeneity.
However both of these papers focussed soley on plasma cells and did not look at the surrounding bone marrow microenvironment.
To truly understand the complexities of MM and treatment of MM, interactions between plasma cells and the bone marrow niche must also be explored using single-cell techniques.


\subsection{Experiment overviews}
Three single-cell experiments, comprising samples from four MM patients, were performed to explore the effect of various compounds (including Halofuginone and NCP26) on MM patient tissue.
The BM samples for experiments 1 and 2 were obtained from two treatment-naive, newly-diagnosed MM patients.
Experiment 3 comprised samples from two patients, both with relapsed MM, therefore both presenting with a degree of acquired anti-cancer drug resistance.
For experiment 1, BM samples were treated for 24 hours with 1\si{\micro\Molar} Casin, GSK-J4, Halofuginone, NCP26, SGC-GAK, Verteporfin or a DMSO control, totalling 7 samples.
For experiment 2, BM samples were treated with with 1\si{\micro\Molar} CAMKK2, CLK or CSNK2 for 24 hours; 1\si{\micro\Molar} SGC-GAK, Halofuginone, NCP26 or a DMSO control for 24 and 48 hours, totalling 11 samples.
For experiment 3, BM samples from patient 3 and 4 were treated for 24 hours with either a DMSO control, 1\si{\micro\Molar} Halofuginone, 1\si{\micro\Molar} NCP26 or 5\si{\micro\Molar} NCP26, totalling 8 samples.

Following compound treatment, single-cell RNA-seq library preparation was performed by Dr Martin Philpott using the Chromium Next GEM Single Cell 3` GEM, Library and Gel Bead Kit v3.1 according to the manufacturer`s instructions.
Indexed libraries were quantitated by TapeStation, pooled and sequenced on an Illumina NovaSeq 6000 (Novogene, UK).

<GENEWIZ experiment 3 >


\section{Data processing}
Initially all four patient samples were processed and integrated together.
However, integration was found to be poor between treatment naive patients and the relapsed patients.
This was expected as MM patients` transcriptome has been shown to change considerably following <XYZ> rounds of treatments {REF}.
Therefore, samples from experiments 1 and 2 (treatment naive patients 1 and 2) were integrated together and samples from experiment 3 (relapsed patients 3 and 4) were integrated together.

Experiments 1 and 2 contained treatment samples that were not of interest to this project.
However, all samples originated from MM patients, therefore all 18 samples were included in the analysis up to and including the integration and annotation stage of data processing.
This was to increase the granularity of the data, and allow for easier annotation of clusters.
Downstream analysis in this work only includes DMSO, Halofuginone or NCP26 treated samples.

\subsection{Analysis overview}

The single cell analysis pipeline outlined in section <ENTER section> was used to process the data.
Kallisto BUS/ BUStools was used to pseudoalign reads and quantify gene expression.
Next, quality control and filtering of the samples was performed.
Poor quality cells are likely to have a low number of genes and UMIs per cell.
Any cells with fewer than 500 UMIs were removed.
Cells with a gene count below 300 or above 6000 were removed.
Cells with a mitochondrial ratio higher than 0.1 were removed (a high proportion of mitochondrial genes indicates mitochondrial contamination from dead or dying cells).
After quality control and filtering of the data, clustering was performed using Seurat v3, followed by integration of all samples, using Seurat v3 and Harmony.
The two experiments were too large to integrate across all samples using the Seurat SCTransform normalisation method, therefore a reference-based approach was taken, whereby a subset of samples were selected (based on their cell richness and relevance to the research question) and listed as `reference datasets' for SCTransform normalisation.
Harmony integration was also implemented, using patient and experiment as additional covariates.


Cell type annotation was performed by using several automated packages (singleR, clustifyr and scClassify), then by fine-tuning manually using a list of known biological markers.
The HumanCellAtlas database was used to inform singleR and clustifyr annotation.
scClassify was performed using a pre-trained scClassify model, based on seven PBMC single-cell datasets (including 10X V2, 10X V3, smartSeq, celSeq, dropSeq and inDrops datasets).
As the reference datasets were based on healthy tissue, they are unable to label pathological cells, like myeloma cells.
Myeloma cells were identified manually using a range of known markers, for example: CD38, CD138, SLAMF7 and BCMA (see table \ref{tab:annotation_markers}).

% Pre RBC dimplots
\begin{figure}[htb]
    \centering
    \includegraphics[width=\textwidth]{figures/Results/single_cell/data_processing/RBC_removal_all_experiments.pdf}
    \caption[Erythrocyte removal from integrated scRNA-seq datasets]{UMAP dimension plots following integration of samples from experiment 1 and 2 (treatment naive patients), and samples from patients 3 and 4 in experiment 3 (relapsed MM).
    [A-D] Experiment 1 and 2-newly diagnosed MM patients.
    [E-H] Experiment 3, patients 3 and 4- relapsed MM patients.
     [A, B, E, F] Integrated UMAP plots before erythrocyte cell and gene removal. Erythrocyte clusters are circled in A) and E).
     [C, D, G, H] UMAP plots following removal of erythrocyte cell clusters and genes and re-integration of samples.
     [B, D, F, H] show the composition of each dataset by experiment or patient.}
    \label{fig:umap_RBC}
\end{figure}

Experiment 2 was found to have an extremely high erythrocyte population (figure \ref{fig:umap_RBC}A and B).
In addition, many other cell populations were expressing erythrocyte specific genes, where we would not expect to see them being expressed, for example MM cells expressing numerous haemoglobin subunit genes.
Many of the variable features that Seurat uses for clustering analysis and dimension reduction were made up of these erythrocyte-specific and haemoglobin genes.
The high expression of these genes was thought to be affecting the integration of the two experiments together.
A theory for the presence of the large number of erythrocytes and un-localised erythrocyte gene expression is that perhaps the BM sample taken for experiment 2 was one of the later samples taken from the patient and contained a large amount of blood.
Library prep clean-up may have missed many of these cells, leading to ambient erythrocyte RNA being present within many droplets.

It was decided to remove the erythrocyte clusters (clusters 3, 8, 9, 10, 11 and 16 in newly diagnosed; clusters 1, 5 and 15 in relapsed) and haemoglobin related genes or erythrocyte specific genes that seemed to be dominating expression in the integrated dataset.
After the integrated Seurat object had the erythrocyte genes and cells removed, it was split back up into separate Seurat objects for each sample, and integration and clustering was performed again.
Seurat`s SCTransform with reference datasets and Harmony (using a multi-covariate model, accounting for each different sample and the two different experiments) were used to re-integrate all samples.
Harmony integration was found to integrate clusters across patients and experiments better than using Seurat`s SCTransform.
The Harmony integrated datasets were taken forwards and used for cell type annotation.
Better integration was achieved after removing erythrocytes and erythrocyte-specific genes (see figure \ref{fig:umap_RBC}D).

A large erythrocyte component was also found for patient 4 for in experiment 3 (figure \ref{fig:umap_RBC}E and F).
The same analysis workflow was applied to experiment 3, removing the erythrocyte cluster and erythrocyte specific genes and re-integrating using Harmony with samples and patients as covariates.

% table cells passing filter
\input{text/Chapter_6_single-cell_PRS/cells_passing_table.tex}

\subsection{Annotation of re-integrated data}


% automated annoation, scclasify, clustifyr
\begin{figure}[htb]
    \centering
    \includegraphics[width=\textwidth]{figures/Results/single_cell/data_processing/automated_markers_ABCD.pdf}
    \caption[Automated annotation of scRNA-seq data]{Automated annotation of MM cell clusters, using the R packages clustifyr and scClassify in combination with reference datasets.
    [A, B] newly diagnosed MM, [C, D] relapsed MM patients.
    The output of automated packages clustifyr and scClassify used to aid cell type annotation.
    Clustifyr assigns a cell type to each cluster of cells, scClassify assigns a cell type to each individual cell (labels have been added post-hoc).
    Clustifyr was used in conjunction with the HumanCellAtlas reference, and scClassify was ran with a pretrained model trained on seven PBMC single cell RNA-seq datasets.
    Both references are from healthy datasets so neither are able to identify the myeloma cell population.}
    \label{fig:annotation_automated}
\end{figure}

R packages clustifyr and scClassify were used to aid in cell type annotation of the integrated datasets, the result of this annotation can be seen in figure \ref{fig:annotation_automated}.
Because healthy tissue datasets were used as references, the myeloma cluster could not be identified using these packages with these references alone.
Both packages either incorrectly labelled the MM cell cluster as B cells, or were unable to assign any cell type to the myeloma cluster with any confidence and left it unassigned.
However, the packages do give a good starting point for more detailed manual annotation using known biological markers.


Figures \ref{fig:mm_markers_naive} and \ref{fig:mm_markers_relapsed} show how the MM cell clusters were identified.
From MM cells, you would expect to see expression of CD38, but lower expression than in normal plasma cells.
You would expect to see high expression of the pathological marker CD138 in MM cells, as well as high expression of SLAMF7, BCMA, KRAS, IGKC and IGL2, however these are not exclusive to the MM cluster.
You would expect to see little or no expression of CD45 and CD19 in the MM cluster and reduced expression of CD20 in MM cells compared to normal B cells.
Using the expression of these markers, clusters 2, 7 and 13 were identified as the MM cell population in the newly diagnosed dataset; and cluster 4 was identified as the MM cell population in the relapsed MM dataset.


% MM markers vln, feature and dotplots
\begin{figure}[p]
    \centering
    \includegraphics[width=\textwidth]{figures/Results/single_cell/data_processing/markers_naive.pdf}
    \caption[MM cluster manual annotation- newly diagnosed MM]{Manual annotation of MM cell clusters in experiment 1 and 2 (newly diganosed MM) using known MM biological markers.
    A) Seurat UMAP feature plots showing expression for MM markers.
    Purple indicates expression of the corresponding gene by a cell.
    B) A dot plot showing the percentage of cells in a cluster expressing a given gene, and the average expression of that gene for the cluster.
    C) Violin plots for each cluster and expression levels for each marker listed above.  }
    \label{fig:mm_markers_naive}
\end{figure}

% MM markers vln, feature and dotplots
\begin{figure}[p]
    \centering
    \includegraphics[width=\textwidth]{figures/Results/single_cell/data_processing/markers_relapsed.pdf}
    \caption[MM cluster manual annotation- relapsed MM]{Manual annotation of MM cell clusters in experiment 3 (relapsed MM) using known biological markers
    A) Seurat UMAP feature plots showing expression for MM markers.
    Purple indicates expression of the corresponding gene by a cell.
    B) A dot plot showing the percentage of cells in a cluster expressing a given gene, and the average expression of that gene for the cluster.
    C) Violin plots for each cluster and expression levels for each marker listed above. }
    \label{fig:mm_markers_relapsed}
\end{figure}


\input{text/Chapter_6_single-cell_PRS/markers_table.tex}

% T / NK cells feature plots

% Use of annotation for new MM datasets

% flush figures out before beginning results section
\clearpage
\section{Results}

\subsection{Newly diagnosed MM}

% Final product of annotation

%% Final annotation
\begin{figure}[hpt]
\centering
\includegraphics[width=\textwidth]{figures/Results/single_cell/naive_full_annotation.pdf}
\caption[Newly-diagnosed MM scRNA-seq full annotation]{Fully annotated UMAP clustering analysis of two newly-diagnosed multiple myeloma (MM) patients.
The MM cell population (circled), consists of three distinct clusters.}
\label{fig:full_anno_naive}
\end{figure}
%%

Figure \ref{fig:full_anno_naive} shows final cell-type annotation for the integrated newly-diagnosed MM patients.
18 distinct clusters were identified with Seurat embeddings.
The expected major immune clusters were identified (such as B cells, T cells and myeloid cells.)
Using the established MM biological markers (shown in figure \ref{fig:mm_markers_naive}), three distinct MM clusters (2, 7 and 13) were identified.

\subsubsection{Composition}
Cluster composition analysis by treatment conditions is shown in figure \ref{fig:composition_naive}.
Halofuginone treatment at both 24 and 48 hours reduced the proportion of cells in the MM cluster (p<0.00001) compared to DMSO.
NCP26 treatment at 24 hours reduced the proportion of cells in the MM cluster (p<0.00001).
Together with dose response curves and cell death assays, this suggests that Halofuginone and NCP26 are selectively killing MM cells to a higher degree than other cell types.

%% SCT UMAP cluster composition separated by treatment condition
\begin{figure}[htb]
\centering
\includegraphics[width=0.7\textwidth]{figures/Results/single_cell/sc_composition_naive.pdf}
\caption[scRNA-seq composition analysis- newly diagnosed MM]{Composition analysis of newly diagnosed MM.
    A) UMAP cell composition plots separated by treatment condition.
    B) Dot plot showing proportion of cells in each cluster for each sample.
    C) Dot plot showing proportion of cells in each cell class for each sample (as labelled in Figure <XYZ>).
    * indicates samples originating from experiment 1, no star indicates an experiment 2 origin.}
\label{fig:composition_naive}
\end{figure}
%%

Halofuginone treatment and NCP26 48h treatment were also found to reduce the proportion of cells in the monocyte cluster (p<0.00001), indicating they may have some off-target effects on myeloid cells.

All of the compounds were used at a concentration of 1\si{\micro\Molar}.
This is approximately 10 times the concentration of Halofuginone's IC50 value on MM cell lines and two times NCP26's IC50 value...

Large number of cells were killed by Halofuginone treatment at both 24 and 48 hours.
Clusters 0, 1 and 5 (T-cells, monocytes and MM cells) seem particularly affected.

\subsubsection{Differential expression}
% Effect on MM cells
% Number of DE genes, pathways enriched. VLN plts - CD138 and DDIT3. AAR

% Effect on other cell populations- e.g. monocytes, why does it affect monocytes

% Velocity analysis


\subsection{Relapsed MM}


\section{Myeloma bone marrow classifier}