\chapter{\label{ch:6-sc}Single-cell RNA-seq analysis of ProRS inhibitors}

%\minitoc

\section{Introduction}
MM cells grow within the bone marrow and are supported as they grow by their microenvironment.
The MM microenvironment comprises a cellular compartment (composed of immune cells, endothelial cells, osteoblasts, osteoclasts and stromal cells) and a non-cellular compartment (composed of the extracellular matrix (ECM), cytokines, chemokines and growth factors)\cite{manier2012bone, kawano2015targeting}.
There are interactions between malignant plasma cells and the surrounding microenvironment.
The bone marrow microenvironment has been indicated to play a supportive role in migration, proliferation, differentiation and drug resistance of malignant plasma cells.
There is evidence linking the tumour microenvironment to progression of MGUS to active MM, for example significant matrix remodelling has been seen between the bone marrow of healthy individuals, MGUS and MM patients\cite{kawano2015targeting}.
Therefore, to get an accurate picture of MM, information must be acquired about the surrounding niche.

Historically, the tumour environment has been investigated following the isolation of populations of cells sorted from the tumour and then sequenced using traditional microarray or bulk RNA-seq techniques.
Bulk techniques measure the average expression across a sample, which is the sum of cell type specific expression weighted by cell type proportions.
Single-cell techniques measure expression for each individual cell and therefore provide information on clonal diversity that may be lost when pooling cells into bulk samples.
Furthermore, multiple myeloma is an extremely heterogeneous disease, this is seen both between patients and within an individual's own tissue.
Applying single-cell techniques to capture the inter- and intra-individual heterogeneity is fundamental to identifying molecular and cellular signatures that define MM\@.

The advent of single-cell technologies has led to a better understanding of the complexity and diversity of the tumour microenvironment.
Seminal papers from Melnekoff et al. (2017)\cite{melnekoff2017single} and Ledergor et al. (2018)\cite{ledergor2018single} use scRNA-seq to reveal clonal transcriptomic heterogeneity in MM samples.
Melnekoff et al. (2017) demonstrated the clonal heterogeneity within MM using samples that were collected from eight relapsed MM patients.
The group performed t-SNE clustering analysis and the samples separated into eight transcriptionally distinct clones, each corresponding to a different patient.
This highlights the inter-patient differences of MM\@.
Ledergor et al. (2018) performed a similar study to evaluate clonal heterogeneity within MM but also had a set of controls with which to compare the MM group.
They found that MM patients have greater inter-individual transcriptional variation, where each MM patient possessed a unique and individual plasma cell transcriptional program.
They also showed substantial intra-tumour heterogeneity (subclonal structures) of plasma cells in a third of their MM patient cohort.
These papers established the importance of using single-cell techniques to study MM, as to not miss the underlying clonal heterogeneity.
However both of these papers focussed soley on plasma cells and did not look at the surrounding bone marrow microenvironment.
To truly understand the complexities of MM and treatment of MM, interactions between plasma cells and the bone marrow niche must also be explored using single-cell techniques.

This chapter uses MM patient-derived bone marrow (BM), from both newly-diagnosed and relapsed patients, to investigate the transcriptional effects of ProRS on MM cells and their surrounding immune microenvironment.

\subsection{Experiment overviews}\label{subsec:scrna_experiment}
Three single-cell experiments, comprising samples from four MM patients, were performed to explore the effect of various compounds (including Halofuginone and NCP26) on MM patient tissue.
The BM samples for experiments 1 and 2 were obtained from two treatment-naive, newly-diagnosed MM patients.
Experiment 3 comprised samples from two relapsed MM patients, therefore both presenting with a degree of acquired anti-cancer drug resistance.
For experiment 1, BM samples were treated for 24 hours with 1\si{\micro\Molar} Casin, GSK-J4, Halofuginone, NCP26, SGC-GAK, Verteporfin or a DMSO control, totalling 7 samples.
For experiment 2, BM samples were treated with with 1\si{\micro\Molar} CAMKK2, CLK or CSNK2 for 24 hours; 1\si{\micro\Molar} SGC-GAK, Halofuginone, NCP26 or a DMSO control for 24 and 48 hours, totalling 11 samples.
For experiment 3, BM samples from patient 3 and 4 were treated for 24 hours with either a DMSO control, 1\si{\micro\Molar} Halofuginone, 1\si{\micro\Molar} NCP26 or 5\si{\micro\Molar} NCP26, totalling 8 samples.

Following compound treatment, scRNA-seq library preparation was performed by Dr Martin Philpott as outlined in section \ref{subsec:10x_method}.
<GENEWIZ experiment 3 >


\section{Data processing}
Initially all four patient samples were processed and integrated together.
However, integration was found to be poor between treatment-naive patients and the relapsed patients.
This was expected as MM patients' transcriptome has been shown to change considerably following successive rounds of treatment cycles.
Therefore, samples from experiments 1 and 2 (treatment-naive patients 1 and 2) were integrated together and samples from experiment 3 (relapsed patients 3 and 4) were integrated together.

Experiments 1 and 2 contained numerous samples that were not of interest to this project.
Yet, all samples originated from MM patient tissue, therefore all 18 samples were included in the analysis pipeline, and were included in integration and annotation steps.
These irrelevant samples were included to increase the granularity of the data, and allow for easier annotation of clusters.
Downstream analyses was only performed for DMSO, Halofuginone and NCP26 samples.

\subsection{Analysis overview}

The single cell analysis pipeline outlined in Section \ref{subsec:updated_scrna} was used to process the scRNA-seq data.
Kallisto BUS/ BUStools was used to pseudoalign reads and quantify gene expression.
Next, quality control and filtering of the samples was performed.
Poor quality cells are likely to have a low number of genes and UMIs per cell.
So any cells with fewer than 500 UMIs were removed.
Cells with a gene count below 300 or above 6000 were also removed.
Cells with a mitochondrial ratio higher than 0.1 were removed (a high proportion of mitochondrial genes indicates mitochondrial contamination from dead or dying cells).

After quality control and filtering of the data, clustering was performed using Seurat v3, followed by integration of all samples, using Seurat and Harmony.
Using the Seurat SCTransform normalisation method of integration, the two experiments were too large to integrate across all samples.
Therefore, a reference-based approach was taken, whereby a subset of samples were selected (based on their cell richness and relevance to the research question) and listed as `reference datasets' for SCTransform normalisation.
Harmony integration was also implemented, using `Patient Number' and `Experiment Number' as additional covariates.

Cell type annotation was performed with several automated packages (singleR, clustifyr and scClassify), and then fine-tuning manually using a list of known biological markers.
The HumanCellAtlas (HCA) database was used to inform singleR and clustifyr annotation.
scClassify was performed using a pre-trained scClassify model, based on seven peripheral blood mononuclear cell (PBMC) scRNA-seq datasets (including data from 10Xv2, 10Xv3, smartSeq, celSeq, dropSeq and inDrops).
As the reference datasets were based on healthy tissue, they were unable to label pathological cells, like MM cells.
Myeloma cells were identified manually using a range of known markers, for example: \textit{CD38}, \textit{CD138}, \textit{SLAMF7} and \textit{BCMA} (see Table \ref{tab:annotation_markers}).

% Pre RBC dimplots
\begin{figure}[htb]
    \centering
    \includegraphics[width=\textwidth]{figures/Results/single_cell/data_processing/RBC_removal_all_experiments.pdf}
    \caption[Erythrocyte removal from integrated scRNA-seq datasets]{UMAP dimension plots following integration of samples from experiment 1 and 2 (treatment naive patients), and samples from patients 3 and 4 in experiment 3 (relapsed MM).
    [a-d] Experiment 1 and 2-newly diagnosed MM patients.
    [e-h] Experiment 3, patients 3 and 4- relapsed MM patients.
     [a, b, e, f] Integrated UMAP plots before erythrocyte cell and gene removal. Erythrocyte clusters are circled in a) and e).
     [c, d, g, h] UMAP plots following removal of erythrocyte cell clusters and genes and re-integration of samples.
     [b, d, f, h] show the composition of each dataset by experiment or patient.}
    \label{fig:umap_RBC}
\end{figure}

Experiment 2 was found to have an extremely high erythrocyte population (Figure \ref{fig:umap_RBC}a and b).
In addition, many other cell populations were expressing erythrocyte-specific genes, where we would not expect to see them expressed, for example MM cells expressing numerous haemoglobin subunit genes.
Many of the variable features that Seurat uses for clustering and dimension reduction were made up of these erythrocyte-specific and haemoglobin genes.
The high expression of these genes was affecting the integration of the two experiments together.
A theory for the presence of the large number of erythrocytes and un-localised erythrocyte gene expression is that perhaps the BM sample taken for experiment 2 was one of the later samples taken from the patient and contained a large amount of blood.
Library prep clean-up may have missed many of these cells, leading to ambient erythrocyte RNA being present within many droplets.

It was decided to remove the erythrocyte clusters (clusters 3, 8, 9, 10, 11 and 16 in newly-diagnosed; clusters 1, 5 and 15 in relapsed) and haemoglobin-related genes or erythrocyte-specific genes that were dominating expression in the integrated dataset.
After the integrated Seurat object had the erythrocyte genes and cells removed, it was split back up into separate Seurat objects for each sample, and integration and clustering was performed again.
Seurat's SCTransform with reference datasets and Harmony (using a multi-covariate model, accounting for each different sample and the two different experiments) were used to re-integrate all samples.
Harmony integration was found to integrate clusters across patients and experiments better than using Seurat's SCTransform.
The Harmony-integrated datasets were taken forwards and used for cell type annotation and further downstream analyses.
Better integration was achieved after removing erythrocytes and erythrocyte-specific genes (see Figure \ref{fig:umap_RBC}d).

A large erythrocyte component was also found for patient 4 in the relapsed dataset (Figures \ref{fig:umap_RBC}e and \ref{fig:umap_RBC}f).
The same analysis workflow outlined above was applied to experiment 3, removing the erythrocyte cluster and erythrocyte-specific genes and re-integrating using Harmony with samples and patients as covariates.

The total number of cells after each data processing step for each patient and experiment is summarised in Table \ref{tab:cells_passing}.
\input{text/Chapter_6_single-cell_PRS/cells_passing_table.tex}
%%%
The number of cells originating from patient 3 after filtering might be a barrier to interpreting meaningful results.
Patients 1, 2 and 4 have sufficient cell numbers to perform downstream analysis, for example differential expression.

\subsection{Annotation of re-integrated data}\label{subsec:sc_annotate}
% automated annoation, scclasify, clustifyr
R packages clustifyr and scClassify were used to aid in cell type annotation of the integrated datasets, the result of this annotation can be seen in Figure \ref{fig:annotation_automated}.
%%
%%% Figure automated package annotation
\begin{figure}[htb]
    \centering
    \includegraphics[width=\textwidth]{figures/Results/single_cell/data_processing/automated_markers_ABCD.pdf}
    \caption[Automated annotation of scRNA-seq data]{Automated annotation of scRNA-seq cell clusters, using the R packages clustifyr and scClassify in combination with reference datasets.
    [a, b] newly-diagnosed MM, [c, d] relapsed MM.
    The output of automated packages clustifyr and scClassify is used to aid cell type annotation.
    Clustifyr assigns a cell type to each cell cluster, whilst scClassify assigns a cell type to each individual cell.
    Clustifyr was used in conjunction with the HumanCellAtlas reference.
    scClassify was ran using a pre-trained model trained on seven PBMC single cell RNA-seq datasets.
    Both references originate from healthy data, therefore neither are able to identify the myeloma cell population.}
    \label{fig:annotation_automated}
\end{figure}

Automated annotation using computational methods gives a good starting point for more detailed manual annotation.
Currently, these methods are not a complete substitute for manual annotation using biological knowledge of cell markers.
Both cell-type references originate from healthy tissue, therefore the myeloma cluster could not be identified using clustifyr, scClassify and these healthy references alone.
Both packages either incorrectly labelled the MM cluster as B cells, or were unable to assign any cell type to the myeloma cluster with any confidence and labelled it unassigned.
Thus, the MM clusters had to be identified manually using MM biological knowledge.
Section \ref{sec:MM_classifier} constructs a MM classifier reference to overcome this problem for future MM scRNA-seq datasets.

Figures \ref{fig:mm_markers_naive} and \ref{fig:mm_markers_relapsed} illustrate MM cell identification for the naive and relapsed datasets, respectively, using gene expression of certain MM markers.
% MM markers vln, feature and dotplots
\begin{figure}[p]
    \centering
    \includegraphics[width=\textwidth]{figures/Results/single_cell/data_processing/markers_naive.pdf}
    \caption[MM cluster manual annotation- newly diagnosed MM]{Manual annotation of multiple myeloma (MM) cell clusters in experiment 1 and 2 (newly-diagnosed patients) using known MM biological markers.
    a) UMAP feature plots of MM marker expression.
    Grey indicates no expression and purple indicates expression.
    b) A dot plot of the percentage of cells expressing a given marker and the average expression (for each cluster).
    c) Violin plots of gene expression for each cluster.}
    \label{fig:mm_markers_naive}
\end{figure}
%
% MM markers vln, feature and dotplots
\begin{figure}[p]
    \centering
    \includegraphics[width=\textwidth]{figures/Results/single_cell/data_processing/markers_relapsed2.pdf}
    \caption[MM cluster manual annotation- relapsed MM]{Manual annotation of MM cell clusters in experiment 3 (relapsed MM) using known biological markers.
    a) UMAP feature plots of MM marker expression.
    b) A dot plot of the percentage of cells expressing a given marker and the average expression (for each cluster).
    c) Violin plots of gene expression for each cluster. }
    \label{fig:mm_markers_relapsed}
\end{figure}
%
For MM cells, you would expect to see: expression of \textit{CD38}, but lower expression than in normal plasma cells;
high expression of \textit{CD138} in MM cells, as well as high expression of \textit{SLAMF7}, \textit{BCMA}, \textit{KRAS}, \textit{IGKC} and \textit{IGL2}, however these are not exclusive to the MM cluster.
You would expect to see little or no expression of \textit{CD45} and \textit{CD19} in the MM cluster and reduced expression of \textit{CD20} in MM cells compared to normal B cells.
You may also expect to see expression of \textit{CD56} in abnormal plasma cells, however this is not always detected at the RNA level.
Other common markers used to identify MM cells include, \textit{CD81}, \textit{B2M} and \textit{CD117}.
Using the expression of a combination of these markers, clusters 2, 7 and 13 were identified as the MM cell population in the newly diagnosed dataset and cluster 4 was identified as MM cells in the relapsed MM dataset.

However, from marker expression alone, the identity of clusters 9 and 12 in the relapsed dataset were unclear.
Both clusters 9 and 12 seem to belong to a B-cell lineage.
Cluster 12 shows increased \textit{CD20} expression, indicating they could be mature B cells.
However, clusters 4 and 9 have an increased expression of \textit{IGKC}, compared to non-B cell clusters.
Indicating that perhaps, all B-cell clusters (4, 9 and 12) in the relapsed dataset, may be in fact MM cells.
It has previously been shown that MM possess a level of plasticity, that could allow de-differentiation into other cell lineages \cite{kotouvcek2014myeloma}, this may account for the \textit{CD20} expression of cluster 12.

Due to the uncertainty of cell-type identity in the relapsed dataset based on MM marker expression alone, inferCNV \cite{patel2014single, infercnv2014} was employed to detect large-scale chromosomal copy number variations (CNVs) and help inform annotation.
The raw counts expression matrix of the relapsed, integrated Seurat object was used as input for inferCNV\@.
`Normal' cells (that is: myeloid cells, T cells and NK cells) were used as references for for the relative expression intensity of the suspected malignant cell clusters (clusters 4, 9 and 12).
A Cutoff value of 0.1 was used for the analysis, to account for the sparsity of droplet-based 3' scRNA-sequencing.
%
% inferCNV
\begin{figure}[htb]
    \centering
    \includegraphics[width=\textwidth]{figures/Results/single_cell/data_processing/inferCNV_relapse.pdf}
    \caption[inferCNV- relapsed MM]{InferCNV results for the relapsed MM dataset.
    [a and b] InferCNV heatmaps.
        The top panel shows expression values for the reference `normal' cells.
        The bottom panel shows expression values for the suspected malignant cells (clusters 4, 9 and 12).
        Red indicates chromosomal region amplifications and blue indicates chromosomal region deletions.
    a) De-noised inferCNV results.
    b) Hidden Markov-Model (HMM) copy number variation (CNV) region predictions.
        Chromosomal deletions predicted for chromosome 13 and chromosomal duplications/gains predicted in chromosomes 11 and 19 for clusters 4 and 9.
        Gains also predicted in chromosome 3 and 9 for cluster 4.
    c) UMAP featureplots demonstrating chromosome 13 loss and chromosomes 3, 9 and 11 duplications.
    }
    \label{fig:inferCNV_relapse}
\end{figure}
%
Figure \ref{fig:inferCNV_relapse} shows the inferCNV results for the relapsed dataset.
Myeloma-defining CNVs were seen in both cluster 4 and cluster 9, including chromosome 13 deletion and duplications of chromosomes 11 and 19.
The inferCNV results also demonstrate that there are different genetic MM subclones within the population, as cluster 4 alone shows regional gain of chromosomes 3, 7, 9 and 16.
This fits with the <FISH - ask martin for ORBID> data for the relapsed patients, whereby they had chromosome 13 deletions...
Therefore, it can be concluded that clusters 4 and 9 are subclonal MM populations.

InferCNV was also used on the naive dataset (Figure \ref{fig:inferCNV_naive}) to check for CNVs.
The inferCNV results confirmed the gene expression-based identification of clusters 2, 7 and 13 as the MM cell population.

% appendix annotation for other clusters - ?
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% Results
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% flush figures out before beginning results section
\clearpage
\section{Results}

\subsection{Newly-diagnosed MM}

% Final product of annotation

%% Final annotation
\begin{figure}[hpt]
\centering
\includegraphics[width=\textwidth]{figures/Results/single_cell/naive_full_annotation.pdf}
\caption[Newly-diagnosed MM scRNA-seq full annotation]{Fully annotated UMAP clustering analysis of two newly-diagnosed multiple myeloma (MM) patients.
The MM cell population (circled), consists of three distinct clusters.}
\label{fig:full_anno_naive}
\end{figure}
%%

Figure \ref{fig:full_anno_naive} shows final cell-type annotation for the newly-diagnosed MM integrated dataset.
18 distinct clusters were identified using Seurat embeddings.
The expected major immune clusters were identified (such as B cells, T cells, NK cells and myeloid cells.)
Using the established MM biological markers (shown in Figure \ref{fig:mm_markers_naive}), three distinct MM clusters (2, 7 and 13) were identified.

\subsubsection{Composition}
Cluster composition analysis by treatment condition is shown in Figure \ref{fig:composition_naive}.
Halofuginone treatment at both 24 and 48 hours reduced the proportion of cells in the MM cluster (p<0.00001) compared to DMSO, for both experiment 1 and 2.
NCP26 treatment at 24 hours reduced the proportion of cells in the MM cluster (p<0.00001), for both experiment 1 and 2.
HF and NCP26 treatment increased the proportion of cells in the CD4+ T cell, CD8+ T cell and B cell clusters.
Together with dose response curves and cell death assays, this suggests that Halofuginone and NCP26 are selectively killing MM cells to a higher degree than other cell types.

%%  UMAP cluster composition separated by treatment condition
\begin{figure}[htb]
\centering
\includegraphics[width=\textwidth]{figures/Results/single_cell/sc_composition_naive2_barchart.pdf}
\caption[scRNA-seq composition analysis- newly diagnosed MM]{Composition analysis of newly diagnosed MM.
    a) UMAP cell composition plots separated by treatment condition.
    b) Dot plot showing proportion of cells in each cluster for each sample.
    c) Dot plot showing proportion of cells in each cell class for each sample (as labelled in Figure \ref{fig:full_anno_naive}).
    d) The proportion of cells in the MM cluster only (stars above bars indicate significant at p<0.01 compared to corresponding control).
    Sample names starting with asterisks originate from experiment 1, no asterisk indicates experiment 2 origin.}
\label{fig:composition_naive}
\end{figure}
%%

Halofuginone treatment and 48-hour NCP26 treatment were also found to reduce the proportion of cells in the monocyte and neutrophil cluster (p<0.00001), indicating they may have some off-target effects on myeloid cells.

All of the compounds were used at a concentration of 1\si{\micro\Molar}.
This is approximately 10 times the concentration of Halofuginone's IC\textsubscript{50} value on MM cell lines and two times NCP26's IC\textsubscript{50} value.
As you can see, large number of cells were killed by Halofuginone treatment at both 24 and 48 hours.
Therefore, the effects seen on the myeloid cells could be due to high dosing of the ProRS inhibitors.

\subsubsection{Differential expression}
% Number of DE genes
% separate into NCP26 and HF??
Next, differential gene expression was investigated using Seurat's FindMarkers function.
At the 48 hour time point substantial cell death is seen, therefore the 24-hour time point will be mainly focussed on.
%
%%  DE barchart
\begin{figure}[htb]
\centering
\includegraphics[width=0.8\textwidth]{figures/Results/single_cell/naive_DEGs_barchart.pdf}
\caption[scRNA-seq DEGs per cell type- newly-diagnosed MM]{Number of differentially expressed genes (DEGs; p\textsubscript{adj}< 0.05) broken down by cell type for two newly-diagnosed MM patients treated with ProRS inhibitors (Halofuginone and NCP26) for 24 hours.
Cell type annotation corresponding to Figure \ref{fig:full_anno_naive}.
Experiment 1 and experiment 2 denote separate experiments, each containing BM samples from different newly-diagnosed MM patients.}
\label{fig:naive_deg_bar}
\end{figure}

Following 24-hour 1\si{\micro\Molar} NCP26 treatment, 1515 genes were differentially expressed (DE; p\textsubscript{adj}< 0.05) in the myeloma cell population in experiment 1, and 1294 genes in experiment 2, compared to DMSO treatment.
Figure \ref{fig:naive_deg_bar} shows the breakdown of DEGs per cell type for NCP26 and HF treatment.
24 hour 1\si{\micro\Molar} NCP26 treatment had very little transcriptional effect on many of the other immune cell types, for example T, B and NK cells, where the number of DEGs is an order of magnitude less than for MM cells.
This corroborates the composition analysis, where MM cells seem more sensitive to ProRS inhibition than other immune cell types.
However, 2016 and 1485 genes were DE in the monocyte cluster following NCP26 treatment in experiment 1 and 2, respectively.
This reflects the composition analysis, where the proportion of cells in the monocyte cluster was markedly reduced.
This could indicate that NCP26 is not selective for MM cells over myeloid cells.

In MM cells, 73 genes were DE following Halofuginone treatment in experiment 1, and 1318 genes in experiment 2.
This reflects results seen in the composition analysis (Figure \ref{fig:composition_naive}), whereby we saw very few cells remaining in the MM cluster for experiment 1.
Due to the low cell number, there is less statistical power and diversity across the cells, therefore you would likely see fewer statistically significant DEGs for this cluster.
Unlike NCP26 treatment, for 24 hour 1\si{\micro\Molar} HF treatment we see a large number of DEGs in B, T and NK cell clusters.
This likely reflects the differences in potency between NCP26 and HF\@.
HF is approximately five times more potent than NCP26.
This fits with composition analysis, where HF showed a great deal of cell killing in the myeloid and MM cluster.

At lower doses (i.e. NCP26 at 1\si{\micro\Molar}) of ProRS inhibition we see clear evidence of greater transcriptional effects on MM cells and monocytes over other immune subtypes.
However at higher doses (i.e. Halofuginone at 1\si{\micro\Molar}) we see substantial cell killing of MM cells and monocytes and larger transcriptional effects on other immune subtypes.
This together with the composition analysis, demonstrates that NCP26 and Halofuginone are selective for MM cells over most immune cells, except for myeloid cells.
However, this could be a problem with the doses used for treatment.
1\si{\micro\Molar} is almost 10 times greater than Halofuginone's IC\textsubscript{50} and two times greater than NCP26's IC\textsubscript{50}.
This experiment should be performed at a lower concentration to ascertain if NCP26 and Halofuginone are more selective for MM cells or monocytes.
Ideally, the experiment would be performed over the course of three days at a lower concentration, as in our cell line studies.
However, human bone marrow samples do not last very well in extended culture, therefore acute treatment was the preferred method.


\subsubsection{Pathway analysis}
Pathway enrichment analysis was performed for the top 200 upregulated and top 200 downregulated DEGs in the MM cluster, following Halofuginone and NCP26 treatment (Figure \ref{fig:naive_mm_pathway_analysis}).
%%  SC MM pathway analysis-- might need to up font size
\begin{figure}[htb]
\centering
\includegraphics[width=\textwidth]{figures/Results/single_cell/naive_mm_pathway_analysis.pdf}
\caption[scRNA-seq MM cluster pathway analysis (newly-diagnosed MM)]{Pathway analysis (Gene ontology biological processes; GOBP) of the MM cluster.
    a) and b) Halofuginone treatment.
    c) and d) NCP26 treatment.
a) and c) GOBP pathway analysis performed using top 200 upregulated DEGs (p\textsubscript{adj} < 0.05 and $\log_{2FC}>0$) ranked by fold change.
b) and d) GOBP pathway analysis performed using top 200 downregulated DEGs (p\textsubscript{adj} < 0.05 and $\log_{2FC}<0$) ranked by fold change.}
\label{fig:naive_mm_pathway_analysis}
\end{figure}
%%

For NCP26 and Halofuginone treatment of MM cells, pathways related to ER-stress and apoptosis are enriched.
The unfolded protein response- another member of the ISR, which shares many effectors with AAR- is also enriched.
This supports our bulk RNA-seq results that the AAR response is activated following NCP26 and Halofuginone treatment of MM cells.

Figure \ref{fig:naive_aar_vln_ftp} shows a more detailed exploration of AAR genes and their expression in MM cells, as well as the immune microenvironment.
%
%%  AAR violin and feature plots
\begin{figure}[htb]
\centering
\includegraphics[width=\textwidth]{figures/Results/single_cell/naive_aar_vln_ftp.pdf}
\caption[scRNA-seq differentially expressed AAR genes- newly diagnosed patients]{A selection of differentially expressed amino acid starvation response (AAR) genes following 24 hour treatment with ProRS inhibitors (newly-diagnosed MM patients).
    a) Violin plots showing expression of selected AAR genes in the MM population following DMSO, Halofuginone and NCP26 treatment.
    b) Feature plots of UMAP clustering, showing gene expression of AAR genes \textit{ATF3}, \textit{GADD34}, \textit{DDIT3} and \textit{TRIB3}.
Feature plots are split into three panels based on treatment condition (DMSO, Halofuginone and NCP26).
The myeloma cell population is circled in the first panel.}
\label{fig:naive_aar_vln_ftp}
\end{figure}
%%
Figure \ref{fig:naive_aar_vln_ftp}a shows increased expression of \textit{ATF4} target genes, such as \textit{ATF3}, \textit{DDIT3} and  \textit{GADD34}, following HF and NCP26 treatment.
tRNA aminoacyl synthetase genes (\textit{WARS1} and \textit{SARS1}) are also over-expressed in MM cells following ProRS inhibitor treatment.
Figure \ref{fig:naive_aar_vln_ftp}b shows selected AAR genes' expression for the entire UMAP clustering plot, separated by treatment condition.
Markedly increased expression of \textit{ATF3} and \textit{TRIB3} can be seen localised to MM cells and monocytes for HF and NCP26 treatment.
Increased expression of \textit{DDIT3} and \textit{GADD34} can be seen across a few different clusters, however it is most pronounced in the MM and monocyte clusters.
Additionally, the apoptotic mediator \textit{TNFRSF10B}, positively regulated by DDIT3, is upregulated in MM cells and monocytes (Figure \ref{fig:naive_path_vln_ftp}b).
Indicating that, although aaRSs are ubiquitous enzymes and ProRS inhibitors cause some degree of activation of the AAR in all cells, they are cytotoxically more selective for MM cells, and lead to an apoptotic cascade of events, preferentially over other cell types.
\textit{CDKN1A} is also overexpressed in MM cells following NCP26 and HF treatment.
\textit{CDKN1A} is a target gene of \textit{ATF4} and arrests cell cycle progression by inhibiting the activity of cyclin-dependent kinases.
HF has previously been show to cause cell cycle arrest and accumulating cells in the G\textsubscript{0}/G\textsubscript{1} phase of cell cycle.
%
%%  other markers violin and feature plots
\begin{figure}[htb]
\centering
\includegraphics[width=\textwidth]{figures/Results/single_cell/naive_pathological_vln_ftp.pdf}
\caption[scRNA-seq differentially expressed MM markers- newly diagnosed patients]{A selection of differentially expressed multiple myeloma (MM) and cell cycle/apoptotic markers in newly diagnosed patients treated with ProRS inhibitors for 24 hours.
    a) Violin plots showing expression of selected genes in the MM population following DMSO, Halofuginone and NCP26 treatment.
    b) Feature plots of UMAP clustering, showing gene expression of MM pathological marker \textit{CD138} and apoptotic marker \textit{TNFRSF10B}.
The myeloma cell population is circled in the first panel.
Halofugione and NCP26 treatment reduce \textit{CD138} expression and increase \textit{TNFRSF10B} expression in the MM clusters.}
\label{fig:naive_path_vln_ftp}
\end{figure}
%%

Moreover, Figure \ref{fig:naive_path_vln_ftp} demonstrates that NCP26 and HF have profound anti-MM effects on MM cells.
NCP26 and HF treatment markedly reduce expression of MM pathological marker \textit{CD138} in MM cells.
Additionally NCP26 and HF treatment reduced expression of \textit{IGKC} and \textit{IGLC2}, the genes coding for the constant regions of immunoglobulin light chains, Kappa and Lambda.
Previously these genes have been implicated in MM outcome.
% https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6771956/

% Effect on other cell populations- e.g. monocytes, why does it affect monocytes

% Velocity analysis

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\clearpage
\subsection{Relapsed MM}
Figure \ref{fig:full_anno_relapse} shows final cell-type annotation for the integrated relapsed MM patients.
15 distinct clusters were identified.
%% Final annotation relapse
\begin{figure}[hpt]
\centering
\includegraphics[width=\textwidth]{figures/Results/single_cell/relapse_full_annotation.pdf}
\caption[Relapsed MM scRNA-seq full annotation]{Fully annotated UMAP clustering analysis of two relapsed multiple myeloma (MM) patients.
15 distinct clusters were identified.
Two subclones of MM were identified (circled; clusters 4 and 9).}
\label{fig:full_anno_relapse}
\end{figure}
%%
The expected major immune clusters were identified (such as B cells, NK cells, T cells and myeloid cells.)
Using the established MM biological markers (shown in Figure \ref{fig:mm_markers_relapsed}), and inferCNV results, two distinct MM subclones (cluster 4 and cluster 9) were identified.
The relapsed patients show substantial transcriptional differences to treatment-naive patients, in both their myeloma cells and their immune microenvironment.

\subsubsection{Composition}
Cluster composition analysis for relapsed MM by treatment is shown in Figure \ref{fig:composition_relapse}.
%% RELAPSE UMAP cluster composition separated by treatment condition
\begin{figure}[htb]
\centering
\includegraphics[width=\textwidth]{figures/Results/single_cell/sc_composition_relapse_barchart.pdf}
\caption[scRNA-seq composition analysis- relapsed MM]{Composition analysis of relapsed MM cells treated for 24 hours with ProRS inhibitors.
    a) UMAP cell composition plots separated by treatment condition.
    b) Dot plot showing proportion of cells in each cluster for each sample.
    c) Dot plot showing proportion of cells in each cell class for each sample (as labelled in Figure \ref{fig:full_anno_relapse}).
    d) The proportion of cells in the MM cluster only (stars above bars indicate significant at p<0.01 compared to DMSO control).
NCP26 treatment reduces the proportion of cells in the MM cluster (p<0.01).}
\label{fig:composition_relapse}
\end{figure}
%%
1\si{\micro\Molar} and 5\si{\micro\Molar} NCP26 treatment reduced the proportion of cells in MM cluster 4 compared to the DMSO control.
% Effect of NCP26 on other clusters. myeloid
There is insufficient evidence (p>0.05) to conclude if Halofuginone treatment affected the proportion of cells in MM cluster 4.
However, Halofuginone significantly reduced the proportion of cells in the myeloid and B cell clusters.
This perhaps indicates that in relapsed MM, Halofuginone is not selective for myeloma cells over other immune cells.
This may indicate an advantage of NCP26 and proline non-competitive ProRS inhibitors in relapsed myeloma.
Neither ProRS inhibitors reduced the proportion of cells in MM cluster 9.

\subsubsection{Differential expression}
% Number of DE genes
Next, differential gene expression was investigated for the relapsed MM patients.
%As we can see in figure \ref{fig:umap_RBC} there are not many cells originating from patient 3, therefore cells originating from patient 4 will be %mainly focussed on.
%
%%  DE barchart relapse
\begin{figure}[htb]
\centering
\includegraphics[width=0.8\textwidth]{figures/Results/single_cell/relapse_DEGs_barchart.pdf}
\caption[scRNA-seq DEGs per cell type- relapsed MM]{Number of differentially expressed genes (DEGs; p\textsubscript{adj}< 0.05) broken down by cell type for a relapsed MM patient (patient 4 only) treated with ProRS inhibitors (Halofuginone and NCP26) for 24 hours.
Cell type annotation corresponding to Figure \ref{fig:full_anno_relapse}.}
\label{fig:relapse_deg_bar}
\end{figure}
%
1073 genes were DE in MM cluster 4 following 24-hour 1\si{\micro\Molar} HF treatment.
157 and 801 genes were DE in the MM cell cluster following 24-hour 1\si{\micro\Molar}  and 5\si{\micro\Molar} NCP26 treatment, respectively.
The breakdown of DEGs per cell type for NCP26 and HF treatment is shown in Figure \ref{fig:relapse_deg_bar}.

4, 1, and 3 genes were DE in MM cluster 9 by Halofuginone, 1\si{\micro\Molar} NCP26  and 5\si{\micro\Molar} NCP26 treatment, respectively.
Together with the composition analysis, this indicates that the cluster 9 MM subclone may be more resistant to ProRS inhibition than the other myeloma subclone.

In contrast to the newly-diagnosed MM data, many of the other immune subtypes are differentially expressed to similar degrees as MM cells by ProRS inhibition.
Additionally, the myeloid cell cluster has more DEGs than MM cells for 1\si{\micro\Molar} and 5\si{\micro\Molar}  NCP26 treatment.
Myeloid cells have fewer DEGs than cluster 4 MM cells for HF treatment, however Figure \ref{fig:composition_relapse} demonstrates a substantial lower proportion of cells in the myeloid cells for the HF-treated sample, therefore you may not expect as many statistically significant DEGs for the few cells remaining in the cluster.
This could indicate that NCP26 and HF are not as selective for MM cells in relapsed MM as they are for newly-diagnosed MM, and that the ProRS inhibitors may not be very effective against some resistant myeloma subclones.

% Pathway
\subsubsection{Pathway analysis}
Pathway enrichment analysis was performed for the top 200 upregulated and top 200 downregulated DEGs in MM cluster 4, following Halofuginone and NCP26 treatment (Figure \ref{fig:relapse_mm_pathway_analysis}).
Pathway analysis could not be performed for MM cluster 9, as there were too few DEGs.
%%%%
%%  Pathway analysis
\begin{figure}[htb]
\centering
\includegraphics[width=\textwidth]{figures/Results/single_cell/relapse_mm_pathway_analysis.pdf}
\caption[scRNA-seq MM cluster pathway analysis (relapsed MM)]{Pathway analysis (Gene ontology biological processes; GOBP) of MM cluster 4 in RRMM dataset.
GOBP pathway analysis performed using top 200 upregulated or downregulated DEGs for ProRS inhibitor (1\si{\micro\Molar} Halofuginone and 1\si{\micro\Molar} or 5\si{\micro\Molar}  NCP26) treatment compared to DMSO control treatment for MM cluster 4.
}
\label{fig:relapse_mm_pathway_analysis}
\end{figure}
%%
Genes involved in translation pathways are downregulated following ProRS inhibition, as well as cell cycle pathways.
Cell death, apoptotic and cell cycle pathways are enriched following ProRS inhibition.
This follows literature and previous sections, whereby AAR activation triggers cell cycle changes, cell death and a global reduction in the translational machinery.
Interestingly, for all three treatments, mRNA splicing was enriched, indicating a potential spliceosomal mechanism for these inhibitors.

Figure \ref{fig:relapse_aar_vln_ftp} shows AAR gene expression in relapsed myeloma cells, as well as the surrounding immune cells.
%
%%  AAR violin and feature plots
\begin{figure}[htb]
\centering
\includegraphics[width=\textwidth]{figures/Results/single_cell/relapse_aar_vln_ftp.pdf}
\caption[scRNA-seq differentially expressed AAR genes- relapsed MM]{A selection of differentially expressed amino acid starvation response (AAR) genes following 24 hour treatment with ProRS inhibitors (newly-diagnosed MM patients).
    a) Violin plots showing AAR gene expression in MM cell cluster 4.
    a) Violin plots showing AAR gene expression in MM cell cluster 9.
    b) Feature plots of UMAP clustering, showing gene expression of AAR genes \textit{ATF3}, \textit{DDIT3} and \textit{GADD34}.
Feature plots are split into four panels based on treatment condition.
The myeloma subclones are circled in the first panel.
}
\label{fig:relapse_aar_vln_ftp}
\end{figure}
%%
AAR gene expression in MM cluster 4 follows the typical patterns of ProRS inhibition, as seen in the naive dataset and bulk RNA-seq data, increased expression of \textit{ATF3}, \textit{GADD34}, \textit{GADD45A}, \textit{DDIT3} and amino acid transporters, and decreased expression of some AAR-related genes such as \textit{DDIT4}.
However, for the other myeloma subclonal population (cluster 9), AAR transcriptional changes following ProRS inhibitor treatment do not seem to fit the typical pattern seen in previous data.
For example, \textit{ATF3}, \textit{GADD34} and \textit{SLC38A2} are upregulated in cluster 9 following NCP26 treatment, however many other AAR genes' expression levels remain unaffected by treatment.
Additionally, some AAR gene expression levels change inversely to how they behave in previous data, for example ProRS inhibition causes upregulation of \textit{DDIT4} and \textit{FAU} in cluster 9, where previously HF and NCP26 treatment has been shown to cause downregulation of these genes in myeloma cells.


\subsection{ProRS inhibitor effect on myeloid cells}
% look up INF-gamma etc. VEGFA
From differential expression and composition analysis, it seems that Halofuginone and NCP26 have a large transcriptional effect on myeloid cells, especially monocytes.
Feature plots and violin plots demonstrate that amino acid starvation response effector genes, such as \textit{DDIT3} and \textit{ATF3} are enriched in monocytes, as well as MM cells.
However, myeloid cells are non-malignant.
ProRS inhibitors were originally used as cancer treatments because tumour cells produce large amounts of protein, and are therefore heavily dependent on translational machinery, like tRNA-synthetase enzymes.
This is especially true for myeloma cells, secreting huge amounts of M protein.
Additionally, tumours are usually extremely proline-rich.
Therefore, it is surprising that ProRS inhibitors are non-selective for MM cells over MM patients' non-malignant myeloid cells.

Previously, Leiba et al. (2012) demonstrated HF's selectivity for MM patients' CD138\textsuperscript{+} tumour cells over PBMCs from healthy donors \cite{leiba2012halofuginone} using a cytotoxicity assay at the range of 25-200\si{\nano\Molar}.
This is a significantly lower concentration of HF than used in the scRNA-seq experiments (1\si{\micro\Molar}), so this could represent dosing differences.
However, it must also be noted that Leiba et al. (2012) did not separate PBMCs into their respective subtypes prior to performing the cytotoxicity assays.
As previously demonstrated, HF treatment had very little effect on B, T and NK cells.
Monocytes only make up approximately 5-10\% of the cells found in PBMCs- therefore, the effect of HF on monocytes could have been missed due to the minimal effects on the other cell types making up PBMCs.
Furthermore, the control PBMCs originated from healthy donors and not from MM patients' immune microenvironment.
The immune microenvironment of MM patients has been shown to be substantially altered and impaired compared to healthy individuals \cite{de2013analysis}, meaning that using healthy PBMCs as a control for the selectivity of HF is not appropriate.
To truly determine the cytotoxic selectivity of ProRS inhibition for MM cells and myeloid cells, cytotoxic assays must be performed for MM cells and myeloid cells originating from MM patients.

Wang et al. (2020) investigated the use of Halofuginone to treat the inflammatory disease chronic periodontitis \cite{wang2020halofuginone}.
Using a mouse model, they demonstrated that HF treatment significantly reduced the expression levels of pro-inflammatory cytokines, for example IL-1$\beta$, IL-6 and TNF-$\alpha$.
HF treatment was also shown to reduce the total percent of infiltrating immune cells and myeloid cells in gingival tissues when compared with those in control, PBS-treated mice.
The percentage of myeloid cells making up the CD45\textsuperscript{+} fraction of cells in gingival tissue was reduced by approximately 40\%, following HF treatment.
The authors demonstrated that HF treatment did not affect the cell viability of BM-derived osteoclast precursors cells (that is monocytes and macrophages).
However, cell viability was investigated with concentrations of HF ranging from 10\si{\pico\Molar} to 100\si{\nano\Molar}, so perhaps cell myeloid death is reserved for higher doses of HF treatment.
The scRNA-seq experiments used 1\si{\micro\Molar} HF, so this may be the concentration levels required to observe ProRS inhibitory effects on myeloid cells.
Additionally, the monocytes/macrophages orignated from mice not humans, and without the immune microenvironment remodelling seen in MM\@.

% Inversely the effects may not be cytotoxicity

\subsubsection{Pathway analysis}
Inflammation etc. etc.
Non-canonical functionality etc. etc.

\subsubsection{Velocity analysis}


\section{Summary}

\section{Discussion and future directions}
