\chapter{Single-cell RNA-seq analysis of ProRS inhibitors}\label{ch:6-sc}

\section{Introduction}
MM cells grow within the bone marrow and are supported as they grow by their microenvironment.
The MM microenvironment comprises a cellular compartment (composed of immune cells, endothelial cells, osteoblasts, osteoclasts and stromal cells) and a non-cellular compartment (composed of the extracellular matrix [ECM], cytokines, chemokines and growth factors)\cite{manier2012bone, kawano2015targeting} with significant interactions between malignant plasma cells and the surrounding microenvironment.
The BM microenvironment has been suggested to play a supportive role in migration, proliferation, differentiation and drug resistance of malignant plasma cells\cite{ho2020role, manier2012bone}.
There is evidence linking the tumour microenvironment to progression of MGUS to active MM, for example significant matrix remodelling has been observed between the bone marrow of healthy individuals, MGUS and MM patients\cite{kawano2015targeting}.
Therefore, to get an accurate picture of MM, information must be acquired about the surrounding niche.

Historically, the tumour environment has been investigated following the isolation of populations of cells sorted from the tumour and then sequenced using traditional microarray or bulk RNA-seq techniques.
Bulk techniques measure the average expression across a sample, which is the sum of cell type specific expression weighted by cell type proportions.
Single-cell techniques measure expression for each individual cell and therefore provide information on cellular diversity that may be lost when pooling cells into bulk samples.
Furthermore, multiple myeloma is an extremely heterogeneous disease, this is seen both between patients and within an individual's own tissue.
Applying single-cell techniques to capture the inter- and intra-individual heterogeneity is fundamental to identifying molecular and cellular signatures that define MM\@.

The advent of single-cell technologies has led to a better understanding of the complexity and diversity of the tumour microenvironment.
Seminal papers from Melnekoff et al. (2017)\cite{melnekoff2017single} and Ledergor et al. (2018)\cite{ledergor2018single} use scRNA-seq to reveal clonal transcriptomic heterogeneity in MM samples.
Melnekoff et al. (2017) demonstrated the clonal heterogeneity within MM using samples that were collected from eight relapsed MM patients.
The group performed t-SNE clustering analysis and the samples separated into eight transcriptionally distinct clones, each corresponding to a different patient.
This highlights the inter-patient differences of MM\@.
Ledergor et al. (2018) performed a similar study to evaluate clonal heterogeneity within MM but also included a set of healthy controls with which to compare the MM group.
They found that MM patients have greater inter-individual transcriptional variation, where each MM patient possessed a unique and individual plasma cell transcriptional program.
They also showed substantial intra-tumour heterogeneity (subclonal structures) of plasma cells in a third of their MM patient cohort.
These papers established the importance of using single-cell techniques to study MM, as to not miss the underlying clonal heterogeneity.
However, both of these papers focussed solely on plasma cells and did not look at the surrounding BM microenvironment.
To truly understand the complexities of MM and treatment of MM, interactions between plasma cells and the BM niche must also be explored using single-cell techniques.

This chapter uses MM patient-derived BM, from both newly-diagnosed and relapsed patients, to investigate the transcriptional effects of ProRS on MM cells and their surrounding immune microenvironment.

\subsection{Overview of experiments}\label{subsec:scrna_experiment}
Three single-cell experiments, comprising samples from four MM patients, were performed to explore the effect of various compounds (including HF and NCP26) on MM patient tissue.
The BM samples for experiments 1 and 2 were obtained from two treatment-na\"ive, newly-diagnosed MM patients.
Experiment 3 comprised samples from two relapsed MM patients, therefore both presenting with a degree of acquired anti-cancer drug resistance.
For experiment 1, BM samples were treated for 24 hours with 1\si{\micro\Molar} Casin, GSK-J4, HF, NCP26, SGC-GAK, Verteporfin or a DMSO control, totalling 7 samples.
For experiment 2, BM samples were treated with with 1\si{\micro\Molar} CAMKK2, CLK or CSNK2 for 24 hours; 1\si{\micro\Molar} SGC-GAK, HF, NCP26 or a DMSO control for 24 and 48 hours, totalling 11 samples.
For experiment 3, BM samples from patient 3 and 4 were treated for 24 hours with either a DMSO control, 1\si{\micro\Molar} HF, 1\si{\micro\Molar} NCP26 or 5\si{\micro\Molar} NCP26, totalling 8 samples.

Following compound treatment, scRNA-seq library preparation was performed by Dr Martin Philpott as outlined in Section \ref{subsec:10x_method}.
% <GENEWIZ experiment 3 >


\section{Data processing}
For multiple scRNA-seq datasets to be analysed together and compared, integration of datasets is required.
Integration methods identify `anchors' across datasets-- cells that are in matched biological state, these cells can then be used to correct for technical variation and batch effects.
Initially, all four patient samples (newly-diagnosed and relapsed) were processed and integrated together.
However, integration was found to be poor between treatment-na\"ive and relapsed patient samples.
The transcriptome of MM patients has been shown to change considerably following successive rounds of treatment cycles, therefore this could explain the poor integration of BM samples from patients at different points in disease progression.
That is, the datasets were too different to identify good anchors between them.
Treatment-na\"ive patient samples (from experiments 1 and 2-- patients 1 and 2) were integrated and analysed together and relapsed patient samples (from experiment 3-- patients 3 and 4) were integrated and analysed together.

Experiments 1 and 2 contained numerous samples that were not of interest to this project.
Yet, all samples originated from MM patient tissue, therefore all 18 samples were processed by the analysis pipeline, up to and including integration and annotation steps.
These samples were included to increase the granularity of the data, and allow for easier annotation of clusters.
Accordingly, downstream analysis was only performed for DMSO, HF and NCP26 samples.

\subsection{Analysis overview}

The single-cell analysis pipeline outlined in Section \ref{subsec:updated_scrna} was used to process the scRNA-seq data.
Kallisto BUS/ BUStools was used to pseudoalign reads and quantify gene expression.
Next, quality control and filtering of the samples was performed (as per Section \ref{subsec:updated_scrna}), then clustering was performed using Seurat, followed by integration using Seurat and Harmony.
Using the Seurat SCTransform normalisation method of integration, the two experiments were too large to integrate across all samples.
Therefore, a reference-based approach was taken, whereby a subset of samples were selected (based on their cell richness and relevance to the research question) and listed as `reference datasets' for SCTransform normalisation.
Harmony integration was also implemented, using `Patient number' and `Experiment number' as additional covariates.

Cell type annotation was performed with several automated packages (singleR, clustifyr and scClassify), and then fine-tuned manually using a list of known biological markers.
No MM classifier or reference was available when performing this analysis (the MM classifier in Section \ref{sec:MM_classifier} was built using these datasets).
Therefore, healthy human references were used for annotation.
The HumanCellAtlas (HCA) database was used to inform singleR and clustifyr annotation.
scClassify was performed in conjunction with a pre-trained scClassify model, based on seven peripheral blood mononuclear cell (PBMC) scRNA-seq datasets (including data from 10Xv2, 10Xv3, smartSeq, celSeq, dropSeq and inDrops).
As the reference datasets were based on healthy tissue, they were unable to label pathological cells, like MM cells.
Myeloma cells were identified manually using a range of known markers, for example: \textit{CD38}, \textit{CD138}, \textit{SLAMF7} and \textit{BCMA} (Table \ref{tab:annotation_markers}).

% Pre RBC dimplots naive
\begin{figure}[htb]
    \centering
    \includegraphics[width=\textwidth]{figures/Results/single_cell/data_processing/RBC_removal_all_experiments_naive.pdf}
    \caption[Erythrocyte removal- newly diagnosed MM]{UMAP clustering following integration of samples from experiment 1 and 2 (treatment-na\"ive MM patients).
    a) and b) Pre erythrocyte cluster and gene removal. Erythrocyte clusters are circled.
    c) and d) Post erythrocyte removal and re-integration of samples.
    a) and c) UMAP clustering.
    b) and d) The composition of each dataset by experiment number.}
    \label{fig:umap_RBC_naive}
\end{figure}

% Pre RBC dimplots relapsed
\begin{figure}[htb]
    \centering
    \includegraphics[width=\textwidth]{figures/Results/single_cell/data_processing/RBC_removal_all_experiments_relapsed.pdf}
    \caption[Erythrocyte removal- relapsed MM]{UMAP clustering following integration of samples from patients 3 and 4 in experiment 3 (relapsed MM patients).
    a) and b) Pre erythrocyte cluster and gene removal. Erythrocyte clusters are circled.
    c) and d) Post erythrocyte removal and re-integration of samples.
    a) and c) UMAP clustering.
    b) and d) The composition of each dataset by patient number.}
    \label{fig:umap_RBC_relapsed}
\end{figure}

Experiment 2 was found to have a high erythrocyte population (Figure \ref{fig:umap_RBC_naive}a and b).
In addition, other cell types were found to be expressing erythrocyte-specific genes, for example MM cells expressing numerous haemoglobin subunit genes.
Many of the variable features that Seurat uses for clustering and dimension reduction were made up of these erythrocyte-specific and haemoglobin genes.
The high expression of these genes was found to be affecting the integration of the two experiments together.
A theory for the presence of the large number of erythrocytes and un-localised erythrocyte gene expression is that perhaps the BM sample taken for experiment 2 was one of the later samples taken from the patient and contained a large amount of blood.
Library prep clean-up may have missed many of these cells, leading to ambient erythrocyte RNA being present within many droplets.
Other MM scRNA-seq studies have observed similar contamination of erythrocyte specific genes, such as \textit{HBB}, \textit{HBA1} and \textit{HBA2}, in non-blood cell populations, despite performing a red blood cell lysis step\cite{chen2021cryopreservation}.

It was decided to remove the erythrocyte clusters (clusters 3, 8, 9, 10, 11 and 16 in newly-diagnosed; clusters 1, 5 and 15 in relapsed) and haemoglobin-related genes or erythrocyte-specific genes that were dominating expression in the integrated dataset.
After the integrated Seurat object had the erythrocyte genes and cells removed, it was split back up into separate Seurat objects for each sample, and integration and clustering was performed again.
Seurat's SCTransform with reference datasets and Harmony (using a multi-covariate model, accounting for each different sample and the two different experiments) were used to re-integrate all samples.
Harmony integration was found to integrate clusters across patients and experiments better than using Seurat's SCTransform.
The Harmony-integrated datasets were taken forwards and used for cell type annotation and further downstream analyses.
Better integration was achieved after removing erythrocytes and erythrocyte-specific genes (Figure \ref{fig:umap_RBC_naive}d).

A large erythrocyte component was also found for patient 4 in the relapsed dataset (Figures \ref{fig:umap_RBC_relapsed}a and \ref{fig:umap_RBC_relapsed}b).
The same corrective workflow outlined above for the treatment-na\"ive dataset was applied to the relapsed dataset, removing the erythrocyte cluster and erythrocyte-specific genes and re-integrating using Harmony with samples and patients as covariates.

The total number of cells after each data processing step for each patient and experiment is summarised in Table \ref{tab:cells_passing}.
\input{text/Chapter_6_single-cell_PRS/cells_passing_table.tex}
%%%
The number of cells originating from patient 3 after filtering might be a barrier to interpreting meaningful results.
Patients 1, 2 and 4 have sufficient cell numbers to perform downstream analysis, for example investigating differential expression.

\subsection{Annotation of re-integrated data}\label{subsec:sc_annotate}
% automated annoation, scclasify, clustifyr
R packages clustifyr and scClassify were used to aid in cell type annotation of the integrated datasets.
The result of this annotation can be seen in Figure \ref{fig:annotation_automated}.
%%
%%% Figure automated package annotation
\begin{figure}[htb]
    \centering
    \includegraphics[width=\textwidth]{figures/Results/single_cell/data_processing/automated_markers_ABCD.pdf}
    \caption[Automated annotation of scRNA-seq data]{Automated annotation of scRNA-seq cell clusters, using the R packages clustifyr and scClassify in combination with reference datasets.
    [a, b] Newly-diagnosed MM, [c, d] relapsed MM.
    The output of automated packages clustifyr and scClassify is used to aid cell type annotation.
    Clustifyr assigns a cell type to each cell cluster, whilst scClassify assigns a cell type to each individual cell.
    Clustifyr was used in conjunction with the HumanCellAtlas reference.
    scClassify was ran using a pre-trained model trained on seven PBMC single-cell RNA-seq datasets.
    Both references originate from healthy data, therefore neither are able to identify the myeloma cell population.}
    \label{fig:annotation_automated}
\end{figure}

Automated annotation using computational methods gives a good starting point for more detailed manual annotation.
Currently, these methods are not a complete substitute for manual annotation using biological knowledge of cell markers.
Both cell-type references originate from healthy tissue, therefore the myeloma cluster could not be identified using clustifyr and scClassify with these healthy references alone.
Both packages either incorrectly labelled the MM cluster as B cells or were unable to assign any cell type to the myeloma cluster with any confidence and labelled it `unassigned'.
Thus, the MM clusters had to be identified manually using MM biological knowledge.
Section \ref{sec:MM_classifier} constructs a MM classifier reference to overcome this problem for future MM scRNA-seq datasets.

Figures \ref{fig:mm_markers_naive} and \ref{fig:mm_markers_relapsed} illustrate MM cell identification for the treatment-na\"ive and relapsed datasets, respectively, using gene expression of certain MM markers.
% MM markers vln, feature and dotplots
\begin{figure}[p]
    \centering
    \includegraphics[width=\textwidth]{figures/Results/single_cell/data_processing/markers_naive.pdf}
    \caption[MM cluster identification- newly diagnosed MM]{Manual annotation of multiple myeloma (MM) cell clusters in experiment 1 and 2 (newly-diagnosed patients) using known MM biological markers.
    a) UMAP feature plots of MM marker expression.
    Grey indicates no expression and purple indicates gene expression.
    b) A dot plot of the percentage of cells expressing a given marker and the average expression (for each cluster).
    c) Violin plots of gene expression for each cluster.}
    \label{fig:mm_markers_naive}
\end{figure}
%
% MM markers vln, feature and dotplots
\begin{figure}[p]
    \centering
    \includegraphics[width=\textwidth]{figures/Results/single_cell/data_processing/markers_relapsed2.pdf}
    \caption[MM cluster identification- relapsed MM]{Manual annotation of MM cell clusters in experiment 3 (relapsed MM) using known biological markers.
    a) UMAP feature plots of MM marker expression.
    b) A dot plot of the percentage of cells expressing a given marker and the average expression (for each cluster).
    c) Violin plots of gene expression for each cluster. }
    \label{fig:mm_markers_relapsed}
\end{figure}
%
For MM cells, one would expect to see: expression of \textit{CD38}, but lower expression than in normal plasma cells;
high expression of \textit{CD138} in MM cells, as well as high expression of \textit{SLAMF7}, \textit{BCMA}, \textit{KRAS}, \textit{IGKC} and \textit{IGL2}, however these are not exclusive to the MM cluster.
You would expect to see little or no expression of \textit{CD45} and \textit{CD19} in the MM cluster and reduced expression of \textit{CD20} in MM cells compared to normal B cells.
One may also expect to see expression of \textit{CD56} (\textit{NCAM1}) in abnormal plasma cells, however this is not always detected at the RNA level.
Other common markers used to identify MM cells include, \textit{CD81}, \textit{B2M} and \textit{CD117}.
Using the expression of a combination of these markers, clusters 2, 7 and 13 were identified as the MM cell population in the newly diagnosed dataset and cluster 4 was identified as MM cells in the relapsed MM dataset.

However, from marker expression alone, the identity of clusters 9 and 12 in the relapsed dataset were unclear.
Both clusters 9 and 12 seem to belong to a B-cell lineage.
Cluster 12 shows increased \textit{CD20} expression, indicating they could be mature B cells.
However, clusters 4, 9 and 12 all have an increased expression of \textit{IGKC}, compared to non-B cell clusters.
Indicating that perhaps, all B-cell clusters (4, 9 and 12) in the relapsed dataset, may be in fact MM cells.
It has previously been shown that MM cells possess a level of plasticity, that could allow de-differentiation into other cell lineages\cite{kotouvcek2014myeloma}, this may account for the \textit{CD20} expression of cluster 12.

Due to the uncertainty of cell-type identity in the relapsed dataset based on MM marker gene expression alone, inferCNV\cite{patel2014single, infercnv2014} was employed to detect large-scale chromosomal copy number variations (CNVs) and help inform annotation.
The raw counts expression matrix of the relapsed, integrated Seurat object was used as input for inferCNV\@.
`Normal' cells (that is: myeloid cells, T cells and NK cells) were used as a reference for the relative expression intensity of the suspected malignant cell clusters (clusters 4, 9 and 12).
A Cutoff value of 0.1 was used for the analysis, to account for the sparsity of droplet-based 3' scRNA-sequencing.
%
% inferCNV
\begin{figure}[htb]
    \centering
    \includegraphics[width=\textwidth]{figures/Results/single_cell/data_processing/inferCNV_relapse.pdf}
    \caption[inferCNV- relapsed MM]{InferCNV results for the relapsed MM dataset.
    [a and b] InferCNV heatmaps.
        The top panel shows expression values for the reference `normal' cells.
        The bottom panel shows expression values for the suspected malignant cells (clusters 4, 9 and 12).
        Red indicates chromosomal region amplifications and blue indicates chromosomal region deletions.
    a) De-noised inferCNV results.
    b) Hidden Markov-Model (HMM) copy number variation (CNV) region predictions.
        Chromosomal deletions predicted for chromosome 13 and chromosomal duplications/gains predicted in chromosomes 11 and 19 for clusters 4 and 9.
        Gains also predicted in chromosome 3 and 9 for cluster 4.
    c) UMAP featureplots demonstrating chromosome 13 loss and chromosomes 3, 9 and 11 duplications.
    }
    \label{fig:inferCNV_relapse}
\end{figure}
%
Figure \ref{fig:inferCNV_relapse} shows the inferCNV results for the relapsed dataset.
Myeloma-defining CNVs were seen in both cluster 4 and cluster 9, including chromosome 13 deletion and duplications of chromosomes 11 and 19.
The inferCNV results also demonstrate that there are different genetic MM subclones within the population, as cluster 4 alone shows regional gain of chromosomes 3, 7, 9 and 16.
Therefore, it can be concluded that clusters 4 and 9 are subclonal MM populations.

InferCNV was also employed for the treatment-na\"ive dataset (Figure \ref{fig:inferCNV_naive}) to check for CNVs.
The inferCNV results confirmed the gene expression-based identification of clusters 2, 7 and 13 as the MM cell population.

% appendix annotation for other clusters - ?
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% Results
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% flush figures out before beginning results section
\clearpage
\section{Results}

\subsection{Newly-diagnosed MM}

% Final product of annotation

%% Final annotation
\begin{figure}[hpt]
\centering
\includegraphics[width=\textwidth]{figures/Results/single_cell/naive_full_annotation.pdf}
\caption[scRNA-seq full annotation- newly-diagnosed MM]{Fully annotated UMAP clustering analysis of two newly-diagnosed multiple myeloma (MM) patients.
The MM cell population (circled) consists of three distinct clusters.}
\label{fig:full_anno_naive}
\end{figure}
%%

Figure \ref{fig:full_anno_naive} shows final cell-type annotation for the newly-diagnosed MM integrated dataset.
18 distinct clusters were identified using Seurat embeddings.
The expected major immune clusters were identified (such as B cells, T cells, NK cells and myeloid cells.)
Using the established MM biological markers (Figure \ref{fig:mm_markers_naive}), three distinct MM clusters (2, 7 and 13) were identified.

\subsubsection{Composition}
Cluster composition analysis by treatment condition is shown in Figure \ref{fig:composition_naive}.
Halofuginone treatment at both 24 and 48 hours reduced the proportion of cells in the MM cluster (p<0.00001) compared to DMSO, for both experiment 1 and 2.
NCP26 treatment at 24 hours reduced the proportion of cells in the MM cluster (p<0.00001), for both experiment 1 and 2.
HF and NCP26 treatment increased the proportion of cells in the CD4+ T cell, CD8+ T cell and B cell clusters.
Together with dose response curves and cell death assays, this suggests that HF and NCP26 are selectively killing MM cells to a higher degree than other cell types.

%%  UMAP cluster composition separated by treatment condition
\begin{figure}[htb]
\centering
\includegraphics[width=\textwidth]{figures/Results/single_cell/sc_composition_naive2_barchart.pdf}
\caption[Composition analysis- newly diagnosed MM]{Composition analysis of newly diagnosed MM.
    a) UMAP cell composition plots separated by treatment condition.
    b) Dot plot showing proportion of cells in each cluster for each sample.
    c) Dot plot showing proportion of cells in each cell class for each sample (as labelled in Figure \ref{fig:full_anno_naive}).
    d) The proportion of cells in the MM cluster only (stars above bars indicate significant at p<0.01 compared to corresponding control).
    Sample names starting with asterisks originate from experiment 1, no asterisk indicates experiment 2 origin.}
\label{fig:composition_naive}
\end{figure}
%%

HF treatment and 48-hour NCP26 treatment were also found to reduce the proportion of cells in the monocyte and neutrophil cluster (p<0.00001), indicating they may have some undesirable off-target effects on myeloid cells.

All of the compounds were used at a concentration of 1\si{\micro\Molar}.
This is approximately 10 times the concentration of HF's IC\textsubscript{50} value on MM cell lines and two times NCP26's IC\textsubscript{50} value.
As you can see, large number of cells were killed by HF treatment at both 24 and 48 hours.
Therefore, the effects seen on the myeloid cells could be due to high dosing of the ProRS inhibitors.

\subsubsection{Differential expression}
% Number of DE genes
% separate into NCP26 and HF??
Next, differential gene expression was investigated using Seurat's \texttt{FindMarkers} function.
At 48 hours there was substantial cell killing, therefore the 24-hour time point will be mainly focussed on.
%
%%  DE barchart
\begin{figure}[htb]
\centering
\includegraphics[width=\textwidth]{figures/Results/single_cell/naive_DEGs_barchart.pdf}
\caption[DEGs per cell type- newly-diagnosed MM]{Number of differentially expressed genes (DEGs; p\textsubscript{adj}< 0.05) by cell type for two newly-diagnosed MM patients treated with ProRS inhibitors (Halofuginone and NCP26) for 24 hours.
Cell type annotation corresponding to Figure \ref{fig:full_anno_naive}.
Experiment 1 and experiment 2 denote separate experiments, each containing BM samples from different newly-diagnosed MM patients.}
\label{fig:naive_deg_bar}
\end{figure}

Following 24-hour 1\si{\micro\Molar} NCP26 treatment, 1515 genes were differentially expressed (DE; p\textsubscript{adj}< 0.05) in the myeloma cell population in experiment 1, and 1294 genes in experiment 2, compared to DMSO treatment.
Figure \ref{fig:naive_deg_bar} shows the breakdown of DEGs per cell type for NCP26 and HF treatment.
24 hour 1\si{\micro\Molar} NCP26 treatment had very little transcriptional effect on many of the other immune cell types, for example T, B and NK cells, where the number of DEGs is an order of magnitude less than for MM cells.
This corroborates the composition analysis, where MM cells seem more sensitive to ProRS inhibition than other immune cell types.
However, 2016 and 1485 genes were DE in the monocyte cluster following NCP26 treatment in experiment 1 and 2, respectively.
This reflects the composition analysis, where the proportion of cells in the monocyte cluster was markedly reduced.
This could indicate that NCP26 is not selective for MM cells over myeloid cells.

In MM cells, 73 genes were DE following HF treatment in experiment 1, and 1318 genes in experiment 2.
In the composition analysis (Figure \ref{fig:composition_naive}), we saw very few cells remaining in the MM cluster for experiment 1 following HF treatment.
Due to the low cell number, there is less statistical power and diversity across the cells, therefore you would likely see fewer statistically significant DEGs for this cluster.
Unlike NCP26 treatment, for 24 hour 1\si{\micro\Molar} HF treatment there were many DEGs in B, T and NK cell clusters.
This likely reflects the differences in potency between NCP26 and HF\@.
HF is approximately five times more potent than NCP26.
This fits with composition analysis, where HF treatment largely reduced the proportion of cells in the myeloid and MM clusters.

At lower doses (i.e. NCP26 at 1\si{\micro\Molar}) of ProRS inhibition we see clear evidence of greater transcriptional effects on MM cells and monocytes over other immune subtypes.
However at higher doses (i.e. HF at 1\si{\micro\Molar}) we see substantial cell killing of MM cells and monocytes and larger transcriptional effects on other immune subtypes.
This together with the composition analysis, demonstrates that NCP26 and halofuginone are selective for MM cells over most immune cells, except for myeloid cells.
However, this could be a problem with the doses used for treatment.
1\si{\micro\Molar} is almost 10 times greater than HF's IC\textsubscript{50} and two times greater than NCP26's IC\textsubscript{50}.
This experiment should be performed at a lower concentration to ascertain if NCP26 and HF are more selective for MM cells or monocytes.
Ideally, the experiment would be performed over the course of three days at a lower concentration, as in our cell line studies.
However, human bone marrow samples do not last very well in extended culture under conditions employed in this study, therefore acute exposure (24 hr) was the preferred method for this experiment.

\subsubsection{Pathway analysis}
Pathway enrichment analysis was performed for the top 200 upregulated and top 200 downregulated DEGs in the MM cluster, following halofuginone and NCP26 treatment (Figure \ref{fig:naive_mm_pathway_analysis}).
%%  SC MM pathway analysis-- might need to up font size
\begin{figure}[htb]
\centering
\includegraphics[width=\textwidth]{figures/Results/single_cell/naive_mm_pathway_analysis.pdf}
\caption[MM cluster pathway analysis- newly-diagnosed MM]{Pathway analysis (Gene ontology biological processes; GOBP) of the MM cluster.
    a) and b) Halofuginone treatment.
    c) and d) NCP26 treatment.
a) and c) GOBP pathway analysis performed using top 200 upregulated DEGs (p\textsubscript{adj} < 0.05 and $\log_{2}$FC>0) ranked by fold change.
b) and d) GOBP pathway analysis performed using top 200 downregulated DEGs (p\textsubscript{adj} < 0.05 and $\log_{2}$FC<0) ranked by fold change.}
\label{fig:naive_mm_pathway_analysis}
\end{figure}
%%

For NCP26 and HF treatment of MM cells, pathways related to ER-stress and apoptosis were enriched.
The unfolded protein response- another member of the ISR, which shares many effectors with AAR- was also enriched.
This supports our bulk RNA-seq results that the amino acid starvation response is activated following NCP26 and HF treatment of MM cells.

Figure \ref{fig:naive_aar_vln_ftp} shows a more detailed exploration of AAR gene expression in MM cells and surrounding immune cells.
%
%%  AAR violin and feature plots
\begin{figure}[htb]
\centering
\includegraphics[width=\textwidth]{figures/Results/single_cell/naive_aar_vln_ftp.pdf}
\caption[Differentially expressed AAR genes- newly diagnosed MM]{A selection of differentially expressed amino acid starvation response (AAR) genes following 24 hour treatment with ProRS inhibitors (newly-diagnosed MM patient bone marrow samples).
    a) Violin plots showing expression of selected AAR genes in the MM population following DMSO, halofuginone and NCP26 treatment.
    b) Feature plots of UMAP clustering, showing gene expression of AAR genes: \textit{ATF3}, \textit{GADD34}, \textit{DDIT3} and \textit{TRIB3}.
Feature plots are split into three panels based on treatment condition (DMSO, halofuginone and NCP26).
The myeloma cell population is circled in the first panel.}
\label{fig:naive_aar_vln_ftp}
\end{figure}
%%
Gene expression of \textit{ATF4} and some of its target genes, such as \textit{ATF3}, \textit{DDIT3} and  \textit{GADD34}, were increased in MM cells, following HF and NCP26 treatment (Figure \ref{fig:naive_aar_vln_ftp}).
tRNA aminoacyl synthetase genes (\textit{WARS1} and \textit{SARS1}) were also over-expressed in MM cells following ProRS inhibitor treatment.
Figure \ref{fig:naive_aar_vln_ftp}b shows selected AAR genes' expression across all cell clusters, separated by treatment condition.
HF and NCP26 treatment markedly increased expression of \textit{ATF3} and \textit{TRIB3} in MM cell and monocyte clusters.
Increased expression of \textit{DDIT3} and \textit{GADD34} can be seen across a few different clusters, however it is most pronounced in the MM and monocyte clusters.
Additionally, the apoptotic mediator \textit{TNFRSF10B}, positively regulated by DDIT3, was upregulated in MM cells and monocytes (Figure \ref{fig:naive_path_vln_ftp}b).
Indicating that, although aaRSs are ubiquitous enzymes and ProRS inhibitors cause some degree of activation of the AAR in all cells, they are cytotoxically more selective for MM cells, and lead to an apoptotic cascade of events, preferentially over other cell types.
\textit{CDKN1A} was also overexpressed in MM cells following NCP26 and HF treatment.
\textit{CDKN1A} is a target gene of \textit{ATF4} and arrests cell cycle progression by inhibiting the activity of cyclin-dependent kinases.
HF has previously been shown to cause cell cycle arrest and to accumulate cells in the G\textsubscript{0}/G\textsubscript{1} phase of the cell cycle.
%
%%  other markers violin and feature plots
\begin{figure}[htb]
\centering
\includegraphics[width=\textwidth]{figures/Results/single_cell/naive_pathological_vln_ftp.pdf}
\caption[Differentially expressed MM markers- newly diagnosed MM]{A selection of differentially expressed multiple myeloma (MM) and cell cycle/apoptotic markers in newly-diagnosed MM patient bone marrow samples treated with ProRS inhibitors for 24 hours.
    a) Violin plots showing expression of selected genes in the MM population following DMSO, halofuginone and NCP26 treatment.
    b) Feature plots of UMAP clustering, showing gene expression of MM pathological marker \textit{CD138} and apoptotic marker \textit{TNFRSF10B}.
The myeloma cell population is circled in the first panel.
Halofugione and NCP26 treatment reduced \textit{CD138} expression and increased \textit{TNFRSF10B} expression in the MM clusters.}
\label{fig:naive_path_vln_ftp}
\end{figure}
%%

Moreover, Figure \ref{fig:naive_path_vln_ftp} demonstrates that NCP26 and HF have profound anti-MM effects on MM cells.
NCP26 and HF treatment markedly reduce expression of MM pathological marker \textit{CD138} in MM cells.
Additionally, NCP26 and HF treatment reduced expression of \textit{IGKC} and \textit{IGLC2}, the genes coding for the constant regions of immunoglobulin light chains, Kappa and Lambda.
Previously these genes have been implicated in MM outcome.
% https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6771956/


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% Relapsed %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\clearpage
\subsection{Relapsed MM}
Figure \ref{fig:full_anno_relapse} shows final cell-type annotation for the integrated relapsed MM patient samples.
Substantial transcriptional differences are apparent between the relapsed patients and the treatment-na\"ive patients, in both their myeloma cells and their immune microenvironment.
15 distinct clusters were identified.
%% Final annotation relapse
\begin{figure}[hpt]
\centering
\includegraphics[width=\textwidth]{figures/Results/single_cell/relapse_full_annotation.pdf}
\caption[scRNA-seq full annotation- relapsed MM]{Fully annotated UMAP clustering analysis of two relapsed multiple myeloma (MM) patients.
15 distinct clusters were identified.
Two subclones of MM were identified (circled; clusters 4 and 9).}
\label{fig:full_anno_relapse}
\end{figure}
%%
The expected major immune clusters were identified (such as B cells, NK cells, T cells and myeloid cells.)
Using the established MM biological markers (Figure \ref{fig:mm_markers_relapsed}), and inferCNV results (Figure \ref{fig:inferCNV_relapse}), two distinct MM subclones (cluster 4 and cluster 9) were identified.
MM cells in cluster 4 were seen to express \textit{CD138}.
However, MM cells in cluster 9 showed little or no \textit{CD138} expression.
As mentioned in Chapter \ref{ch:4-Pipelines}, these \textit{CD138}\textsuperscript{-} MM cells are likely to have higher proliferative potential than \textit{CD138}\textsuperscript{+} MM cells.
They are also sometimes regarded to have `stem-cell' like clonogenic properties and to be more resistant to chemotherapy than their \textit{CD138}\textsuperscript{+} counterparts\cite{setiadi2019cd138}.
This \textit{CD138}\textsuperscript{-} subclone of MM cells would not have been sequenced if \textit{CD138}\textsuperscript{+} enrichment techniques were used for sorting cells.

\subsubsection{Composition}
Cluster composition analysis for relapsed MM by treatment is shown in Figure \ref{fig:composition_relapse}.
%% RELAPSE UMAP cluster composition separated by treatment condition
\begin{figure}[htb]
\centering
\includegraphics[width=\textwidth]{figures/Results/single_cell/sc_composition_relapse_barchart.pdf}
\caption[Composition analysis- relapsed MM]{Composition analysis of relapsed multiple myeloma (MM) patient cells treated for 24 hours with ProRS inhibitors.
    a) UMAP cell composition plots separated by treatment condition.
    b) Dot plot showing proportion of cells in each cluster for each sample.
    c) Dot plot showing proportion of cells in each cell class for each sample (as labelled in Figure \ref{fig:full_anno_relapse}).
    d) The proportion of cells in MM cluster 4 only.
NCP26 treatment reduced the proportion of cells in the MM cluster (p<0.01).
    e) The proportion of cells in MM cluster 9 only.}
\label{fig:composition_relapse}
\end{figure}
%%
1\si{\micro\Molar} and 5\si{\micro\Molar} NCP26 treatment reduced the proportion of cells in MM cluster 4 compared to the DMSO control.
% Effect of NCP26 on other clusters. myeloid
There is insufficient evidence (p>0.05) to conclude if HF treatment affected the proportion of cells in MM cluster 4.
However, HF significantly reduced the proportion of cells in the myeloid and B cell clusters.
This perhaps indicates that in relapsed MM, HF is not selective for myeloma cells over other immune cells.
This may indicate an advantage of NCP26 and proline non-competitive ProRS inhibitors in relapsed MM\@.
\textit{PYCR1} (catalyzes the conversion of P5C to proline) expression has been shown to be increased in MM cells in relapsed/refractory MM patients\cite{oudaert2022pyrroline}, therefore relapsed MM patients may have increased proline concentration in MM cells than treatment-na\"ive MM patients.
Neither ProRS inhibitor reduced the proportion of cells in MM cluster 9.

\subsubsection{Differential expression}
% Number of DE genes
Next, differential gene expression was investigated for the relapsed MM patients.
%As we can see in figure \ref{fig:umap_RBC} there are not many cells originating from patient 3, therefore cells originating from patient 4 will be %mainly focussed on.
%
%%  DE barchart relapse
\begin{figure}[htb]
\centering
\includegraphics[width=\textwidth]{figures/Results/single_cell/relapse_DEGs_barchart.pdf}
\caption[DEGs per cell type- relapsed MM]{Number of differentially expressed genes (DEGs; p\textsubscript{adj}< 0.05) by cell type for a relapsed MM patient (patient 4 only) treated with ProRS inhibitors (halofuginone and NCP26) for 24 hours.
Cell type annotation corresponds to Figure \ref{fig:full_anno_relapse}.}
\label{fig:relapse_deg_bar}
\end{figure}
%
1073 genes were DE in MM cluster 4 following 24-hour 1\si{\micro\Molar} HF treatment.
157 and 801 genes were DE in MM cluster 4 following 24-hour 1\si{\micro\Molar}  and 5\si{\micro\Molar} NCP26 treatment, respectively.
4, 1, and 3 genes were DE in MM cluster 9 by HF, 1\si{\micro\Molar} NCP26  and 5\si{\micro\Molar} NCP26 treatment, respectively.
Together with the composition analysis, this indicates that the cluster 9 MM subclone may be more resistant to ProRS inhibition than the other MM subclone.

The breakdown of DEGs per cell type for NCP26 and HF treatment is shown in Figure \ref{fig:relapse_deg_bar}.
In contrast to the newly-diagnosed MM data, many of the other immune subtypes are differentially expressed to similar degrees as MM cells by ProRS inhibition.
Additionally, there were more DEGs for the myeloid cluster than for MM cells for 1\si{\micro\Molar} and 5\si{\micro\Molar}  NCP26 treatment.
Myeloid cells have fewer DEGs than cluster 4 MM cells for HF treatment, however, as seen in Figure \ref{fig:composition_relapse}, there is a substantial lower proportion of cells in the myeloid cluster for the HF-treated sample, therefore one may not expect as many statistically significant DEGs for the few cells remaining in the cluster.
Under these experimental conditions, it seems NCP26 and HF may not be as selective for MM cells in relapsed MM as they are for newly-diagnosed MM, and that the ProRS inhibitors may not be very effective against some resistant myeloma subclones.
However, further experiments with varying dosing conditions and samples from multiple relapsed MM patients would be required to validate this.

% Pathway
\subsubsection{Pathway analysis}
Pathway enrichment analysis was performed for the top 200 upregulated and top 200 downregulated DEGs in MM cluster 4, following HF and NCP26 treatment (Figure \ref{fig:relapse_mm_pathway_analysis}).
Pathway analysis could not be performed for MM cluster 9, as there were too few DEGs.
%%%%
%%  Pathway analysis
\begin{figure}[htb]
\centering
\includegraphics[width=\textwidth]{figures/Results/single_cell/relapse_mm_pathway_analysis.pdf}
\caption[MM cluster pathway analysis- relapsed MM]{Pathway analysis (Gene ontology biological processes; GOBP) of multiple myeloma (MM) cluster 4 in the relapsed MM dataset.
GOBP pathway analysis performed using top 200 upregulated or downregulated DEGs for ProRS inhibitor (1\si{\micro\Molar} halofuginone and 1\si{\micro\Molar} or 5\si{\micro\Molar}  NCP26) treatment compared to DMSO control treatment for MM cluster 4.
}
\label{fig:relapse_mm_pathway_analysis}
\end{figure}
%%
Genes involved in translation, NF-$\kappa$B signalling and amino acid metabolism pathways were downregulated following ProRS inhibition (Figure \ref{fig:relapse_mm_pathway_analysis}).
Cell death, apoptotic and cell cycle pathways were enriched following ProRS inhibition.
This follows the previous chapter and established literature, whereby AAR activation triggers cell cycle changes, cell death and a global reduction in the translational machinery.
Interestingly, for all three treatments, mRNA splicing was enriched, indicating a potential spliceosomal mechanism for these inhibitors.
mRNA splicing has been implicated in the mechanism of amino acid starvation before\cite{deval2009amino}, and that the spliceosomal response to amino acid depletion does not require GCN2 activity.
% look up in data if its ribosomal stuff

Figure \ref{fig:relapse_aar_vln_ftp} shows AAR gene expression in the relapsed patients' MM cells, as well as the surrounding immune cells.
%
%%  AAR violin and feature plots
\begin{figure}[htb]
\centering
\includegraphics[width=\textwidth]{figures/Results/single_cell/relapse_aar_vln_ftp.pdf}
\caption[Differentially expressed AAR genes- relapsed MM]{A selection of differentially expressed amino acid starvation response (AAR) genes following 24 hour treatment with ProRS inhibitors (relapsed MM patients).
    a) Violin plots- AAR gene expression in MM cell cluster 4.
    b) Violin plots- AAR gene expression in MM cell cluster 9.
    c) Feature plots of UMAP clustering, showing gene expression of AAR genes \textit{ATF3}, \textit{DDIT3} and \textit{GADD34}.
Feature plots are split into four panels based on treatment condition.
The myeloma subclones are circled in the first panel.
}
\label{fig:relapse_aar_vln_ftp}
\end{figure}
%%
AAR gene expression in MM cluster 4 follows the typical patterns of ProRS inhibition, as seen in the treatment-na\"ive dataset and bulk RNA-seq data: increased expression of \textit{ATF3}, \textit{GADD34}, \textit{GADD45A}, \textit{DDIT3} and amino acid transporters, and decreased expression of some AAR-related genes such as \textit{DDIT4}.
However, the other myeloma subclone (cluster 9) does not display the typical AAR transcriptional changes following ProRS inhibitor treatment, as seen in previous data.
For example, the expression of some AAR genes (such as \textit{ATF3}, \textit{GADD34} and \textit{SLC38A2}) was increased in cluster 9 following NCP26 treatment, as we would expect.
However, the expression of many other AAR genes was unaffected by ProRS inhibitor treatment.
Additionally, the expression of some AAR genes was the inverse to previous data, for example ProRS inhibition caused upregulation of \textit{DDIT4} and \textit{FAU} in cluster 9, where previously HF and NCP26 treatment has been shown to cause downregulation of these genes in MM cells.
This could represent a cyto-protective effect of the AAR in the resistant MM subclone rather than a pro-death signalling response.
In fact, as seen in Figure \ref{fig:composition_relapse}e, HF and NCP26 treatment increased the proportion of cells in MM cluster 9.
%% Past present tense, sounds shit!!!

\subsection{ProRS inhibitor effect on myeloid cells}
% look up INF-gamma etc. VEGFA
From differential expression analysis HF and NCP26 treatment seem to have a large transcriptional effect on myeloid cells, especially monocytes/macrophages.
Composition analysis also revealed that ProRS inhibition significantly reduced the proportion of cells in myeloid clusters, in both treatment-na\"ive and relapsed MM scRNA-seq datasets.
Feature plots and violin plots demonstrated that amino acid starvation response effector genes, such as \textit{DDIT3} and \textit{ATF3} were upregulated in monocytes.
However, myeloid cells are non-malignant.
ProRS inhibitors were originally thought of as good potential candidates as anti-cancer drugs because tumour cells produce large amounts of protein, and are therefore heavily dependent on translational machinery, like tRNA-synthetase enzymes.
This is especially true for myeloma cells, secreting large amounts of M protein.
%Additionally, tumours are usually extremely proline-rich.
Therefore, it is surprising that ProRS inhibitors seem to be non-selective for MM cells over MM patients' non-malignant myeloid cells.

% only BM macrophages. associated with MM. Blood Macrophages might be different
%These results suggest that macrophages protect MM cells from apoptosis via inhibiting Bcl-XL-dependent caspase activation ??
% https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3570938/

\subsubsection{Pathway analysis}
Differential expression and pathway enrichment analysis was performed for NCP26 and HF treatment in monocyte/macrophages.
Cell cycle arrest, apoptosis, programmed cell death and stress responses were enriched in ProRS inhibitor treated samples (Figure \ref{fig:myeloid_pathway_naive} and Figure \ref{fig:myeloid_pathway_relapsed}).
This indicates that the loss of cells in the myeloid cell cluster is due to cell death.
%%  myeloid pathway naive
\begin{figure}[htb]
\centering
\includegraphics[width=\textwidth]{figures/Results/single_cell/myeloid/myeloid_naive_pathway.pdf}
\caption[Myeloid cell pathway analysis- newly diagnosed MM]{Pathway analysis of ProRS inhibitor treatment in monocytes/macrophages in newly-diagnosed MM.
a-c) Halofuginone treatment vs DMSO.
d-f) NCP26 treatment vs DMSO.
a) and d) Gene-set enrichment analysis (GSEA) dotplot of top enriched pathways.
b) and e) XGR gene ontology biological processes (GOBP) of top upregulated genes ($\log_{2}FC >1$).
c) and f) XGR gene ontology biological processes (GOBP) of top downregulated genes ($\log_{2}FC < -1$).
}
\label{fig:myeloid_pathway_naive}
\end{figure}
%%

%%  myeloid pathway relapsed subset
\begin{figure}[htb]
\centering
\includegraphics[width=\textwidth]{figures/Results/single_cell/myeloid/myeloid_relapse_pathway_subset.pdf}
\caption[Myeloid cell pathway analysis- relapsed MM]{Pathway analysis of ProRS inhibitor treatment in monocytes/macrophages in relapsed MM.
a) Gene-set enrichment analysis (GSEA) dotplot of top enriched pathways for 5\si{\micro\Molar} NCP26 treatment vs DMSO.
b) XGR gene ontology biological processes (GOBP) pathway analysis of top upregulated genes (($\log_{2}FC >1$) for 1\si{\micro\Molar} NCP26 treatment vs DMSO.
c) XGR GOBP pathway analysis of top downregulated genes ($\log_{2}FC < -1$) for 1\si{\micro\Molar} NCP26 treatment vs DMSO.
}
\label{fig:myeloid_pathway_relapsed}
\end{figure}
%%
Immune and inflammatory responses were modulated following ProRS inhibition in myeloid cells.
Neutrophil degranulation, secretory granule, lysosome, chemotaxis and migration pathways were suppressed in myeloid cells.
Additionally, mitochondria-related pathways were suppressed, such as oxidative phosphorylation, the electron transport chain and mitochondrial membrane components.
This may indicate some mitochondrial dysfunction in myeloid cells following ProRS inhibition.

\clearpage
\section{Discussion}
This chapter has demonstrated the activation of the AAR and apoptotic mechanisms, as well as other anti-MM effects, following ProRS inhibitors in primary MM patient samples, using scRNA-seq.
In newly-diagnosed and relapsed patients, HF and NCP26 treatment elicited a strong transcriptional change and substantial cell death in the MM cell cluster, this was demonstrated using composition and DE analyses.
This effect was selective for MM cells over NK cells, T cells and B cells.
This follows the original hypothesis that (despite the ubiquity of aaRS enzymes) aaRSs would make a good target in MM, and aaRS inhibitors would be selective for MM cells over benign immune counterparts, due to translational and paraprotein burden on MM cells.
However, it has been shown in DE and composition analyses (of a single relapsed MM patient) that the ProRS inhibitors also seems to elicit apoptosis in myeloid cells.
Additionally, in the relapsed dataset, MM cell cluster 9 appears to be resistant to ProRS inhibitors for 24 hour treatment at the given drug concentrations.

\subsection{Relapsed resistant MM cluster}
Clusters 4 and 9 in the relapsed MM dataset were identified as MM clusters using gene expression data and inferCNV analysis.
From inferCNV, both clusters were found to have myeloma-defining chromosome 13 deletions.
The cells in cluster 9 were shown to express very little or no \textit{CD138}.
As discussed in Section \ref{subsec:mm_class_diss}, CD138\textsuperscript{-} MM cells have been reported to have a higher proliferative potential than CD138\textsuperscript{+} plasma cells.
Often the presence of CD138\textsuperscript{-} MM cells is also associated with poor prognosis.
A decrease in CD138 expression is seen in MM patients with relapsed/progressive disease compared with untreated MM patients\cite{kawano2012multiple}.
This was corroborated by our scRNA-seq data, whereby there was no CD138\textsuperscript{-} MM cell population in the newly-diagnosed MM dataset, but there was for the relapsed MM dataset.
Decreased \textit{CD138} expression has previously been associated with reduced sensitivity to the MM drug, lenalidomide\cite{kawano2012multiple}.
Cluster 9 in the relapsed dataset (with no or little \textit{CD138} expression) was found to be largely resistant to 24 hour treatment of NCP26 and HF treatment at the given concentrations, whereas CD138\textsuperscript{+} cluster 4 was sensitive to ProRS inhibition.
This suggests that CD138\textsuperscript{-} MM cells may have increased resistance to anti-MM drugs.
However, this experiment only looked at 24-hour ProRS inhibitor treatment, for only two relapsed MM patients.
Many MM scRNA-seq studies (such as Cohen et al. 2021, which researched MM drug-resistance pathways\cite{cohen2021identification}), use CD138\textsuperscript{+} plasma cell enrichment techniques prior to sequencing.
Thereby, the researchers likely missed subclonal MM populations, which may have been contributing greatly to resistant mechanisms in MM\@.
More MM researchers, especially those interested in drug resistance mechanisms, should sequence the whole BM niche, not just the CD138\textsuperscript{+} fraction.
The MM classifiers generated in Chapter 4 can simplify the process of MM cell identification computationally, so that researchers are less reliant on CD138\textsuperscript{+} enrichment techniques.

Differential expression analysis of relapsed MM subclonal clusters (cluster 4 vs cluster 9) revealed signature markers of drug-resistance (Figure \ref{fig:cluster4_9_pathway}).
Genes involved in the unfolded protein response pathway were suppressed, as well as ER-associated genes.
This is a known mechanism action of PIs, therefore it makes sense that the UPR is modulated in patients that have been treated with PIs previously then relapsed.
The execution of phase of apoptosis was also negatively regulated, indicating cells in cluster 9 are more resistant to apoptotic signals.

\subsection{Effect on myeloid cells}
From the results it is clear that, at the given concentrations,  NCP26 and HF 24-hour treatment adversely affected monocytes and macrophages.
ProRS inhibition elicited a strong transcriptional response, and the myeloid cell cluster was significantly reduced following treatment.
Additionally, from DE analysis and pathway enrichment analysis, genes involved in apoptotic mechanisms and cell cycle arrest were enriched, indicating that the reduction of cells in the myeloid cluster was due to cell death.

Previously, Leiba et al. (2012) demonstrated HF's selectivity for MM patients' CD138\textsuperscript{+} tumour cells over PBMCs from healthy donors\cite{leiba2012halofuginone} using a cytotoxicity assay at the range of 25-200\si{\nano\Molar}.
This is a significantly lower concentration of HF than used in the scRNA-seq experiments (1\si{\micro\Molar}), so this could represent dosing differences.
However, it must also be noted that Leiba et al. (2012) did not separate PBMCs into their respective subtypes prior to performing the cytotoxicity assays.
As demonstrated in this chapter, HF treatment has very little effect on B, T and NK cells.
Monocytes only make up approximately 5-10\% of the cells found in PBMCs-- therefore, the effect of HF on myeloid cells could have been missed due to the minimal effects on the other cell types making up the majority of PBMCs.
Furthermore, the control PBMCs originated from healthy donors and not from MM patients.
The immune microenvironment of MM patients has been shown to be substantially altered and impaired compared to healthy individuals\cite{de2013analysis}, meaning that using healthy PBMCs as a control for the selectivity of HF may not be entirely appropriate.
To truly determine the cytotoxic selectivity of ProRS inhibition for MM cells and myeloid cells, cytotoxic assays must be performed in parallel for MM cells and myeloid cells originating from MM patients.

Wang et al. (2020) investigated the use of halofuginone to treat the inflammatory disease chronic periodontitis\cite{wang2020halofuginone}.
Using a mouse model of chronic periodontitis, they demonstrated that HF treatment significantly reduced the expression levels of pro-inflammatory cytokines, for example IL-1$\beta$, IL-6 and TNF-$\alpha$.
HF treatment was also shown to reduce the total percent of infiltrating immune cells and myeloid cells in gingival tissues when compared with those in control, PBS-treated mice.
The percentage of myeloid cells making up the CD45\textsuperscript{+} fraction of cells in gingival tissue was reduced by approximately 40\%, following HF treatment.
The authors demonstrated that HF treatment did not affect the cell viability of BM-derived osteoclast precursors cells (that is monocytes and macrophages).
However, cell viability was investigated with concentrations of HF ranging from 10\si{\pico\Molar} to 100\si{\nano\Molar}, so perhaps myeloid death is reserved for higher doses of HF treatment.
Our scRNA-seq experiments used 1\si{\micro\Molar} HF, so this may be the concentration levels required to observe ProRS inhibitory effects on myeloid cells.

Previously, the proteasome inhibitor MG132 was shown to induce macrophage apoptosis\cite{wang2021proteasome}.
A number of proteasome inhibitors have been approved for use in multiple myeloma, in both newly-diagnosed and relapsed MM, and they are regarded as part of the cornerstone for modern MM treatment.
Macrophage apoptosis has not been a barrier in the use of PIs in MM, therefore myeloid cell death may not limit the usage of ProRS inhibitors in the clinic.
As discussed in the previous chapter, amino acid depletion has been implicated in the mechanism of action of proteasome inhibitors.
Wang et al. (2021) asserted that macrophage apoptosis following MG132 treatment was due to promoting ROS production and mitochondrial dysfunction\cite{wang2021proteasome}.
As shown above, ProRS inhibition in myeloid cells resulted in suppression of several mitochondrial pathways, as well as induction of oxidative stress.
Therefore, mitochondrial dysfunction may play a role in the adverse effect of ProRS inhibition on myeloid cells.

GCN2 has been shown to have an important role in myeloid cell homeostasis and immune function\cite{halaby2019gcn2,ravishankar2015amino}, it has also been suggested to be a critical mediator of apoptosis vs autophagy during inflammation.
Targeting amino acid metabolism and GCN2 function in myeloid cells, has been suggested as a great potential clinical target\cite{liu2014role}.
One of the many roles of macrophages is clearance of apoptotic cells\cite{gordon2018macrophage}.
In the presence of apoptotic cells, significant induction of \textit{DDIT3} expression, ATF4 protein, increased phosphorylation of eIF2$\alpha$ and metabolic stress has been observed in macrophages\cite{ravishankar2012tolerance, ravishankar2015amino}.
This response was not elicited in the absence of GCN2\cite{ravishankar2015amino}, perhaps indicating that the presence of other apoptotic cells (such as apoptotic MM cells) may further activate GCN2 and AAR signalling in myeloid cells.
If this were the case, it would explain why in isolated toxicity studies no selectivity is shown for myeloid cells, and why myeloid cells are more sensitive to ProRS inhibition than other immune cells.

% Conversely, a subset of myeloid cells, such as tumour-associated macrophages (TAMs), have been implicated in MM cell survival, protection from drug-induced apoptosis, and drug resistance mechanisms.
%So perhaps if these TAMs are killed by HF and NCP26 this could actually have a positive impact on treatment by removing the protection and delaying drug resistance onset.
%% DEath of myeloid cells could help killing cancer cells then, and delay drug resistance. Good thing? Negative impact on treatment
%% https://pubmed.ncbi.nlm.nih.gov/27976373/
% other therapies suppress bone marrow and kill immune cells.
% Mouse after 6 weeks no weight loss- healthy other than tumour.

The effects of ProRS inhibition in myeloid cells should be examined in more detail.
The degree of myeloid suppression from ProRS inhibition should be investigated in CD14+ cells collected from numerous MM patient blood samples (less invasive than bone marrow samples).
Preferential selectivity of ProRS inhibitors for MM cells should be confirmed using dose-response experiments, using MM cells and MM patient-derived immune cells, separated into their constituent cell types.
These experiments should be performed with and without apoptotic cells present.
It would also be interesting to see if the effect of ProRS inhibition on myeloid cells extends to healthy individuals or other disease pathologies.
Moreover, the mechanism underlying cell death in myeloid cells should be elucidated.
The role of mitochondrial dysfunction in ProRS inhibition should be explored.
Another avenue to investigate could be the suppression of immune/inflammatory pathways seen in myeloid cells following HF and NCP26 treatment.
Additionally, the non-canonical functionality of EPRS should be investigated.
HF and NCP26 inhibit the proline domain of the bifucntional aaRS EPRS\@.
EPRS is a member of the MSC\@.
EPRS is released from the MSC in myeloid cells upon IFN-$\gamma$ stimulation, which causes a two-step phosphorylation of two serines in the linker region of human EPRS\cite{arif2009two}.
EPRS combines with other proteins (NSAP1, L13a and GAPDH) to form the GAIT complex, which represses translation of numerous inflammatory-related transcripts\cite{arif2018gait}.
IFN-$\gamma$ signalling has been implicated in the mechanism of action of ProRS inhibitors\cite{cheng2012halofugine}.
Therefore, this non-canonical functionality of EPRS could play a role in ProRS inhibition's mechanism of action and could potentially explain the cytotoxic-selectivity of HF and NCP26 for myeloid cells.
For example, the formation of the GAIT complex could plausibly affect availability of ProRS for its canonical function.
It would be interesting to see if other aaRS inhibitors, (acting on other aaRSs which do not engage in GAIT complex non-canonical functionality), have a similar effect on MM and myeloid cells to determine if the non-canonical functionality of EPRS plays any role in the mechanism of ProRS inhibitors.

A decrease in the number of functional myeloid cells could lead to myeloid suppression (myelosuppression/ bone marrow suppression/ immunosupression), which in turn makes patients more susceptible to infection.
As MM is a disease of elderly people, new therapeutics must be tolerable for patients and not excessively immunosuppressive, especially if they are used to treat RRMM patients, who have increased bone marrow suppression.
Substantial myelosuppression can also dictate the dosing regimen of a drug, to balance the efficacy of the drug with the associated side effects and infection risk.
However, from in vivo MM xenograft murine models\cite{bottpreclinical2022}, the mice tolerated NCP26 treatment well, with no signs of overt toxicity such as weight loss.
This indicates that the effect of ProRS inhibition on myeloid cells may not even be an issue in vivo.
Furthermore, myelosuppression is a common side effects of many other cancer therapies, especially in the treatment of hematologic malignancies.
Therefore, it would be unlikely that ProRS inhibitor myeloid cell killing would constrain its clinical use in MM\@.

\section{Future directions}
% Summary few sentences
% then into future work
This work has demonstrated the effectiveness of ProRS inhibition in MM using both MM cell lines and primary patient BM samples, and accordingly presents a novel therapeutic target in MM\@.
Further pre-clinical validation of aaRS inhibition in MM is required.
The aim of pre-clinical studies is to demonstrate the effectiveness of a target in a disease model prior to moving to human testing.
These models aim to recapitulate the disease as accurately as possible, and in the case of MM, provide insights on the interactions between malignant cells and the surrounding microenvironment.
The scRNA-seq experiments in this work constitute samples from four MM patients, two of whom are newly-diagnosed and two of whom have relapsed.
Considering MM is an extremely heterogeneous disease, this small sample size does not reflect the total variation seen in MM in the wider population.
Additionally, only acute treatment of ProRS inhibitors over 24 hour and 48 hour periods was studied.
More scRNA-seq experiments should be performed with varying dosing conditions, with BM samples from many more patients.

In the clinic, treatment cycles often last between four and six months.
Our scRNA-seq methodology is limited by the duration human BM samples last in culture.
The in vivo murine model used in \cite{bottpreclinical2022} to test NCP26 demonstrated that NCP26 was well tolerated for up to six weeks-- a significantly longer continuous exposure time than we managed for scRNA-seq.
However, mouse models aren't without their limitations.
For example, severe combined immunodeficient (SCID) mice (as used in \cite{bottpreclinical2022}) have a genetic immune deficiency that means they lack B and T cells, allowing for xenoengraftment of human tissue.
This means that the effect of compounds on B cells and T cells cannot be monitored.
Moreover, subcutaneous injection of MM cells mean that the MM cells proliferate in the absence of the bone marrow microenvironment\cite{rossi2018mouse}, which supports the development, migration, proliferation and differentiation of malignant plasma cells, as well as contributing to acquired anti-cancer drug resistance.
Thereby, not modelling a key factor in MM pathology.
However, this model does allow for easy monitoring and measuring of tumour size during treatment.
Each mouse model platform has its own strength and weakness, so it is important to select the right model for each specific pre-clinical research question.
Recently, in a pre-print by Khan et al. (2022) a new step-wise, directed-differentiation protocol was described to generate 3D, vascularised BM organoid structures, which capture key features of human BM (stroma, lumen-forming sinusoidal vessels and myeloid cells)\cite{khan2022human}.
This platform supports growth of primary cells from blood cancer patients and would allow for ex vivo mechanistic studies of MM cells and their interactions within a human BM-like milieu\cite{khan2022human}.
MM cells that had been cryopreserved and thawed prior to use were shown to rapidly lose viability when plated in vitro, however, were sustained by the organoid platform for more than 10 days\cite{khan2022human}.
Several other 3D ex vivo BM niche models for MM have been proposed\cite{jakubikova2016novel,ferrarini20173d}.
Utilisation of these platforms would allow for investigation of ProRS inhibitors within a MM BM microenvironment (with all relevant immune cells), for longer exposure time than our previous methodology would allow.

Prior to clinical application of ProRS inhibitors, a lead compound should be developed to target ProRS, that shares NCP26's proline-uncompetitive mode of action.
NCP26 has the properties of a good chemical tool: its biological target is well-defined (ProRS), it demonstrates selectivity, and it has $<$ 1\si{\micro\Molar} target potency for cell-based assays.
However, drugs which are used in the clinic have many other requirements in terms of their pharmacokinetics, pharmacodynamics, and bioavailability.
Therefore, a compound is required which targets ProRS, but also has drug-like properties.
NCP26 was well tolerated in a murine model, so might be a good drug candidate for clinical use, however its pharmacokinetic and pharmacodynamic properties need further investigation.

Borrelidin, (ThrRS inhibitor)  and CysSA, (cysteinyl adenylate substrate analogue) were both shown to reduce proliferative activity of several MM cell lines\cite{bottpreclinical2022}.
The effectiveness of other aminoacyl tRNA synthetase inhibitors should also be tested in MM\@.
This could present a whole new set of potential therapeutics to treat MM\@.
Moreover, some could be more effective or appropriate in MM, with different characteristics or side-effect profiles.
aaRSs with different non-canonical functions could present differently.
Compound screens for all known aaRS inhibitors should be conducted across numerous MM cell lines.

Prolyl-tRNA synthetase (ProRS) is an enzyme which catalyzes the coupling of tRNA\textsuperscript{pro} to proline, therefore the effect of ProRS inhibitors on tRNAs is of great interest.
Additionally, transfer RNA-derived small RNAs (tDRs) have been shown to play pivotal roles in stress responses\cite{li2022distinct}.
tRNA-seq should be performed for ProRS inhibitor treated MM cells and ProRS inhibitor resistant cell lines.
tRNAnalysis introduced in Chapter \ref{ch:4-Pipelines} would then be used to analyse the resulting data.
tRNA charging should also be investigated for ProRS inhibitors, to confirm the mechanism of action upstream of GCN2 phosphorylation and activation of \textit{ATF4} target genes.
The proportion of charged and uncharged tRNAs for ProRS inhibitor treated MM cells could be calculated by performing a qPCR-based experiment.
This constitutes a future area of research to understand the effects of ProRS inhibitors on a molecular level.


% Alternative splicing...
%(paper: The
%7 complexity of ProRS inhibitor responses and connection to Myc-mediated proliferation and
%8 survival is further highlighted by regulation of RNA splicing and RNA binding proteins such as
%9 HNRNPK (54), as observed in the patient derived myeloma samples, linking post-transcriptional
%10 and translational processes as an important axis to regulate the phenotypic responses.)
%
%https://www.mdpi.com/1422-0067/19/2/545/htm
%Borrelidin ThrRS threonyl-tRNA synthetase
%In colon tumor cells, the spliceosome-associated protein FBP21 (formin binding protein 21) was the target of borrelidin
%
%
%Reduction in CD138 (SDC1), essential survival factor?? regulating its bone
%7 marrow localisation and microenvironment interactions.. CD138- myeloma though
%Abrogates syntheis. ref 65 in paper
%
%Resistant cells, resistance mechanisms to GCN2 related stressors??
