\chapter{\label{ch:6-sc}Single-cell RNA-seq analysis of PRS inhibitors}

%\minitoc

\section{Introduction}
MM cells grow within the bone marrow and are supported as they grow by their microenvironment.
The MM microenvironment comprises a cellular compartment (composed of immune cells, endothelial cells, osteoblasts, osteoclasts and stromal cells) and a non-cellular compartment (composed of the extracellular matrix (ECM), cytokines, chemokines and growth factors)\cite{manier2012bone, kawano2015targeting}.
There are interactions between malignant plasma cells and the surrounding microenvironment.
The bone marrow microenvironment has been indicated to play a supportive role in migration, proliferation, differentiation and drug resistance of malignant plasma cells.
There is evidence linking the tumour microenvironment to progression of MGUS to active MM, for example significant matrix remodelling has been seen between the bone marrow of healthy individuals, MGUS and MM patients\cite{kawano2015targeting}.
Therefore, to get an accurate picture of MM, information must be acquired about the surrounding niche.

Historically, the tumour environment has been investigated following the isolation of populations of cells sorted from the tumour and then sequenced using traditional microarray or bulk RNA-seq techniques.
Bulk techniques measure the average expression across a sample, which is the sum of cell type specific expression weighted by cell type proportions.
Single-cell techniques measure expression for each individual cell and therefore provide information on clonal diversity that may be lost when pooling cells into bulk samples.
Furthermore, multiple myeloma is an extremely heterogeneous disease, this is seen both between patients and within an individual's own tissue.
Applying single-cell techniques to capture the inter- and intra-individual heterogeneity is fundamental to identifying molecular and cellular signatures that define MM\@.

The advent of single-cell technologies has led to a better understanding of the complexity and diversity of the tumour microenvironment.
Seminal papers from Melnekoff et al. (2017)\cite{melnekoff2017single} and Ledergor et al. (2018)\cite{ledergor2018single} use scRNA-seq to reveal clonal transcriptomic heterogeneity in MM samples.
Melnekoff et al. (2017) demonstrated the clonal heterogeneity within MM using samples that were collected from eight relapsed MM patients.
The group performed t-SNE clustering analysis and the samples separated into eight transcriptionally distinct clones, each corresponding to a different patient.
This highlights the inter-patient differences of MM\@.
Ledergor et al. (2018) performed a similar study to evaluate clonal heterogeneity within MM but also had a set of controls with which to compare the MM group.
They found that MM patients have greater inter-individual transcriptional variation, where each MM patient possessed a unique and individual plasma cell transcriptional program.
They also showed substantial intra-tumour heterogeneity (subclonal structures) of plasma cells in a third of their MM patient cohort.
These papers established the importance of using single-cell techniques to study MM, as to not miss the underlying clonal heterogeneity.
However both of these papers focussed soley on plasma cells and did not look at the surrounding bone marrow microenvironment.
To truly understand the complexities of MM and treatment of MM, interactions between plasma cells and the bone marrow niche must also be explored using single-cell techniques.


\subsection{Experiment overviews}
Two single-cell experiments were performed to explore the effect of various compounds (including Halofuginone and NCP26) on MM patient tissue.
For experiment 1, bone marrow (BM) samples were collected from a treatment naive <xyz> MM patient.
The BM samples was treated for 24 hours with 1\si{\micro\Molar} Casin, GSK-J4, Halofuginone, NCP26, SGC-GAK, Verteporfin or a DMSO control, totalling seven conditions.
For experiment 2, BM samples were collected from a treatment naive <xyz CD56 etc> MM patient.
The BM samples were treated with 1\si{\micro\Molar} CAMKK2, CLK or CSNK2 for 24 hours, and 1\si{\micro\Molar} SGC-GAK, Halofuginone, NCP26 or a DMSO control for 24 and 48 hours, totalling 11 conditions.
Following compound treatment, 10X single-cell RNA-seq library preparation was performed separately for each experiment, by Dr Philpott, as outlined in section XYZ.

\section{Data processing}
All 18 samples were included in the analysis up to and including the integration and annotation stage of data processing.
This was to increase the granularity of the data, and allow for easier annotation of clusters.
Downstream analysis was only performed for DMSO, Halofuginone or NCP26 treated samples.


The single cell analysis pipeline outlined in section <ENTER section> was used to process the data.
Kallisto BUS/ BUStools was used to pseudoalign reads and quantify gene expression.
Next, quality control and filtering of the samples was performed.
Poor quality cells are likely to have a low number of genes and UMIs per cell.
Any cells with fewer than 500 UMIs were removed.
Cells with a gene count below 300 or above 6000 were removed.
Cells with a mitochondrial ratio higher than 0.1 were removed (a high proportion of mitochondrial genes indicates mitochondrial contamination from dead or dying cells).
After quality control and filtering of the data, clustering was performed using Seurat v3, followed by integration of all samples, using Seurat v3 and Harmony.
The two experiments were too large to integrate across all samples using the Seurat SCTransform normalisation method, therefore a reference-based approach was taken, whereby a subset of samples were selected (based on their cell richness and relevance to the research question) and listed as `reference datasets' for SCTransform normalisation.
Following integration of the samples, annotation of the cells was performed using a combination of automated packages: SingleR, Clustifyr and scClassify, and manual annotation using known biological markers.
The HumanCellAtlas database was used to inform singleR and clustifyr annotation.
scClassify was performed using a pre-trained scClassify model, based on seven PBMC single-cell datasets (including 10X V2, 10X V3, smartSeq, celSeq, dropSeq and inDrops datasets).
As the reference datasets originated from healthy tissue, they are unable to label pathological cells, such as myeloma cells.
Myeloma cells were identified manually using a range of biological markers, including: CD38, CD138, SLAMF7 (CD319), BCMA, IGKC, IGLC2, KRAS, CD45, CD20, and CD19.

% Pre RBC dimplots
\begin{figure}[htb]
    \centering
    \includegraphics[width=0.9\textwidth]{figures/Results/single_cell/data_processing/pre_RBC_removal_dimplots.pdf}
    \caption[Integrated experiments pre-erythrocyte removal]{UMAP dimension plots following integration of samples from experiment 1 and 2.
    A) Annotated UMAP of integrated experiments 1 and 2. The erythocyte clusters are circled in black.
    B) UMAP of integrated experiments, cells coloured by the experiment they originated from. The erythrocyte population of cells appears to originate mostly from experiment 2. }
    \label{fig:umap_pre_RBC}
\end{figure}

Experiment 2 was found to have an extremely high erythrocyte population (figure \ref{fig:umap_pre_RBC}).
In addition, many cell populations were expressing erythrocyte specific genes, where we would not expect to see them being expressed, for example MM cells expressing numerous haemoglobin subunit genes.
The high expression of these erythrocyte genes was found to affect the integration of the two experiments together.
It is thought that perhaps the BM sample taken for experiment 2 was one of the last samples taken and contained a large amount of blood.
Library preparation clean-up may have missed many of these cells, leading to ambient erythrocyte RNA being present within many droplets.
It was decided to remove the cells from the erythrocyte clusters (3, 8, 9, 10, 11, 16) and haemoglobin related genes/ erythrocyte specific genes that seemed to be dominating expression in the integrated dataset.
Following the removal of these genes and cells, integration and clustering was performed again.
Harmony was used to re-integrate all samples, using a multi-covariate model, accounting for batch effects for each sample and for the two separate experiments.
Better integration of the two experiments was achieved (figure \ref{fig:umap_post_RBC}).


% Pre RBC dimplots
\begin{figure}[htb]
    \centering
    \includegraphics[width=0.9\textwidth]{figures/Results/single_cell/data_processing/RBC_removed_dimplots.pdf}
    \caption[Integrated experiments post-erythrocyte removal]{UMAP dimension plots of experiment 1 and 2 re-integrated after removal of erythrocytes and erythrocyte specfic genes.
    A) Clustering of integrated data.
    B) UMAP of integrated experiments, cells coloured by the experiment they originated from.
    Better overlap of the two experiments. }
    \label{fig:umap_post_RBC}
\end{figure}

\subsection{Annotation of re-integrated data}

\input{text/Chapter_6_single-cell_PRS/markers_table.tex}

Feature plots, violin plots, dot plots.

\section{Results}

\subsection{Composition}
Cluster composition is shown in figures \ref{fig:sc_comp_clusters} and \ref{fig:sc_umap_comp}.
Large number of cells were killed by Halofuginone treatment at both 24 and 48 hours.
Clusters 0, 1 and 5 (T-cells, monocytes and MM cells) seem particularly affected.

%% SCT UMAP cluster composition separated by treatment condition
\begin{figure}[htb]
\centering
\includegraphics[width=0.7\textwidth]{figures/Results/single_cell/composition_umap.png}
\caption[UMAP cluster composition]{SCT integrated UMAP clustering. Cell composition separated by treatment conditions. }
\label{fig:sc_umap_comp}
\end{figure}
%%


% Barchart and Dotplot
\begin{figure}[pt]
%1
\begin{subfigure}[t]{0.5\textwidth}
    \includegraphics[width=\textwidth]{figures/Results/single_cell/composition_dotplot.png}
    \caption{Dot plot cell proportions}
    \label{fig:comp_dot}
\end{subfigure}
%2
%\medskip
\begin{subfigure}[t]{0.5\textwidth}
    \includegraphics[width=\textwidth]{figures/Results/single_cell/composition_dotplot_nCells.png}
    \caption{Dot plot total cell number}
    \label{fig:comp_dot_ncells}
\end{subfigure}
%
\medskip
%3
\begin{subfigure}[t]{0.5\textwidth}
    \includegraphics[width=\textwidth]{figures/Results/single_cell/composition_barchart.png}
    \caption{Bar chart cell proportions}
    \label{fig:comp_bar}
\end{subfigure}
%4
\begin{subfigure}[t]{0.5\textwidth}
    \includegraphics[width=\textwidth]{figures/Results/single_cell/composition_barchart_nCells.png}
    \caption{Bar chart total cell number}
    \label{fig:comp_bar_ncells}
\end{subfigure}
%
\caption[Single-cell cluster composition]{Single-cell cluster composition for each sample, either as a proportion of cells for each sample or raw cell counts.}
\label{fig:sc_comp_clusters}
\end{figure}


\subsection{Myeloma cells}

\subsection{T cells}

\subsection{Monocytes}


\section{Myeloma bone marrow classifier}