\chapter{\label{ch:2-litreview}Background}

%\minitoc

\section{Metabolic changes in MM}

\subsection{The amino acid starvation response}

\section{Drug resistance in MM}

\subsection{Genomic changes in drug resistant MM}


\subsection{Epigenetic changes in drug resistant MM}


\subsection{Metabolic changes in drug resistant MM}


\section{Preliminary work}\label{sec:preliminary}
% 2
\subsection{Epigenetic compound library screen}
REMAKE FIGURE--- LOADS OF SPELLING MISTAKES AND LOOKS AWFUL!!!
Previously in the Oppermann group laboratory, Dr James Dunford performed a compound screen against WT and proteasome inhibitor-resistant AMO-1 cells with an epigenetic compound library (figure \ref{fig:epigenetic_pie}) and cell viability assays.

%% Pie chart of compound library
\begin{figure}[htb]
\centering\includegraphics[width=0.9\textwidth]{figures/Introduction/epigenetic_library_pie.png}
\caption[Epigenentic compound library pie chart]{The Oppermann group epigenetic compound library.
Proportions of targets in the 144-compound library. }
\label{fig:epigenetic_pie}
\end{figure}
%%

The results from the screen can be seen in figure \ref{fig:compound_screen}.
Six compounds, shown circled in red, were identified as compounds of interest.
SGC-CBP30, TRIM24i, OF1 and GSK959 are bromodomain inhibitors, TMP269 is a HDAC inhibitor and TDOSI000054a (T54) is a methyl lysine binder.
These compounds were of interest as they had little to no effect on WT cells and decreased cell viability of resistant cells in the presence of their respective proteasome inhibitor.
This indicated that the compounds were not just killing cells with their own distinct mechanism of action, but instead seemed to be reversing proteasome inhibitor resistance and re-sensitising resistant AMO-1 cells to proteasome inhibitors.
These compounds were then taken forwards and used to treat carfilzomib resistant AMO-1 cells (aCFZ) and bulk RNA-seq perform to try and understand their mechanism of action.

%% Compound screen -- remake figure at some point.
\begin{figure}[hp]
\centering\includegraphics[width=0.9\textwidth]{figures/Introduction/python_jim_screening_heatmap_annotated.png}
\caption[Epigenentic compound library screen]{Preliminary epigenetic compound library screen, previously performed in the lab by Dr James Dunford.
138 epigenetic compounds were screened against proteasome inhibitor sensitive (WT) AMO-1 cells, carfilzomib resistant (aCFZ) and bortezomib resistant AMO-1 cells, in the presence of their respective proteasome inhibitor (+ 100\si{\nano\Molar} B/C ) and without (no B/C).
Six compounds of interest were identified, circled in red in the figure.}
\label{fig:compound_screen}
\end{figure}
%%



Bromodomains (BRDs) are conserved modular protein-protein interaction domains.
Primarily, BRDs recognise acetylated lysine (Lys) residues in histone tails.
Proteins containing bromodomains are epigenetic regulators and regulate gene expression (on their own or as part of a larger complex) via chromatin remodelling, histone modification, histone recognition and transcriptional machinery regulation [REPHRASE].
Proteins containing BRDs have frequently been seen to be dysregulated in cancer.
The compound SGC-TRIM24 is a dual inhibitor that targets the bromodomains of TRIM24 and BRPF.


( cite: https://www.nature.com/articles/nrm.2016.143)


Bromodomains are ``readers'' that bind acetylated lysines in histone tails


\subsection{Metabolic compound library screen}