\chapter{Supplementary figures}

% ARS2 gene barcharts
\begin{figure}[h]
%1
\centering
\begin{subfigure}{\textwidth}
    %\centering
    \includegraphics[width=\textwidth]{figures/appendix/aaRS2_wt_barchart.pdf}
    \caption{WT AMO-1 cells}
\end{subfigure}
\medskip
%2
\begin{subfigure}{\textwidth}
    \includegraphics[width=\textwidth]{figures/appendix/aaRS2_cfz_barchart.pdf}
    \caption{CFZr cells}
\end{subfigure}
\caption[Bulk RNA-seq: mitochndrial aaRS gene expression change]{Mitochondrial aaRS gene expression change ProRS inhibitor/carfilzomib treated vs DMSO control.
Stars indicate significance at adjusted p-value < 0.05.}
\label{fig:ARS2_barchart}
\end{figure}

% western blot heatmap
\begin{figure}[h]
\centering
\includegraphics[width=0.5\textwidth]{figures/appendix/western_gcn2_supplementary.pdf}
\caption[GCN2 and eIF2$\alpha$ western blot ]{Western blot demonstrating dose-dependent NCP26 canonical ISR activation with GCN2 and eIF2$\alpha$ phosphorylation. AMO-1 and MM.1S myeloma cell lines used.
Experiment performed by international collaborators on a multiple-lab collaborative research paper\cite{bottpreclinical2022}.
}
\label{fig:sup_western}
\end{figure}
%%

% Proline supplementation
\begin{figure}[h]
\centering
\includegraphics[width=0.5\textwidth]{figures/appendix/excess_proline.pdf}
\caption[ProRS inhibitor treatment with proline supplementation]{ProRS inhibitors supplemented with excess proline.
AMO-1 cells treated with 500\si{\nano\Molar} of halofuginone or NCP26 for 48 hours in the presence of varying concentrations of proline (0, 1, 5, 10, and 20 \si{\milli\Molar}).
Experiment performed using triplicates.
Bars represent mean cell viability.
Experiment performed by international collaborators on a multiple-lab collaborative research paper\cite{bottpreclinical2022}.
}
\label{fig:sup_proline_excess}
\end{figure}
%%

% Heatmap MTT proliferation, MM cell lines other aaRS
\begin{figure}[h]
\centering
\includegraphics[width=0.5\textwidth]{figures/appendix/MM_cell_lines_other_aaRS.pdf}
\caption[Other aaRS inhibitors' anti-proliferative activity in MM cell lines]{Anti-proliferative activities of aaRS inhibitors in MM cell lines.
Experiment performed by international collaborators on a multiple-lab collaborative research paper\cite{bottpreclinical2022}.
1\si{\micro\Molar} NCP26, NCP22, halofuginone, halofuginol (MAZ1805), borrelidin (threonyl-tRNA synthetase/ThrRS inhibitor), or 5 \si{\micro\Molar} L-ProSA, D-ProSA and CysSA (aminoacyl adenylate substrate analogues).
72 hour MTT assay, n=2-5 independent experiments in triplicate technical repeats).
}
\label{fig:sup_cell_line_aaRS}
\end{figure}
%%

% Bulk NCP26R sup
\begin{figure}[h]
\centering
\includegraphics[width=\textwidth]{figures/appendix/NCP26R_supp1.pdf}
\caption[NCP26R bulk RNA-seq extra figures]{NCP26R vs control L363 bulk RNA-seq.
a) and b) sample clustering.
a) PCA plot- separated across PC1 into two distinct groups: control L363 samples and NCP26 resistant (NCP26R) L363 samples.
b) Sample-sample distance clustering.
c) Mitochondrial aaRS gene Log2FC L363 controls vs NCP26R. Stars indicate differential expression at p\textsubscript{adj}<0.05.
d) GSEA CNET plot for suppressed ER-associated pathways.
e) GSEA CNET plot for top enriched pathways. Numerous activated DNA packaging pathways and many histone genes upregulated.
}
\label{fig:sup_NCP26R_bulk}
\end{figure}
%%

% Marker table
\afterpage{\clearpage}
\input{text/Chapter_6_single-cell_PRS/markers_table.tex}

% inferCNV
\begin{figure}[h]
    \centering
    \includegraphics[width=\textwidth]{figures/Results/single_cell/data_processing/inferCNV_naive.pdf}
    \caption[inferCNV- newly-diagnosed MM]{InferCNV results for the newly-diagnosed MM dataset.
    [a and b] InferCNV heatmaps.
        The top panel shows expression values for the reference `normal' cells.
        The bottom panel shows expression values for the suspected malignant cells (clusters 2, 7 and 13), and other B-cell lineages (B cells and plasma cells).
        Red indicates chromosomal region amplifications and blue indicates chromosomal region deletions.
    a) De-noised inferCNV results.
    b) Hidden Markov-Model (HMM) copy number variation (CNV) region predictions.
        Only some chromosomal region gains predicted in MM clusters 2 and 13, and the plasma cell cluster.
    }
    \label{fig:inferCNV_naive}
\end{figure}
%

% cluster 4 9 pathway
\begin{figure}[h]
    \centering
    \includegraphics[width=\textwidth]{figures/appendix/cluster4_9_pathway.pdf}
    \caption[scRNA-seq relapsed MM subclone pathway enrichment]{scRNA-seq relapsed MM subclones (cluster 4 and 9) pathway enrichment analysis.
Cluster 4 (ProRS inhibitor sensitive) vs Cluster 9 (ProRS inhibitor resistant at given experimental conditions).
    a) ClusterProfiler gene set enrichment analysis (GSEA) dotplot.
    b) XGR gene ontology biological processes (GOBP) for top upregulated genes ($\log_{2}FC > 0.5$ and p\textsubscript{0.05} < 0.05).
    c) XGR gene ontology biological processes (GOBP) for top downregulated genes ($\log_{2}FC < -1$ and p\textsubscript{0.05} < 0.05).
    Unfolded protein response is suppressed, as well as ER-associated pathways. Negative regulation of apopotosis execution is enriched.
    }
    \label{fig:cluster4_9_pathway}
\end{figure}
%